{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Kaggle Medicine - NB 1"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## 1. Import des librairies "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [],
   "source": [
    "import numpy as np\n",
    "import pandas as pd\n",
    "import gensim"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [],
   "source": [
    "training_text = pd.read_csv(\"../Data/W2V_training_text\", delimiter='\\|\\|', engine = 'python')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 25,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>ID</th>\n",
       "      <th>Text</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>0</td>\n",
       "      <td>Cyclin dependent kinases  regulate a variety o...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>1</td>\n",
       "      <td>Abstract Background  Non small cell lung canc...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>2</td>\n",
       "      <td>Abstract Background  Non small cell lung canc...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>3</td>\n",
       "      <td>Recent evidence has demonstrated that acquired...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>4</td>\n",
       "      <td>Oncogenic mutations in the monomeric Casitas B...</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "   ID                                               Text\n",
       "0   0  Cyclin dependent kinases  regulate a variety o...\n",
       "1   1   Abstract Background  Non small cell lung canc...\n",
       "2   2   Abstract Background  Non small cell lung canc...\n",
       "3   3  Recent evidence has demonstrated that acquired...\n",
       "4   4  Oncogenic mutations in the monomeric Casitas B..."
      ]
     },
     "execution_count": 25,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "training_text.head()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {},
   "outputs": [],
   "source": [
    "training_text.dropna(axis=0, inplace=True)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 29,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "array(['Cyclin dependent kinases  regulate a variety of fundamental cellular processes  CDK stands out as one of the last orphan CDKs for which no activating cyclin has been identified and no kinase activity revealed  Previous work has shown that CDK silencing increases ETS  driven activation of the MAPK pathway  which confers tamoxifen resistance to breast cancer cells  The precise mechanisms by which CDK modulates ETS activity  and more generally the functions of CDK  remain elusive  Here we demonstrate that CDK is a cyclin dependent kinase by identifying cyclin M as an activating cyclin  Cyclin M  an orphan cyclin  is the product of FAMA  whose mutations cause STAR syndrome  a human developmental anomaly whose features include toe syndactyly  telecanthus  and anogenital and renal malformations  We show that STAR syndrome associated cyclin M mutants are unable to interact with CDK  Cyclin M silencing phenocopies CDK silencing in increasing c Raf and in conferring tamoxifen resistance to breast cancer cells  CDK cyclin M phosphorylates ETS in vitro  and in cells it positively controls ETS degradation by the proteasome  ETS protein levels are increased in cells derived from a STAR patient  and this increase is attributable to decreased cyclin M levels  Altogether  our results reveal an additional regulatory mechanism for ETS  which plays key roles in cancer and development  They also shed light on the molecular mechanisms underlying STAR syndrome Cyclin dependent kinases  play a pivotal role in the control of a number of fundamental cellular processes   The human genome contains  genes encoding proteins that can be considered as members of the CDK family owing to their sequence similarity with bona fide CDKs  those known to be activated by cyclins   Although discovered almost  y ago   CDK remains one of the two CDKs without an identified cyclin partner  This knowledge gap has largely impeded the exploration of its biological functions  CDK can act as a positive cell cycle regulator in some cells  or as a tumor suppressor in others   CDK interacts with the ETS  transcription factor and inhibits its transcriptional activity through an unknown mechanism   CDK knockdown derepresses ETS  which increases the expression of the c Raf protein kinase  activates the MAPK pathway  and induces resistance of MCF cells to tamoxifen  Here  we deorphanize CDK by identifying cyclin M  the product of FAMA  as a binding partner  Mutations in this gene that predict absence or truncation of cyclin M are associated with STAR syndrome  whose features include toe syndactyly  telecanthus  and anogenital and renal malformations in heterozygous females   However  both the functions of cyclin M and the pathogenesis of STAR syndrome remain unknown  We show that a recombinant CDK cyclin M heterodimer is an active protein kinase that phosphorylates ETS in vitro  Cyclin M silencing phenocopies CDK silencing in increasing c Raf and phospho ERK expression levels and in inducing tamoxifen resistance in estrogen receptor   breast cancer cells  We show that CDK cyclin M positively controls ETS degradation by the proteasome  through the phosphorylation of two neighboring serines  Finally  we detect an increased ETS expression level in cells derived from a STAR patient  and we demonstrate that it is attributable to the decreased cyclin M expression level observed in these cells Previous SectionNext SectionResultsA yeast two hybrid  screen unveiled an interaction signal between CDK and a mouse protein whose C terminal half presents a strong sequence homology with the human FAMA gene product   We thus performed YH mating assays to determine whether human CDK interacts with human cyclin M   The longest CDK isoform  expressed as a bait protein produced a strong interaction phenotype with full length cyclin M  but no detectable phenotype with cyclin D  p   and Cdi   which are known binding partners of other CDKs   CDK and CDK also produced YH signals with cyclin M  albeit notably weaker than that observed with CDK   An interaction phenotype was also observed between full length cyclin M and CDK proteins expressed as bait and prey  respectively   We then tested different isoforms of CDK and cyclin M originating from alternative gene splicing  and two truncated cyclin M proteins corresponding to the hypothetical products of two mutated FAMA genes found in STAR syndrome patients   None of these shorter isoforms produced interaction phenotypes  Fig   In a new window Download PPTFig   CDK and cyclin M form an interaction complex   Schematic representation of the different protein isoforms analyzed by YH assays  Amino acid numbers are indicated  Black boxes indicate internal deletions  The red box indicates a differing amino acid sequence compared with CDK P   YH assay between a set of CDK proteins expressed as baits  and CDK interacting proteins expressed as preys   pEG and pJG  are the empty bait and prey plasmids expressing LexA and B  respectively  lacZ was used as a reporter gene  and blue yeast are indicative of a YH interaction phenotype   YH assay between the different CDK and cyclin M isoforms  The amino terminal region of ETS  known to interact with CDK   was also assayed   Western blot analysis of Myc CDK  and CycM V His expression levels in transfected HEK cells   Western blot analysis of Myc CDK  immunoprecipitates obtained using the anti Myc antibody   Inputs  correspond to  μg total lysates obtained from HEK cells coexpressing Myc CDK  and CycM V His   Western blot analysis of immunoprecipitates obtained using the anti CDK antibody or a control goat antibody  from human breast cancer MCF cells   Input  corresponds to  μg MCF total cell lysates  The lower band of the doublet observed on the upper panel comigrates with the exogenously expressed untagged CDK and thus corresponds to endogenous CDK  The upper band of the doublet corresponds to a nonspecific signal  as demonstrated by it insensitivity to either overexpression of CDK  or silencing of CDK   Another experiment with a longer gel migration is shown in Fig  SD Next we examined the ability of CDK and cyclin M to interact when expressed in human cells   We tested wild type CDK  and a kinase dead  mutant bearing a DA amino acid substitution that abolishes ATP binding   We expressed cyclin M V His and or Myc CDK  in a human embryonic kidney cell line   The expression level of cyclin M V His was significantly increased upon coexpression with Myc CDK  and  to a lesser extent  that of Myc CDK  was increased upon coexpression with cyclin M V His   We then immunoprecipitated Myc CDK proteins and detected the presence of cyclin M in the CDK  and  immunoprecipitates only when these proteins were coexpressed pair wise   We confirmed these observations by detecting the presence of Myc CDK in cyclin M V His immunoprecipitates   These experiments confirmed the lack of robust interaction between the CDK P isoform and cyclin M   To detect the interaction between endogenous proteins  we performed immunoprecipitations on nontransfected MCF cells derived from a human breast cancer  CDK and cyclin M antibodies detected their cognate endogenous proteins by Western blotting  We readily detected cyclin M in immunoprecipitates obtained with the CDK antibody but not with a control antibody   These results confirm the physical interaction between CDK and cyclin M in human cells To unveil a hypothesized CDK cyclin M protein kinase activity  we produced GST CDK and StrepII cyclin M fusion proteins in insect cells  either individually or in combination  We observed that GST CDK and StrepII cyclin M copurified  thus confirming their interaction in yet another cellular model   We then performed in vitro kinase assays with purified proteins  using histone H as a generic substrate  Histone H phosphorylation was detected only from lysates of cells coexpressing GST CDK and StrepII cyclin M  No phosphorylation was detected when GST CDK or StrepII cyclin M were expressed alone  or when StrepII cyclin M was coexpressed with GST CDK   Next we investigated whether ETS  which is known to interact with CDK    is a phosphorylation substrate of CDK cyclin M  We detected strong phosphorylation of ETS by the GST CDK StrepII cyclin M purified heterodimer  whereas no phosphorylation was detected using GST CDK alone or GST CDK StrepII cyclin M heterodimer  Fig   In a new window Download PPTFig   CDK is a cyclin M dependent protein kinase   In vitro protein kinase assay on histone H  Lysates from insect cells expressing different proteins were purified on a glutathione Sepharose matrix to capture GST CDK fusion proteins alone  or in complex with STR CycM fusion protein  Purified protein expression levels were analyzed by Western blots   The kinase activity was determined by autoradiography of histone H  whose added amounts were visualized by Coomassie staining    Same as in A  using purified recombinant His ETS as a substrate CDK silencing has been shown to increase ETS driven c RAF transcription and to activate the MAPK pathway   We investigated whether cyclin M is also involved in this regulatory pathway  To aim at a highly specific silencing  we used siRNA pools  at low final concentration   Both CDK and cyclin M siRNA pools silenced the expression of their cognate targets  and  interestingly  the cyclin M siRNA pool also caused a marked decrease in CDK protein level   These results  and those shown in Fig  D  suggest that cyclin M binding stabilizes CDK  Cyclin M silencing induced an increase in c Raf protein and mRNA levels  and in phosphorylated ERK and ERK protein levels   similarly to CDK silencing  As expected from these effects   CDK and cyclin M silencing both decreased the sensitivity of ER  MCF cells to tamoxifen  to a similar extent  The combined silencing of both genes did not result in a higher resistance to the drug   Altogether  these observations demonstrate a functional interaction between cyclin M and CDK  which negatively controls ETS Fig   In a new window Download PPTFig   Cyclin M silencing up regulates c Raf expression   Western blot analysis of endogenous CDK and cyclin M expression levels in MCF cells  in response to siRNA mediated gene silencing   Western blot analysis of endogenous c Raf expression levels in MCF cells  in response to CDK or cyclin M silencing  A quantification is shown in Fig  SA   Quantitative RT PCR analysis of CDK  cyclin M  and c Raf mRNA levels  in response to CDK  or cyclin M  silencing    P         P     We then wished to explore the mechanism by which CDK cyclin M controls ETS  ETS is a short lived protein degraded by the proteasome   A straightforward hypothesis is that CDK cyclin M positively controls ETS degradation  We thus examined the impact of CDK or cyclin M silencing on ETS expression levels  The silencing of CDK and that of cyclin M caused an increase in the expression levels of an exogenously expressed Flag ETS protein   as well as of the endogenous ETS protein   This increase is not attributable to increased ETS mRNA levels  which marginally fluctuated in response to CDK or cyclin M silencing   We then examined the expression levels of the Flag tagged ETS protein when expressed alone or in combination with Myc CDK or  CDK  with or without cyclin M V His  Flag ETS was readily detected when expressed alone or  to a lesser extent  when coexpressed with CDK  However  its expression level was dramatically decreased when coexpressed with CDK alone  or with CDK and cyclin M   These observations suggest that endogenous cyclin M levels are in excess compared with those of CDK in MCF cells  and they show that the major decrease in ETS levels observed upon CDK coexpression involves CDK kinase activity  Treatment of cells coexpressing Flag ETS  CDK  and cyclin M with the proteasome inhibitor MG largely rescued Flag ETS expression levels  Fig   In a new window Download PPTFig   CDK cyclin M controls ETS stability in human cancer derived cells   Western blot analysis of endogenous ETS expression levels in MCF cells  in response to siRNA mediated CDK and or cyclin M silencing  A quantification is shown in Fig  SB   Western blot analysis of exogenously expressed Flag ETS protein levels in MCF cells cotransfected with empty vectors or coexpressing Myc CDK   or Myc CDK CycM V His  The latter cells were treated for  h with the MG proteasome inhibitor  Proper expression of CDK and cyclin M tagged proteins was verified by Western blot analysis   Western blot analysis of expression levels of exogenously expressed Flag ETS wild type or mutant proteins in MCF cells  in the absence of  or in response to  Myc CDK CycM V His expression  Quantifications are shown in Fig  S C and D A mass spectrometry analysis of recombinant ETS phosphorylated by CDK cyclin M in vitro revealed the existence of multiple phosphorylated residues  among which are two neighboring phospho serines  that may form a phosphodegron    To confirm this finding  we compared the phosphorylation level of recombinant ETSwt with that of ETSSASA protein  a mutant bearing alanine substitutions of these two serines  As expected from the existence of multiple phosphorylation sites  we detected a small but reproducible  significant decrease of phosphorylation level of ETSSASA compared with ETSwt   thus confirming that Ser Ser are phosphorylated by CDK cyclin M  To establish a direct link between ETS phosphorylation by CDK cyclin M and degradation  we examined the expression levels of Flag ETSSASA  In the absence of CDK cyclin M coexpression  it did not differ significantly from that of Flag ETS  This is contrary to that of Flag ETSDBM  bearing a deletion of the N terminal destruction  box that was previously shown to be involved in APC Cdh mediated degradation of ETS    However  contrary to Flag ETS wild type  the expression level of Flag ETSSASA remained insensitive to CDK cyclin M coexpression   Altogether  these results suggest that CDK cyclin M directly controls ETS degradation through the phosphorylation of these two serines Finally  we studied a lymphoblastoid cell line derived from a patient with STAR syndrome  bearing FAMA mutation c  G A  predicted to result in aberrant splicing   In accordance with incomplete skewing of X chromosome inactivation previously found in this patient  we detected a decreased expression level of cyclin M protein in the STAR cell line  compared with a control lymphoblastoid cell line  In line with our preceding observations  we detected an increased expression level of ETS protein in the STAR cell line compared with the control   We then examined by quantitative RT PCR the mRNA expression levels of the corresponding genes  The STAR cell line showed a decreased expression level of cyclin M mRNA but an expression level of ETS mRNA similar to that of the control cell line   To demonstrate that the increase in ETS protein expression is indeed a result of the decreased cyclin M expression observed in the STAR patient derived cell line  we expressed cyclin M V His in this cell line  This expression caused a decrease in ETS protein levels  Fig   In a new window Download PPTFig   Decreased cyclin M expression in STAR patient derived cells results in increased ETS protein level   Western blot analysis of cyclin M and ETS protein levels in a STAR patient derived lymphoblastoid cell line and in a control lymphoblastoid cell line  derived from a healthy individual  A quantification is shown in Fig  SA   Quantitative RT PCR analysis of cyclin M and ETS mRNA levels in the same cells     P       Western blot analysis of ETS protein levels in the STAR patient derived lymphoblastoid cell line transfected with an empty vector or a vector directing the expression of cyclin M V His  Another Western blot revealing endogenously and exogenously expressed cyclin M levels is shown in Fig  SB  A quantification of ETS protein levels is shown in Fig  SC Previous SectionNext SectionDiscussionIn this work  we unveil the interaction between CDK  the last orphan CDK discovered in the pregenomic era   and cyclin M  the only cyclin associated with a human genetic disease so far  and whose functions remain unknown   The closest paralogs of CDK within the CDK family are the CDK proteins  which interact with L type cyclins   Interestingly  the closest paralog of these cyclins within the cyclin family is cyclin M   The fact that none of the shorter CDK isoforms interact robustly with cyclin M suggests that alternative splicing of the CDK gene  plays an important role in regulating CDK functions The functional relevance of the interaction between CDK and cyclin M is supported by different observations  Both proteins seem to enhance each other s stability  as judged from their increased expression levels when their partner is exogenously coexpressed  and from the much reduced endogenous CDK expression level observed in response to cyclin M silencing   CDK is subject to ubiquitin mediated degradation   Our observations suggest that cyclin M protects CDK from such degradation and that it is the only cyclin partner of CDK  at least in MCF cells  They also suggest that cyclin M stability is enhanced upon binding to CDK  independently from its kinase activity  as seen for cyclin C and CDK   We uncover a cyclin M dependent CDK protein kinase activity in vitro  thus demonstrating that this protein  which was named a CDK on the sole basis of its amino acid sequence  is indeed a genuine cyclin dependent kinase  Our YH assays reveal that truncated cyclin M proteins corresponding to the hypothetical products of two STAR syndrome associated FAMA mutations do not produce an interaction phenotype with CDK  Hence  regardless of whether these mutated mRNAs undergo nonsense mediated decay  or give rise to truncated cyclin M proteins  females affected by the STAR syndrome must exhibit compromised CDK cyclin M kinase activity at least in some tissues and during specific developmental stages We show that ETS  a known interactor of CDK  is a phosphorylation substrate of CDK cyclin M in vitro and that CDK cyclin M kinase activity positively controls ETS degradation by the proteasome  This control seems to be exerted through a very fine mechanism  as judged from the sensitivity of ETS levels to partially decreased CDK and cyclin M levels  achieved in MCF cells and observed in STAR cells  respectively  These findings offer a straightforward explanation for the already reported up regulation of ETS driven transcription of c RAF in response to CDK silencing   We bring evidence that CDK cyclin M directly controls ETS degradation through the phosphorylation of two neighboring serines  which may form a noncanonical β TRCP phosphodegron    Because none of these two serines precede a proline  they do not conform to usual CDK phosphorylation sites  However  multiple so called transcriptional CDKs   have been shown to phosphorylate a variety of motifs in a non proline directed fashion  especially in the context of molecular docking with the substrate   Here  it can be hypothesized that the high affinity interaction between CDK and the Pointed domain of ETS   would allow docking mediated phosphorylation of atypical sites  The control of ETS degradation involves a number of players  including APC Cdh  and the cullin RING ligase CRL   The formal identification of the ubiquitin ligase involved in the CDK cyclin M pathway and the elucidation of its concerted action with the other ubiquitin ligases to regulate ETS degradation will require further studies Our results present a number of significant biological and medical implications  First  they shed light on the regulation of ETS  which plays an important role in development  and is frequently deregulated in many cancers   Second  our results contribute to the understanding of the molecular mechanisms causing tamoxifen resistance associated with reduced CDK expression levels  and they suggest that  like CDK   cyclin M could also be a predictive clinical marker of hormone therapy response of ERα positive breast cancer patients  Third  our findings offer an interesting hypothesis on the molecular mechanisms underlying STAR syndrome  Ets transgenic mice showing a less than twofold overexpression of Ets present severe cranial abnormalities   and those observed in STAR patients could thus be caused at least in part by increased ETS protein levels  Another expected consequence of enhanced ETS expression levels would be a decreased risk to develop certain types of cancers and an increased risk to develop others  Studies on various mouse models  have revealed that ETS dosage can repress or promote tumor growth and  hence  that ETS exerts noncell autonomous functions in cancer   Intringuingly  one of the very few STAR patients identified so far has been diagnosed with a nephroblastoma   Finally  our findings will facilitate the general exploration of the biological functions of CDK and  in particular  its role in the control of cell division  Previous studies have suggested either a positive role in cell cycle control  or a tumor suppressive activity in some cancers   The severe growth retardation exhibited by STAR patients strongly suggests that CDK cyclin M plays an important role in the control of cell proliferation Previous SectionNext SectionMaterials and MethodsCloning of CDK and cyclin M cDNAs  plasmid constructions  tamoxifen response analysis  quantitative RT PCR  mass spectrometry experiments  and antibody production are detailed in SI Materials and Methods Yeast Two Hybrid Interaction Assays  We performed yeast interaction mating assays as previously described  Mammalian Cell Cultures and Transfections  We grew human HEK and MCF cells in DMEM supplemented with    FBS   and we grew lymphoblastoid cells in RPMI  GlutaMAX supplemented with    FBS  We transfected HEK and MCF cells using Lipofectamine   for plasmids  Lipofectamine RNAiMAX  for siRNAs  and Jetprime  for plasmids siRNAs combinations according to the manufacturers  instructions  We transfected lymphoblastoid cells by electroporation   For ETS stability studies we treated MCF cells  h after transfection with  μM MG  for  h Coimmunoprecipitation and Western Blot Experiments  We collected cells by scraping in PBS  and lysed them by sonication in a lysis buffer containing  mM β glycerophosphate   mM p nitrophenylphosphate   mM  propanesulfonic acid     mM EGTA   mM MgCl   mM Na vanadate   mM NaF  mM phenylphosphate     Nonidet P   and a protease inhibitor mixture   We spun the lysates  min at     g at   C  collected the supernatants  and determined the protein content using a Bradford assay  We performed the immunoprecipitation experiments on  μg of total proteins  in lysis buffer  We precleared the lysates with  μL of protein A or G agarose beads  incubated  h   C on a rotating wheel  We added  μg of antibody to the supernatants  incubated  h   C on a rotating wheel  added  μL of protein A or G agarose beads  and incubated  h   C on a rotating wheel  We collected the beads by centrifugation  s at     g at   C and washed three times in a bead buffer containing  mM Tris    mM NaF   mM NaCl   mM EDTA   mM EGTA     Nonidet P   and a protease inhibitor coktail   We directly added sample buffer to the washed pellets  heat denatured the proteins  and ran the samples on   Bis Tris SDS PAGE  We transferred the proteins onto Hybond nitrocellulose membranes and processed the blots according to standard procedures  For Western blot experiments  we used the following primary antibodies  anti Myc   anti V   anti tubulin   anti CDK   anti CycM   anti Raf   anti ETS   anti Flag   and anti actin   We used HRP coupled anti goat   anti mouse  or anti rabbit  as secondary antibodies  We revealed the blots by enhanced chemiluminescence  Production and Purification of Recombinant Proteins GST CDK StrepII CycM  We generated recombinant bacmids in DHBac Escherichia coli and baculoviruses in Sf cells using the Bac to Bac system  as described by the provider   We infected Sf cells with GST CDK   producing viruses  or coinfected the cells with StrepII CycM producing viruses  and we collected the cells  h after infection  To purify GST fusion proteins  we spun  mL cells and resuspended the pellet in  mL lysis buffer  containing a protease inhibitor mixture   We lysed the cells by sonication  spun the lysate  min at     g  collected the soluble fraction  and added it to a  mL glutathione Sepharose matrix  We incubated  h at   C  washed four times with lysis buffer  one time with kinase buffer A   and finally resuspended the beads in  μL kinase buffer A containing    glycerol for storage His ETS  We transformed Origami DE  with the His ETS expression vector  We induced expression with   mM isopropyl β d  thiogalactopyranoside for  h at   C  To purify His ETS  we spun  mL cells and resuspended the pellet in  mL lysis buffer  containing a protease inhibitor mixture without EDTA   We lysed the cells at   bar using a cell disruptor and spun the lysate  min at     g  We collected the soluble fraction and added it to  μL Cobalt beads   After  h incubation at   C on a rotating wheel  we washed four times with lysis buffer  To elute  we incubated beads  min with elution buffer  containing the protease inhibitor mixture  spun  s at     g  and collected the eluted protein Protein Kinase Assays  We mixed glutathione Sepharose beads     μM BSA   mM DTT   μM ATP   μCi ATP    μM histone H  or  μM His ETS and added kinase buffer A  up to a total volume of  μL  We incubated the reactions  min at   C  added Laemli sample buffer  heat denatured the samples  and ran   Bis Tris SDS PAGE  We cut gel slices to detect GST CDK and StrepII CycM by Western blotting  We stained the gel slices containing the substrate with Coomassie   dried them  and detected the incorporated radioactivity by autoradiography  We identified four unrelated girls with anogenital and renal malformations  dysmorphic facial features  normal intellect and syndactyly of toes  A similar combination of features had been reported previously in a mother daughter pair   These authors noted clinical overlap with Townes Brocks syndrome but suggested that the phenotype represented a separate autosomal dominant entity   Here we define the cardinal features of this syndrome as a characteristic facial appearance with apparent telecanthus and broad tripartite nasal tip  variable syndactyly of toes    hypoplastic labia  anal atresia and urogenital malformations   We also observed a variety of other features    Figure   Clinical and molecular characterization of STAR syndrome   Figure    Clinical and molecular characterization of STAR syndrome   Facial appearances of cases     and toe syndactyly      or    in these cases illustrate recognizable features of STAR syndrome   Anal atresia and hypoplastic labia are not shown   X ray films of the feet of case  showing only four rays on the left and delta shaped th and th metatarsals on the right    Array CGH data  Log ratio represents copy number loss of six probes spanning between   and   kb  with one probe positioned within FAMA  The deletion does not remove parts of other functional genes   Schematic structure of FAMA and position of the mutations  FAMA has five coding exons   The cyclin domain  is encoded by exons    The horizontal arrow indicates the deletion extending   in case   which includes exons  and   whereas the horizontal line below exon  indicates the deletion found in case   which removes exon  and some   sequence  The pink horizontal bars above the boxes indicate the amplicons used for qPCR and sequencing   The mutation dupT  results in an immediate stop codon  and the  G A and  G A splice mutations in cases    and  are predicted to be deleterious because they alter the conserved splice donor and acceptor site of intron   respectively   Full size image   Table   Clinical features in STAR syndrome cases  Table    Clinical features in STAR syndrome cases  Full table  On the basis of the phenotypic overlap with Townes Brocks  Okihiro and Feingold syndromes  we analyzed SALL   SALL  and MYCN but found no mutations in any of these genes   Next  we carried out genome wide high resolution oligonucleotide array comparative genomic hybridization  analysis  of genomic DNA from the most severely affected individual  and identified a heterozygous deletion of     kb on Xq  which removed exons  and  of FAMA   Using real time PCR  we confirmed the deletion in the child and excluded it in her unaffected parents   Through CGH with a customized oligonucleotide array enriched in probes for Xq  followed by breakpoint cloning  we defined the exact deletion size as   bp   Fig  j and Supplementary Figs    online   The deletion removes the coding regions of exons  and  as well as intron     bp of intron   and   bp of   sequence  including the   UTR and the entire KRTP pseudogene   Paternity was proven using routine methods  We did not find deletions overlapping FAMA in the available copy number variation  databases   Subsequently  we carried out qPCR analysis of the three other affected individuals  and the mother daughter pair from the literature   In case   we detected a de novo heterozygous deletion of     kb overlapping exon    Using Xq targeted array CGH and breakpoint cloning  we identified a deletion of   bp   Fig  j and Supplementary Figs      which removed   bp of intron   all of exon   including the   UTR  and   bp of   sequence   We found heterozygous FAMA point mutations in the remaining cases   In case   we identified the mutation  G A  affecting the splice donor site of intron   In case   we identified the frameshift mutation dupT  which immediately results in a premature stop codon NXfsX  In cases  and   we detected the mutation  G A  which alters the splice acceptor site of intron   We validated the point mutations and deletions by independent rounds of PCR and sequencing or by qPCR  We confirmed paternity and de novo status of the point mutations and deletions in all sporadic cases  None of the mutations were seen in the DNA of  unaffected female controls  and no larger deletions involving FAMA were found in  unrelated array CGH investigations   By analyzing X chromosome inactivation   we found complete skewing of X inactivation in cases  and   and almost complete skewing in case   suggesting that cells carrying the mutation on the active X chromosome have a growth disadvantage during fetal development  Using RT PCR on RNA from lymphoblastoid cells of case    we did not find any aberrant splice products as additional evidence that the mutated allele is inactivated  Furthermore  FAMA is subjected to X inactivation  In cases  and   the parental origin of the deletions could not be determined  as a result of lack of informative SNPs  Case   the mother of case   gave birth to two boys  both clinically unaffected   We cannot exclude that the condition is lethal in males  No fetal losses were reported from any of the families   The function of FAMA is unknown  The gene consists of five coding exons  and the  bp coding region encodes a protein of  amino acids  GenBank lists a mRNA length of   bp for the reference sequence   Expression of the gene  was found in  of  adult tissues including kidney  colon  cervix and uterus  but not heart   Expression was also noted in  of  listed tumor tissues as well as in embryo and fetus  Genes homologous to FAMA  are found on the X chromosome in the chimpanzee and the dog  The zebrafish has a similar gene on chromosome   However  in the mouse and rat  there are no true homologs  These species have similar but intronless genes on chromosomes   and    most likely arising from a retrotransposon insertion event  On the murine X chromosome  the flanking genes Atpb and Dusp are conserved  but only remnants of the FAMA sequence can be detected   FAMA contains a cyclin box fold domain  a protein binding domain found in cyclins with a role in cell cycle and transcription control  No human phenotype resulting from a cyclin gene mutation has yet been reported  Homozygous knockout mice for Ccnd  are viable but small and have reduced lifespan  They also have dystrophic changes of the retina  likely as a result of decreased cell proliferation and degeneration of photoreceptor cells during embryogenesis     Cyclin D colocalizes with SALL in the nucleus  and both proteins cooperatively mediate transcriptional repression  As the phenotype of our cases overlaps considerably with that of Townes Brocks syndrome caused by SALL mutations  we carried out co immunoprecipitation to find out if SALL or SALL would interact with FAMA in a manner similar to that observed for SALL and cyclin D  We found that FAMA interacts with SALL but not with SALL   supporting the hypothesis that FAMA and SALL participate in the same developmental pathway   How do FAMA mutations lead to STAR syndrome  Growth retardation  and retinal abnormalities  are reminiscent of the reduced body size and retinal anomalies in cyclin D knockout mice    Therefore  a proliferation defect might be partly responsible for STAR syndrome  To address this question  we carried out a knockdown of FAMA mRNA followed by a proliferation assay  Transfection of HEK cells with three different FAMA specific RNAi oligonucleotides resulted in a significant reduction of both FAMA mRNA expression and proliferation of transfected cells   supporting the link between FAMA and cell proliferation   We found that loss of function mutations of FAMA result in a rather homogeneous clinical phenotype  The additional anomalies in case  are likely to result from an effect of the  kb deletion on expression of a neighboring gene  possibly ATPB or DUSP  However  we cannot exclude that the homogeneous phenotype results from an ascertainment bias and that FAMA mutations  including missense changes  could result in a broader spectrum of malformations  The genes causing the overlapping phenotypes of STAR syndrome and Townes Brocks syndrome seem to act in the same pathway  Of note  MYCN  a gene mutated in Feingold syndrome  is a direct regulator of cyclin D   thus  it is worth exploring whether the phenotypic similarities between Feingold and STAR syndrome might be explained by direct regulation of FAMA by MYCN   FAMA is located approximately   Mb centromeric to MECP on Xq  Duplications overlapping both MECP and FAMA have been described and are not associated with a clinical phenotype in females  but no deletions overlapping both MECP and FAMA have been observed to date  Although other genes between FAMA and MECP have been implicated in brain development  FAMA and MECP are the only genes in this region known to result in X linked dominant phenotypes  thus  deletion of both genes on the same allele might be lethal in both males and females',\n",
       "       ' Abstract Background  Non small cell lung cancer  is a heterogeneous group of disorders with a number of genetic and proteomic alterations  c CBL is an E ubiquitin ligase and adaptor molecule important in normal homeostasis and cancer  We determined the genetic variations of c CBL  relationship to receptor tyrosine kinases   and functionality in NSCLC   Methods and Findings  Using archival formalin fixed paraffin embedded  extracted genomic DNA  we show that c CBL mutations occur in somatic fashion for lung cancers  c CBL mutations were not mutually exclusive of MET or EGFR mutations  however they were independent of p and KRAS mutations  In normal tumor pairwise analysis  there was significant loss of heterozygosity  for the c CBL locus  and none of these samples revealed any mutation in the remaining copy of c CBL  The c CBL LOH also positively correlated with EGFR and MET mutations observed in the same samples  Using select c CBL somatic mutations such as SN HY  QE and W   transfected in NSCLC cell lines  there was increased cell viability and cell motility   Conclusions  Taking the overall mutation rate of c CBL to be a combination as somatic missense mutation and LOH  it is clear that c CBL is highly mutated in lung cancers and may play an essential role in lung tumorigenesis and metastasis   Go to  Introduction In the US alone  each year approximately   people are diagnosed with lung cancers  out of which more than   of them succumb to the disease   This number is roughly equivalent to the combined mortality rates of cancers of the breast  prostate  colon  liver  kidney and melanoma   In addition the prognosis is usually poor and the five year survival rate is less than    There are also significant ethnic differences for lung cancer  and the outcome is worse for blacks compared to whites  Gender differences are also striking with women having significantly better prognosis as compared to men  There are a number of genetic alterations that can occur in lung cancer  As an example  in NSCLC  mutations in KRAS  p  EGFR and MET have been identified  Many of these pathways  especially Receptor Tyrosine Kinases  are controlled by c CBL   CBL  is a mammalian gene located on human chromosome q   and is involved in cell signaling and protein ubiquitination   CBL proteins belong to the RING finger class of ubiquitin ligases  and there are three homologues c CBL  CBL b  CBL    The c CBL and CBL b genes are ubiquitously expressed with the highest levels in hematopoietic tissues   c CBL consists of four regions encoding for functionally distinct protein domains  the N terminal tyrosine kinase binding  domain  the linker region  the catalytic RING finger domain  the proline rich region and the c terminal ubiquitin associated  domain that also overlaps with a leucine zipper  domain   Both TKB and RING finger domains are essential for ligand induced ubiquitination of RTKs         The RING finger domain is required for the recruitment of E ubiquitin conjugating enzymes  The TKB domain includes four helix bundle   a calcium biding EF hand  and a modified SH domain  which binds to phosphotyrosine residues         In addition  the proline rich region of c CBL can associate with the SH domain of Grb  which can indirectly recruit c CBL to RTKs via the GRB adaptor protein        c CBL also binds to EGFR and acts as the E that targets EGFR for ubiquitination and degradation  Furthermore  CBL desensitizes EGF signaling and opposes cellular proliferation induced by EGF   EGF activation also appears to activate the tyrosine kinase SRC  which phosphorylates c CBL and in turn activates the ubiquitination and degradation of EGFR       A recent study shows that defective endocytosis of EGFR is characterized by a deletion mutant and the point mutation LR  whereby its association with c CBL and subsequent ubiquitination are impaired   Recently  the first human c CBL mutations were reported in acute myeloid leukemia  patients   The mutation RQ inhibits FMS like tyrosine kinase   internalization and ubiquitination    Not only can E activity be important in oncogenesis  c CBL has a dual but separate function as a signal transduction molecule  We have previously shown that c CBL is important in binding CRKL and BCR ABL in hematopoietic cells  Also  it can bind and modulate functions of cytoskeleton by binding to proteins like talin and paxillin  The TKB domain is important in binding to a number of molecules  and they then function in signal transduction   Given the critical role of CBL in normal homeostasis and cancer  we hypothesized that it might be mutated in lung cancers  In this study  we report novel c CBL somatic mutations SN HY  QE and W  in Caucasian  Taiwanese and African American lung cancer patients  respectively  Expressing these mutations in NSCLC cell lines lead to increased proliferation and cell motility  We show that c CBL mutations occur with or without MET or EGFR mutations but are mutually exclusive of a LOH at the c CBL locus  Additionally  c CBL LOH is associated with either MET or EGFR mutations  We thus hypothesize that c CBL mutations might contribute to the oncogenic potential of MET and EGFR in lung cancer   Go to  Methods Ethics Statement  Written consent on all research on human subjects has been obtained from the Institutional Review Board  University of Chicago and covers all research performed in the laboratory  The following is their contact information  Institutional Review Board  The University of Chicago  McGiffert Hall   S  Woodlawn Ave   nd floor  Chicago  IL   Written informed consents were received from all patients whose tissue samples were used for this study   Tissue Samples  Lung cancer tissue and paired adjacent normal lung tissues were obtained from  Caucasian   African Americans and  Taiwanese NSCLC patients who were recruited at the University of Chicago Hospital   and Taipei Veterans General Hospital of Taiwan  after obtaining appropriate Institutional Review Board permission and informed consent from the patients  Out of  samples   were men   were women and  were unknown with age at diagnosis ranging from  to  years  In terms of tumor types   were adenocarcinoma   were squamous cell carcinoma and  were large cell carcinoma   were stage I   were stage II   were stage III  and  were stage IV    Cell Culture  Human non small cell lung carcinoma cells A and H were maintained in DMEM and RPMI   respectively  Human embryonic kidney T cells were cultured in DMEM  Media were supplemented with   fetal bovine serum   units ml of penicillin  and  µg ml of streptomycin   Cells were cultured at  C in a humidified incubator containing   CO   c CBL Gene Mutational Analysis  Exons  to  of c CBL gene were individually amplified by polymerase chain reaction   Primers are listed in Table S  PCR conditions were  cycle of  C for  minutes   cycles of  C for  seconds   C for  seconds and  C for  minutes  and one cycle of  C for  minutes  PCR products were treated with ExoSAP IT  and sequenced by Big Dye Terminator Chemistry   Sequencing was performed on the forward coding strand with confirmation of c CBL alterations performed by sequencing the reverse strand as well  Chromatograms were analyzed for mutations using Mutation Surveyor v     Plasmid Constructs and Site Directed Mutagenesis  The wild type c CBL cDNA insert was subcloned into the pAlterMax expression vector using XhoI and SalI restriction enzyme sites   Using this parental plasmid pAlterMax c CBL  the TKB domain double mutation   the point mutation   and the C terminal point mutation W  of c CBL were created using the following primers    GCTGGCGCTAAAGAATAACCCACCTTATATCTTAGAC   and   CTACCAGATACCTACCAGTATCTCCGTACTATCTTGTC   for the double mutation SN HY    CTTTACCCGACTCTTTGAGCCCTGGTCCTCTTTGC   for QE  and   CAGCTCCTCCTTTGGCTGATTGTCTCTGGATGGTGATC   for W  along with their complementary primers using the QuickChange Site Directed Mutagenesis XL kit  according to the manufacturer s instructions  The constructs were confirmed for the point mutations by standard DNA sequencing of both strands   Loss of Heterozygosity  Analysis  Five microsatellites on chromosome   were selected for analysis   Established microsatellite markers and respective primer sequences were selected from the GeneLoc database   Primers were custom designed and each forward primer was fluorescently labeled at the   end with FAM  PET  NED  or VIC   Primer annealing temperatures and duplex scores were evaluated with NIST Primer Tools   Primers were verified by performing PCR with control DNA  and resolving the products on agarose gels  Bands were visualized with an UV transilluminator  Genomic DNA was extracted from tumor samples and paired normal lung tissue  Primers were grouped into multiplex combinations shown in Table S  Marker DS served as an internal control to check for consistency in PCRs and of peaks from capillary electrophoresis  Multiplex PCRs were carried out in a volume of  µL that contained  µL genomic DNA     µM of each primer    µM dNTPs  X PCR buffer containing MgCl  and   U Taq DNA polymerase  PCR was performed on the ABI GeneAmp  PCR System under the following conditions   min at  C   cycles of  sec at  C   min at  C   min at  C  and  min at  C  The PCR products were separated by capillary electrophoresis on an ABI XL DNA Analyzer  Chromatograms were analyzed with Peak Scanner   and GeneMapper   software  for allelic alterations  The area of the peaks produced by the DNA PCR products was quantified for each allele  The ratio of the allelic areas was calculated for each tumor and paired normal DNA sample  When the qLOH  was    or    for c CBL and at least one other q marker in at least two separate experiments  the sample was considered as having an allelic imbalance and interpreted as LOH  Samples were evaluated in at least two separate experiments and samples showing prospective LOH at c CBL repeated a third time which included a new control marker at the BAX locus  on chromosome  to verify integrity of sample DNA   Transfection of c Cbl Constructs  The A cell line was transfected using the Fugene HD  reagent according to the manufacturer s instructions  Eight µg of plasmid DNA  containing either no insert   wild type c CBL  SN HY c CBL  QE c CBL or W  CBL was used for transfection in a  well culture plate  Cells were harvested  h after transfection and analyzed for expression   c CBL Knockdown  c CBL knockdown was performed using lentiviral transduction using MISSION lentiviral transduction particles  as per manufacturer s instructions  Briefly    H cells well were seeded in  well plates and infected the following day with c CBL lentiviral shRNA constructs  To generate stable c CBL knockdown cell lines  cells were selected for  days with  µg ml puromycin  c CBL levels were determined using whole cell lysates by immunoblotting with anti CBL antibody    Cell Viability Assay  Cells were transfected as described above in the transfection assay  Forty eight hours after transfection  viability of cells was assessed using Trypan Blue exclusion   Wound Healing Assay  A cells were seeded in  well plates and cultured for  h until   confluent  The medium was then changed and the cells were transfected as described in the transfection assay  Twelve hours after transfection  a straight scratch was made across the cell layer using a  ml pipette tip  The cells were then gently washed with   PBS to remove cellular debris and the media was replaced  Photographs were taken of the wound region every  h until  h   Western Blot Analysis  Forty eight hours after transfection  cells were collected and washed twice in X PBS  then lysed in ice cold lysis buffer  for  minutes  The lysate was centrifuged at   rpm for  minutes at  C  and protein content of the supernatant was measured  Total cell lysates  were separated by SDS PAGE electrophoresis and the gels transferred onto nitrocellulose membranes   Membranes were blocked with   non fat dry milk in phosphate buffered saline containing Tween    for  h at room temperature and incubated with the appropriate primary antibody at  C overnight  Membranes then were washed three times with PBST and probed with appropriate horseradish peroxidase  conjugated secondary antibody for  h at room temperature  The membranes were again washed three times in PBST and bands were visualized using Western blot chemiluminescence reagent  on a Chemidoc Gel documentation system   Antibodies were obtained from Santa Cruz Biotechnologies and used at the following dilutions    Flow Cytometry  Cell cycle analysis was carried out by flow cytometry  Approximately   cells were grown in media containing   FBS  Cells were harvested by trypsin EDTA treatment  washed with X PBS three times and fixed with ice cold   ethanol for  h  Cells were washed again with cold PBS and stained with a solution containing  µg mL propidium iodide   µg mL RNase A  and    Triton X  for  minutes in the dark  Cell cycle analysis was performed using a Guava PCA  flow cytometer    Ubiquitin Ligase Activity  T cells were maintained in culture in DMEM supplemented with   FBS and   penicillin  and streptomycin  were transfected with   µg EGFR pcDNA and  µg HA tagged c CBL constructs as indicated using calcium phosphate according to manufacturer s protocol   Twenty four hours post transfection  cells were starved overnight in DMEM supplemented with    FBS  and then treated with or without EGF  for  min  The cells were collected and washed two times in ice cold PBS containing   mM sodium orthovanadate then lysed in ice cold lysis buffer   Lysates were cleared of debris by centrifugation at   g for  min at  C  EGFR immunoprecipitations were performed on  µg of cleared lysate using  ng of rabbit anti EGFR and Protein A G Plus Sepharose overnight at  C  Precipitations were washed  times in lysis buffer before boiling in Laemmli buffer  Elutions were immunoblotted with anti ubiquitin and EGFR  Twenty micrograms of cleared lysate were immunoblotted for each of the c CBL constructs using anti HA   Statistical Analysis  Mutation rates between different groups were compared using Fisher s exact test  For continuous variables  group comparisons were performed using analysis of variance  followed by Sidak s adjustment for multiple comparisons  Experiments involving repeated measurements over time were analyzed using repeated measures ANOVA with the Greenhouse Geisser adjustment to the degrees of freedom  Analyses were conducted using STATA  software    Go to  Results c CBL Gene Mutations in Lung Cancer  To investigate the role of c CBL in lung cancer  we analyzed its genomic DNA in tumor and paired normal samples drawn from multiple ethnicities  The lung tumor samples represented Caucasians   African Americans   and Taiwanese  lung cancer patients  We designed  pairs of primers to sequence the coding region of c CBL gene that spans exons  to    We identified  unique somatic mutations in c CBL exons among  different patients  A variation LF  a known SNP  in exon  was also detected  Importantly  the eight novel non synonymous mutations were confirmed by sequencing both strands of c CBL genomic DNA obtained from lung tumor samples   Moreover  none of the  mutations were detected in the corresponding normal tissue  indicating that these were somatic mutations  Four synonymous single nucleotide variations  were also identified but were not used further in this study   Table  Table  c CBL mutation analysis in  lung cancer patient tumor tissues  Three of the  novel non synonymous mutations were located in the TKB  domain   one in the RING finger domain   one in the proline rich region   and three in the C terminal region  of the c CBL protein   In Figure B  we show model chromatograms of representative samples   Figure  Figure  c CBL mutations and LOH in non small cell lung cancer  q LOH of c CBL Gene  Paired lung tumor and normal lung tissue samples from Taiwanese patients  were investigated for LOH  Eight  showed LOH at the c CBL locus on chromosome  while  samples  revealed normal allelic contribution at the microsatellite markers    c CBL Mutations in Different Ethnic Groups  The c CBL double mutant SN HY was found in the same patient and the overall mutation rate for c CBL in lung tumors was      The frequency of c CBL mutation was highest in large cell carcinoma  followed by squamous carcinoma  and the least was observed in adenocarcinoma    although these rates were not statistically significant   Mutation rates were    among Caucasians      in African Americans   and     in the Taiwanese population  Additionally two Taiwanese patients with lung cancer  had the known SNP LF  Ethnic differences were not statistically significant  however the power to detect differences was low   Mutations in MET and EGFR Can Be Co Associated with c CBL Alterations  Since East Asians with lung cancer have a higher frequency of EGFR and MET mutations in lung tumors     we also determined mutations in EGFR and MET in the same Taiwanese cohort samples and compared the results with the observed c CBL alterations   In the  samples tested  we did not find any overlap between c CBL mutations and c CBL LOH   Of the three c CBL mutants   one of the samples had a MET mutation  and the other had an EGFR mutation   Among the  samples that had a LOH at the c CBL locus  had an additional mutation in MET  and  had an EGFR exon  deletion  Twenty six samples had neither c CBL mutation nor c CBL LOH   Among these  samples  had a MET mutation    had an EGFR mutation  and  had no other MET or EGFR mutation  Thus the rate of MET or EGFR mutations among patients with LOH at the c CBL locus  was similar to that seen in patients without c CBL mutation or LOH    These  patients with no identifiable mutation in c CBL  MET or EGFR represented    of the  patients analyzed in the Taiwanese patient cohort  Conversely     Taiwanese lung cancer patients have an identifiable mutation in either c CBL  MET or EGFR or a combination of the three genes   Additionally  we determined p and KRAS mutations in these Taiwanese cohorts  Two p and  KRAS mutation were detected  The single KRAS mutation overlapped with one p mutation  This patient also had the EGFR exon  deletion but had no c CBL mutation  The other p mutation sample had a c CBL LOH with concurrent MET NS mutation  Thus  in the Taiwanese samples analyzed  p KRAS mutations and c CBL mutations were mutually exclusive    Figure  Figure  c CBL mutations and relationship to MET and EGFR mutations in lung cancer  Cellular Functions of c CBL Alterations in the Context of Lung Tumorigenesis  A  E activity is intact in the mutant c CBL proteins To investigate whether the different c CBL mutations affect the E activity  EGFR was chosen as a model substrate for c CBL E function  All of the c CBL mutants tested enhanced ubiquitination of the activated EGFR similar to the wild type c CBL protein  This result demonstrates that the catalytic activity of the c CBL mutants is not impaired when EGFR was the substrate     Figure  Figure  Ubiquitination  viability  expression and cell cycle analysis of various c CBL mutants  B  Effect on lung cancer cell viability The effect of a representative c CBL mutant from each of the three ethnic backgrounds on lung cancer cell viability in cell lines was determined  SN HY double mutation  QE  and W  were identified in lung tumor samples obtained from a Caucasian  a Taiwanese and an African American  respectively  As described in methods  the c CBL wild type  and the above three mutants were expressed after cloning them into pAlterMax vector in A cells  These cells express relatively low basal levels of endogenous c CBL   Transfection efficiency was comparable between different groups and the number of cells transfected with c CBL wild type construct was about   compared to control cells that were transfected with the empty vector  Cells transfected with SN HY  QE and W  c CBL mutant constructs resulted in increased number of viable cells that was        and    higher respectively  relative to the empty vector control transfected cells and significantly different from the wild type construct    Relative levels of c CBL protein in whole cell lysates prepared from samples obtained from a parallel experiment was determined  The c CBL protein levels in samples representing untransfected and empty vector transfected cells were comparable and those representing the c CBL WT and the three c CBL mutants were comparable    C  Effect on cell cycle To investigate if the increases in cell viability in different c CBL mutants are due to increased cellular proliferation  a cell cycle analysis was performed  A cells were transfected with the c CBL WT or the three different mutants  SN HY  QE and W   The empty vector transfectant was used as a control  Forty eight hours after transfection cell cycle analysis was performed as described in materials and methods  There was no significant change in the subG  G or the S phase of the cell cycle among the different mutants compared to the WT construct   The G M phase of the cell cycle showed an increase in cell numbers for the three mutants  SN HY  QE and W   when compared to the WT but again the difference was not statistically significant     D  Effect on cell motility To investigate the effect of the expression of the above three c CBL mutants on cell migration  we carried out wound healing assay as described in materials and methods  The closing of the scratch or the wound was monitored at         and  h    In all the samples  that represented cells transfected with mutants  the wound gap was much smaller than that seen in the sample that represented cells transfected with c CBL WT   We also determined the rate of wound closure for all the five groups  At  h  wild type c CBL transfectants showed    open wound while the SN HY  QE and W  mutants showed        and    open wound respectively     Figure  Figure  c CBL mutations affect wound healing in A cells  E  c CBL knockdown increases cell viability It is hypothesized that the LOH seen in our samples could lead to decreased expression of c CBL  Thus we tested the effect of c CBL knockdown in lung cancer cells  Compared to A  H lung cancer cells express relatively high levels of endogenous c CBL   c CBL expression was knocked down using lentiviral construct that expressed c CBL specific shRNA and compared the results with those that were transduced with scrambled shRNA and the results are shown in Figure   We identified several clones that revealed varying degrees of c CBL knockdown showing different sets of c CBL lentiviral shRNA knockdown efficiency   Of all the clones tested  Clone  was chosen for further experiments  Equal amount of cells were seeded in a  well plate and the cell proliferation was measured at various times and the results are depicted in Figure B  As expected  number of cells increased in a time dependent fashion from  to   relative to scrambled shRNA as control in a span of  h    The cell cycle phases in H cells that were knocked down with c CBL shRNA were looked at and compared with the scrambled shRNA  There were no discernable differences between these two constructs in the different phases of the cell cycle    Figure  Figure  Knockdown of c CBL using an shRNA increases cell proliferation  Go to  Discussion Our results demonstrate that c CBL is somatically mutated  in lung cancers  and can significantly contribute to enhanced cell viability and motility  There was also a high prevalence of LOH with respect to c CBL in lung tumors that harbored MET or EGFR mutation   In the present study  we have demonstrated the occurrence of c CBL mutations in lung cancer patients  especially with different ancestral variations  Mutations in c CBL have been recently reported in juvenile myelomonocytic leukemia and myeloid malignancies  In the AML study  the mutation RQ located in the junction of the RING finger and the linker region inhibited FMS like tyrosine kinase   internalization and ubiquitination   thus contributing to the gain in function for the RTK  In addition  mutations such as HY  CR  and LP mapped to the RING finger domain and the linker region of c CBL that is required for its E activity           Additionally  homozygous mutations in the RING finger domain of the c CBL gene were described as a result of acquired Uniparental Disomy    It is important to note that our results indicate LOH at q locus and these are mutually exclusive from missense mutations of c CBL  The somatic mutations were all heterozygous  The mutations in AML led to abrogation of the E activity leading to prolonged RTK activation  In addition mutants located on the linker region surrounding the RING finger domain exhibited enhanced AKT signaling in response to cytokine stimulation   In addition  it was shown in NHT cells  that neither mutations in the RING finger nor the linker region causes transformation  however while certain mutations perturbs the ubiquitination  others affect receptor recycling and prolong kinase activity    We report here c CBL mutations that mapped not only to the RING finger domain  but also to the TKB domain  proline rich domain and the C terminal region  but none mapped to the linker region as reported in the AML studies described above           In addition   mutants that we detected were found in different ethnic backgrounds  For example  SN HY  QE  W  were detected in Caucasians  Taiwanese and African Americans  respectively  The results point out not only the difference between lung cancer and other cancers  but also genetic polymorphism among different races in the same cancer  Interestingly  there is a large disparity between African American and other ethnic populations with lung cancer   We have previously shown that there was a low frequency of EGFR and MET mutation in African Americans as compared with Taiwanese and Caucasians   In this study  the number of African American samples analyzed was relatively fewer and we found  mutations that are unique to this ethnicity  It would now behoove us to further study the genetic alterations that can occur and determine the targeted therapeutics for African Americans  Our results provide evidence of the importance of c CBL in tumorigenesis and potential signaling  Our prediction  based on the AML data  would be that the VI RING finger domain mutation would affect the E activity  Also  it will be important to determine the binding partners for c CBL in the TKB domain and proline rich domain mutations  It has previously been shown that the TKB domain can bind to growth factor receptors and it will be important to determine the cross binding of these mutants to MET and or EGFR  It would also be important in the future to look at fluorescence in situ hybridization copy number changes in c CBL in lung cancer   c CBL plays an important role in down regulating RTK mediated signaling through K poly ubiquitination and subsequent downregulation of RTKs followed by lysosomal degredation   Mono ubiquitination or ubiquitinated with K linked chains of substrates by c CBL may lead to enhancement of biological and biochemical functions   The mutations that we analyzed in our studies all point out to the fact that the E activity of c CBL on EGFR is intact  the EGFR levels in the various mutants remain same   Multiple kinases  both RTKs and non RTKs could be acted upon by c CBL  including ERBs  PDGFR  FMS  MET  c Kit  VEGFR  FLT   RON  FGFR  IR  as well as SYK  FYN  LCK  FGR  LYN and c ABL   In lung cancers the relevant substrates of c CBL in terms of degradation or signal transduction are yet to be identified   The observation that c CBL somatic mutations  especially SN HY  QE and W  showed increased cell viability and cell motility that is in agreement with the physiological role for D cbl in the regulation of apoptosis and differentiation identified in Drosophila is very significant   It has been previously shown that activating c CBL mutation downregulates EGFR signaling and decreases cellular proliferation and migration in breast cancer cell lines   Although the role of c CBL in the negative regulation of RTKs is well substantiated  thereby suggesting that it is a natural tumor suppressor  studies in cancer cells have revealed both tumor suppressor and tumor promoting activities depending on the type of c CBL mutation and the number of alleles at the c CBL locus   In agreement with the above  the three c CBL mutants described here appear to have tumor growth and metastasis promoting properties  Although these mutants are outside of the RING finger or the linker region of c CBL  their downstream effects are significant so as to cause increased proliferation and migration  but the substrate affected by these mutations are not known yet  This raises the possibility that some of the cellular functions of c CBL are independent of its ubiquitin ligase activity  an area that we are currently investigating  The oncogenic nature of RTKs  addiction of cancers to growth signals and given the clustering of c CBL  EGFR and MET mutations  it is possible that the transforming effect of c CBL mutations is most likely a combinatorial effect of the three  We also show that LOH for c CBL was found in a significant number of samples that harbored MET or EGFR mutations  The fact that about   of lung tumor samples are likely to have c CBL mutations and an additional   are likely to harbor c CBL related LOH  makes c CBL a highly mutated molecule in lung cancer  Since LOH alone is not enough to cause a transforming event       associated mutation in the MET or EGFR locus or yet another RTK discussed above may play a role in carcinogenesis  We predict that this LOH in c CBL results in haploinsufficiency that downplays RTK ubiquitination leading to hyperactivity of the RTKs  However  whether this is sufficient cause for tumorigenesis remains to be determined  Consistent with our hypothesis is the fact that c cbl    mice have increased kinase activity in lymphocytes  but is not sufficient for tumor formation       c CBL LOH could also lead to increased expression of c CBL from the other allele to compensate for a loss of an allele  Alternately  there could be some form of synergy working with reduced c CBL levels and mutated receptors that exacerbate the phenotype of each alone   Previous studies from our lab and others have shown that East Asians with lung cancers have relatively high frequencies of gain of function of mutations in RTKs such as EGFR and MET   In a cohort of Japanese patients an activating MET mutation was identified in the splice region that deletes the juxtamembrane domain that is involved E activity of c CBL   This study also found that activation of MET is mutually exclusive of EGFR  KRAS and HER gene mutations   We failed to detect such mutations in significant numbers in lung tumor samples obtained from African Americans  and Caucasian  patients  One MET mutation was identified in each of the groups whereas  and  EGFR mutations were identified in the African American and Caucasian cohorts respectively  EGFR mutations have earlier been identified as one of the key mutations affecting lung adenocarcinoma patients in a comprehensive study of  patients   Our study encompasses different histologies of NSCLC  However  the published series did not find any mutations in c CBL or MET unlike our study that encompassed different subtypes of NSCLC  It is important to note  we have recently shown that MET mutations in lung cancer are in majority germline   We have reported earlier c CBL mutations in a small cohort of Taiwanese lung cancer samples   In our efforts to understand the ethnic differences in the lung oncogenome  we also looked at PAX transcription factors such as PAX and PAX that are highly expressed in lung cancers  however there was no preferential expression or mutations of the above genes in lung tumor samples of African Americans  In this study  we show relatively high frequency of c CBL mutations in lung cancers  especially in the large cell type among Caucasians and particularly among African Americans  We therefore propose c CBL as an efficacious target for lung cancers in African Americans that needs to be further substantiated  This is all the more important because the prognosis for African Americans with lung cancer  especially for men is much poorer compared to their Caucasian counterparts    In conclusion  the results presented in this study demonstrate that c CBL is frequently mutated or even lost in lung cancers  Our results support a role for c CBL mutants that are independent of its ubiquitination activity  Given the relatively high mutation rates in c CBL as well as RTKs such as MET and EGFR  it is likely that their combined effect could be synergistic in promoting tumorigenesis',\n",
       "       ' Abstract Background  Non small cell lung cancer  is a heterogeneous group of disorders with a number of genetic and proteomic alterations  c CBL is an E ubiquitin ligase and adaptor molecule important in normal homeostasis and cancer  We determined the genetic variations of c CBL  relationship to receptor tyrosine kinases   and functionality in NSCLC   Methods and Findings  Using archival formalin fixed paraffin embedded  extracted genomic DNA  we show that c CBL mutations occur in somatic fashion for lung cancers  c CBL mutations were not mutually exclusive of MET or EGFR mutations  however they were independent of p and KRAS mutations  In normal tumor pairwise analysis  there was significant loss of heterozygosity  for the c CBL locus  and none of these samples revealed any mutation in the remaining copy of c CBL  The c CBL LOH also positively correlated with EGFR and MET mutations observed in the same samples  Using select c CBL somatic mutations such as SN HY  QE and W   transfected in NSCLC cell lines  there was increased cell viability and cell motility   Conclusions  Taking the overall mutation rate of c CBL to be a combination as somatic missense mutation and LOH  it is clear that c CBL is highly mutated in lung cancers and may play an essential role in lung tumorigenesis and metastasis   Go to  Introduction In the US alone  each year approximately   people are diagnosed with lung cancers  out of which more than   of them succumb to the disease   This number is roughly equivalent to the combined mortality rates of cancers of the breast  prostate  colon  liver  kidney and melanoma   In addition the prognosis is usually poor and the five year survival rate is less than    There are also significant ethnic differences for lung cancer  and the outcome is worse for blacks compared to whites  Gender differences are also striking with women having significantly better prognosis as compared to men  There are a number of genetic alterations that can occur in lung cancer  As an example  in NSCLC  mutations in KRAS  p  EGFR and MET have been identified  Many of these pathways  especially Receptor Tyrosine Kinases  are controlled by c CBL   CBL  is a mammalian gene located on human chromosome q   and is involved in cell signaling and protein ubiquitination   CBL proteins belong to the RING finger class of ubiquitin ligases  and there are three homologues c CBL  CBL b  CBL    The c CBL and CBL b genes are ubiquitously expressed with the highest levels in hematopoietic tissues   c CBL consists of four regions encoding for functionally distinct protein domains  the N terminal tyrosine kinase binding  domain  the linker region  the catalytic RING finger domain  the proline rich region and the c terminal ubiquitin associated  domain that also overlaps with a leucine zipper  domain   Both TKB and RING finger domains are essential for ligand induced ubiquitination of RTKs         The RING finger domain is required for the recruitment of E ubiquitin conjugating enzymes  The TKB domain includes four helix bundle   a calcium biding EF hand  and a modified SH domain  which binds to phosphotyrosine residues         In addition  the proline rich region of c CBL can associate with the SH domain of Grb  which can indirectly recruit c CBL to RTKs via the GRB adaptor protein        c CBL also binds to EGFR and acts as the E that targets EGFR for ubiquitination and degradation  Furthermore  CBL desensitizes EGF signaling and opposes cellular proliferation induced by EGF   EGF activation also appears to activate the tyrosine kinase SRC  which phosphorylates c CBL and in turn activates the ubiquitination and degradation of EGFR       A recent study shows that defective endocytosis of EGFR is characterized by a deletion mutant and the point mutation LR  whereby its association with c CBL and subsequent ubiquitination are impaired   Recently  the first human c CBL mutations were reported in acute myeloid leukemia  patients   The mutation RQ inhibits FMS like tyrosine kinase   internalization and ubiquitination    Not only can E activity be important in oncogenesis  c CBL has a dual but separate function as a signal transduction molecule  We have previously shown that c CBL is important in binding CRKL and BCR ABL in hematopoietic cells  Also  it can bind and modulate functions of cytoskeleton by binding to proteins like talin and paxillin  The TKB domain is important in binding to a number of molecules  and they then function in signal transduction   Given the critical role of CBL in normal homeostasis and cancer  we hypothesized that it might be mutated in lung cancers  In this study  we report novel c CBL somatic mutations SN HY  QE and W  in Caucasian  Taiwanese and African American lung cancer patients  respectively  Expressing these mutations in NSCLC cell lines lead to increased proliferation and cell motility  We show that c CBL mutations occur with or without MET or EGFR mutations but are mutually exclusive of a LOH at the c CBL locus  Additionally  c CBL LOH is associated with either MET or EGFR mutations  We thus hypothesize that c CBL mutations might contribute to the oncogenic potential of MET and EGFR in lung cancer   Go to  Methods Ethics Statement  Written consent on all research on human subjects has been obtained from the Institutional Review Board  University of Chicago and covers all research performed in the laboratory  The following is their contact information  Institutional Review Board  The University of Chicago  McGiffert Hall   S  Woodlawn Ave   nd floor  Chicago  IL   Written informed consents were received from all patients whose tissue samples were used for this study   Tissue Samples  Lung cancer tissue and paired adjacent normal lung tissues were obtained from  Caucasian   African Americans and  Taiwanese NSCLC patients who were recruited at the University of Chicago Hospital   and Taipei Veterans General Hospital of Taiwan  after obtaining appropriate Institutional Review Board permission and informed consent from the patients  Out of  samples   were men   were women and  were unknown with age at diagnosis ranging from  to  years  In terms of tumor types   were adenocarcinoma   were squamous cell carcinoma and  were large cell carcinoma   were stage I   were stage II   were stage III  and  were stage IV    Cell Culture  Human non small cell lung carcinoma cells A and H were maintained in DMEM and RPMI   respectively  Human embryonic kidney T cells were cultured in DMEM  Media were supplemented with   fetal bovine serum   units ml of penicillin  and  µg ml of streptomycin   Cells were cultured at  C in a humidified incubator containing   CO   c CBL Gene Mutational Analysis  Exons  to  of c CBL gene were individually amplified by polymerase chain reaction   Primers are listed in Table S  PCR conditions were  cycle of  C for  minutes   cycles of  C for  seconds   C for  seconds and  C for  minutes  and one cycle of  C for  minutes  PCR products were treated with ExoSAP IT  and sequenced by Big Dye Terminator Chemistry   Sequencing was performed on the forward coding strand with confirmation of c CBL alterations performed by sequencing the reverse strand as well  Chromatograms were analyzed for mutations using Mutation Surveyor v     Plasmid Constructs and Site Directed Mutagenesis  The wild type c CBL cDNA insert was subcloned into the pAlterMax expression vector using XhoI and SalI restriction enzyme sites   Using this parental plasmid pAlterMax c CBL  the TKB domain double mutation   the point mutation   and the C terminal point mutation W  of c CBL were created using the following primers    GCTGGCGCTAAAGAATAACCCACCTTATATCTTAGAC   and   CTACCAGATACCTACCAGTATCTCCGTACTATCTTGTC   for the double mutation SN HY    CTTTACCCGACTCTTTGAGCCCTGGTCCTCTTTGC   for QE  and   CAGCTCCTCCTTTGGCTGATTGTCTCTGGATGGTGATC   for W  along with their complementary primers using the QuickChange Site Directed Mutagenesis XL kit  according to the manufacturer s instructions  The constructs were confirmed for the point mutations by standard DNA sequencing of both strands   Loss of Heterozygosity  Analysis  Five microsatellites on chromosome   were selected for analysis   Established microsatellite markers and respective primer sequences were selected from the GeneLoc database   Primers were custom designed and each forward primer was fluorescently labeled at the   end with FAM  PET  NED  or VIC   Primer annealing temperatures and duplex scores were evaluated with NIST Primer Tools   Primers were verified by performing PCR with control DNA  and resolving the products on agarose gels  Bands were visualized with an UV transilluminator  Genomic DNA was extracted from tumor samples and paired normal lung tissue  Primers were grouped into multiplex combinations shown in Table S  Marker DS served as an internal control to check for consistency in PCRs and of peaks from capillary electrophoresis  Multiplex PCRs were carried out in a volume of  µL that contained  µL genomic DNA     µM of each primer    µM dNTPs  X PCR buffer containing MgCl  and   U Taq DNA polymerase  PCR was performed on the ABI GeneAmp  PCR System under the following conditions   min at  C   cycles of  sec at  C   min at  C   min at  C  and  min at  C  The PCR products were separated by capillary electrophoresis on an ABI XL DNA Analyzer  Chromatograms were analyzed with Peak Scanner   and GeneMapper   software  for allelic alterations  The area of the peaks produced by the DNA PCR products was quantified for each allele  The ratio of the allelic areas was calculated for each tumor and paired normal DNA sample  When the qLOH  was    or    for c CBL and at least one other q marker in at least two separate experiments  the sample was considered as having an allelic imbalance and interpreted as LOH  Samples were evaluated in at least two separate experiments and samples showing prospective LOH at c CBL repeated a third time which included a new control marker at the BAX locus  on chromosome  to verify integrity of sample DNA   Transfection of c Cbl Constructs  The A cell line was transfected using the Fugene HD  reagent according to the manufacturer s instructions  Eight µg of plasmid DNA  containing either no insert   wild type c CBL  SN HY c CBL  QE c CBL or W  CBL was used for transfection in a  well culture plate  Cells were harvested  h after transfection and analyzed for expression   c CBL Knockdown  c CBL knockdown was performed using lentiviral transduction using MISSION lentiviral transduction particles  as per manufacturer s instructions  Briefly    H cells well were seeded in  well plates and infected the following day with c CBL lentiviral shRNA constructs  To generate stable c CBL knockdown cell lines  cells were selected for  days with  µg ml puromycin  c CBL levels were determined using whole cell lysates by immunoblotting with anti CBL antibody    Cell Viability Assay  Cells were transfected as described above in the transfection assay  Forty eight hours after transfection  viability of cells was assessed using Trypan Blue exclusion   Wound Healing Assay  A cells were seeded in  well plates and cultured for  h until   confluent  The medium was then changed and the cells were transfected as described in the transfection assay  Twelve hours after transfection  a straight scratch was made across the cell layer using a  ml pipette tip  The cells were then gently washed with   PBS to remove cellular debris and the media was replaced  Photographs were taken of the wound region every  h until  h   Western Blot Analysis  Forty eight hours after transfection  cells were collected and washed twice in X PBS  then lysed in ice cold lysis buffer  for  minutes  The lysate was centrifuged at   rpm for  minutes at  C  and protein content of the supernatant was measured  Total cell lysates  were separated by SDS PAGE electrophoresis and the gels transferred onto nitrocellulose membranes   Membranes were blocked with   non fat dry milk in phosphate buffered saline containing Tween    for  h at room temperature and incubated with the appropriate primary antibody at  C overnight  Membranes then were washed three times with PBST and probed with appropriate horseradish peroxidase  conjugated secondary antibody for  h at room temperature  The membranes were again washed three times in PBST and bands were visualized using Western blot chemiluminescence reagent  on a Chemidoc Gel documentation system   Antibodies were obtained from Santa Cruz Biotechnologies and used at the following dilutions    Flow Cytometry  Cell cycle analysis was carried out by flow cytometry  Approximately   cells were grown in media containing   FBS  Cells were harvested by trypsin EDTA treatment  washed with X PBS three times and fixed with ice cold   ethanol for  h  Cells were washed again with cold PBS and stained with a solution containing  µg mL propidium iodide   µg mL RNase A  and    Triton X  for  minutes in the dark  Cell cycle analysis was performed using a Guava PCA  flow cytometer    Ubiquitin Ligase Activity  T cells were maintained in culture in DMEM supplemented with   FBS and   penicillin  and streptomycin  were transfected with   µg EGFR pcDNA and  µg HA tagged c CBL constructs as indicated using calcium phosphate according to manufacturer s protocol   Twenty four hours post transfection  cells were starved overnight in DMEM supplemented with    FBS  and then treated with or without EGF  for  min  The cells were collected and washed two times in ice cold PBS containing   mM sodium orthovanadate then lysed in ice cold lysis buffer   Lysates were cleared of debris by centrifugation at   g for  min at  C  EGFR immunoprecipitations were performed on  µg of cleared lysate using  ng of rabbit anti EGFR and Protein A G Plus Sepharose overnight at  C  Precipitations were washed  times in lysis buffer before boiling in Laemmli buffer  Elutions were immunoblotted with anti ubiquitin and EGFR  Twenty micrograms of cleared lysate were immunoblotted for each of the c CBL constructs using anti HA   Statistical Analysis  Mutation rates between different groups were compared using Fisher s exact test  For continuous variables  group comparisons were performed using analysis of variance  followed by Sidak s adjustment for multiple comparisons  Experiments involving repeated measurements over time were analyzed using repeated measures ANOVA with the Greenhouse Geisser adjustment to the degrees of freedom  Analyses were conducted using STATA  software    Go to  Results c CBL Gene Mutations in Lung Cancer  To investigate the role of c CBL in lung cancer  we analyzed its genomic DNA in tumor and paired normal samples drawn from multiple ethnicities  The lung tumor samples represented Caucasians   African Americans   and Taiwanese  lung cancer patients  We designed  pairs of primers to sequence the coding region of c CBL gene that spans exons  to    We identified  unique somatic mutations in c CBL exons among  different patients  A variation LF  a known SNP  in exon  was also detected  Importantly  the eight novel non synonymous mutations were confirmed by sequencing both strands of c CBL genomic DNA obtained from lung tumor samples   Moreover  none of the  mutations were detected in the corresponding normal tissue  indicating that these were somatic mutations  Four synonymous single nucleotide variations  were also identified but were not used further in this study   Table  Table  c CBL mutation analysis in  lung cancer patient tumor tissues  Three of the  novel non synonymous mutations were located in the TKB  domain   one in the RING finger domain   one in the proline rich region   and three in the C terminal region  of the c CBL protein   In Figure B  we show model chromatograms of representative samples   Figure  Figure  c CBL mutations and LOH in non small cell lung cancer  q LOH of c CBL Gene  Paired lung tumor and normal lung tissue samples from Taiwanese patients  were investigated for LOH  Eight  showed LOH at the c CBL locus on chromosome  while  samples  revealed normal allelic contribution at the microsatellite markers    c CBL Mutations in Different Ethnic Groups  The c CBL double mutant SN HY was found in the same patient and the overall mutation rate for c CBL in lung tumors was      The frequency of c CBL mutation was highest in large cell carcinoma  followed by squamous carcinoma  and the least was observed in adenocarcinoma    although these rates were not statistically significant   Mutation rates were    among Caucasians      in African Americans   and     in the Taiwanese population  Additionally two Taiwanese patients with lung cancer  had the known SNP LF  Ethnic differences were not statistically significant  however the power to detect differences was low   Mutations in MET and EGFR Can Be Co Associated with c CBL Alterations  Since East Asians with lung cancer have a higher frequency of EGFR and MET mutations in lung tumors     we also determined mutations in EGFR and MET in the same Taiwanese cohort samples and compared the results with the observed c CBL alterations   In the  samples tested  we did not find any overlap between c CBL mutations and c CBL LOH   Of the three c CBL mutants   one of the samples had a MET mutation  and the other had an EGFR mutation   Among the  samples that had a LOH at the c CBL locus  had an additional mutation in MET  and  had an EGFR exon  deletion  Twenty six samples had neither c CBL mutation nor c CBL LOH   Among these  samples  had a MET mutation    had an EGFR mutation  and  had no other MET or EGFR mutation  Thus the rate of MET or EGFR mutations among patients with LOH at the c CBL locus  was similar to that seen in patients without c CBL mutation or LOH    These  patients with no identifiable mutation in c CBL  MET or EGFR represented    of the  patients analyzed in the Taiwanese patient cohort  Conversely     Taiwanese lung cancer patients have an identifiable mutation in either c CBL  MET or EGFR or a combination of the three genes   Additionally  we determined p and KRAS mutations in these Taiwanese cohorts  Two p and  KRAS mutation were detected  The single KRAS mutation overlapped with one p mutation  This patient also had the EGFR exon  deletion but had no c CBL mutation  The other p mutation sample had a c CBL LOH with concurrent MET NS mutation  Thus  in the Taiwanese samples analyzed  p KRAS mutations and c CBL mutations were mutually exclusive    Figure  Figure  c CBL mutations and relationship to MET and EGFR mutations in lung cancer  Cellular Functions of c CBL Alterations in the Context of Lung Tumorigenesis  A  E activity is intact in the mutant c CBL proteins To investigate whether the different c CBL mutations affect the E activity  EGFR was chosen as a model substrate for c CBL E function  All of the c CBL mutants tested enhanced ubiquitination of the activated EGFR similar to the wild type c CBL protein  This result demonstrates that the catalytic activity of the c CBL mutants is not impaired when EGFR was the substrate     Figure  Figure  Ubiquitination  viability  expression and cell cycle analysis of various c CBL mutants  B  Effect on lung cancer cell viability The effect of a representative c CBL mutant from each of the three ethnic backgrounds on lung cancer cell viability in cell lines was determined  SN HY double mutation  QE  and W  were identified in lung tumor samples obtained from a Caucasian  a Taiwanese and an African American  respectively  As described in methods  the c CBL wild type  and the above three mutants were expressed after cloning them into pAlterMax vector in A cells  These cells express relatively low basal levels of endogenous c CBL   Transfection efficiency was comparable between different groups and the number of cells transfected with c CBL wild type construct was about   compared to control cells that were transfected with the empty vector  Cells transfected with SN HY  QE and W  c CBL mutant constructs resulted in increased number of viable cells that was        and    higher respectively  relative to the empty vector control transfected cells and significantly different from the wild type construct    Relative levels of c CBL protein in whole cell lysates prepared from samples obtained from a parallel experiment was determined  The c CBL protein levels in samples representing untransfected and empty vector transfected cells were comparable and those representing the c CBL WT and the three c CBL mutants were comparable    C  Effect on cell cycle To investigate if the increases in cell viability in different c CBL mutants are due to increased cellular proliferation  a cell cycle analysis was performed  A cells were transfected with the c CBL WT or the three different mutants  SN HY  QE and W   The empty vector transfectant was used as a control  Forty eight hours after transfection cell cycle analysis was performed as described in materials and methods  There was no significant change in the subG  G or the S phase of the cell cycle among the different mutants compared to the WT construct   The G M phase of the cell cycle showed an increase in cell numbers for the three mutants  SN HY  QE and W   when compared to the WT but again the difference was not statistically significant     D  Effect on cell motility To investigate the effect of the expression of the above three c CBL mutants on cell migration  we carried out wound healing assay as described in materials and methods  The closing of the scratch or the wound was monitored at         and  h    In all the samples  that represented cells transfected with mutants  the wound gap was much smaller than that seen in the sample that represented cells transfected with c CBL WT   We also determined the rate of wound closure for all the five groups  At  h  wild type c CBL transfectants showed    open wound while the SN HY  QE and W  mutants showed        and    open wound respectively     Figure  Figure  c CBL mutations affect wound healing in A cells  E  c CBL knockdown increases cell viability It is hypothesized that the LOH seen in our samples could lead to decreased expression of c CBL  Thus we tested the effect of c CBL knockdown in lung cancer cells  Compared to A  H lung cancer cells express relatively high levels of endogenous c CBL   c CBL expression was knocked down using lentiviral construct that expressed c CBL specific shRNA and compared the results with those that were transduced with scrambled shRNA and the results are shown in Figure   We identified several clones that revealed varying degrees of c CBL knockdown showing different sets of c CBL lentiviral shRNA knockdown efficiency   Of all the clones tested  Clone  was chosen for further experiments  Equal amount of cells were seeded in a  well plate and the cell proliferation was measured at various times and the results are depicted in Figure B  As expected  number of cells increased in a time dependent fashion from  to   relative to scrambled shRNA as control in a span of  h    The cell cycle phases in H cells that were knocked down with c CBL shRNA were looked at and compared with the scrambled shRNA  There were no discernable differences between these two constructs in the different phases of the cell cycle    Figure  Figure  Knockdown of c CBL using an shRNA increases cell proliferation  Go to  Discussion Our results demonstrate that c CBL is somatically mutated  in lung cancers  and can significantly contribute to enhanced cell viability and motility  There was also a high prevalence of LOH with respect to c CBL in lung tumors that harbored MET or EGFR mutation   In the present study  we have demonstrated the occurrence of c CBL mutations in lung cancer patients  especially with different ancestral variations  Mutations in c CBL have been recently reported in juvenile myelomonocytic leukemia and myeloid malignancies  In the AML study  the mutation RQ located in the junction of the RING finger and the linker region inhibited FMS like tyrosine kinase   internalization and ubiquitination   thus contributing to the gain in function for the RTK  In addition  mutations such as HY  CR  and LP mapped to the RING finger domain and the linker region of c CBL that is required for its E activity           Additionally  homozygous mutations in the RING finger domain of the c CBL gene were described as a result of acquired Uniparental Disomy    It is important to note that our results indicate LOH at q locus and these are mutually exclusive from missense mutations of c CBL  The somatic mutations were all heterozygous  The mutations in AML led to abrogation of the E activity leading to prolonged RTK activation  In addition mutants located on the linker region surrounding the RING finger domain exhibited enhanced AKT signaling in response to cytokine stimulation   In addition  it was shown in NHT cells  that neither mutations in the RING finger nor the linker region causes transformation  however while certain mutations perturbs the ubiquitination  others affect receptor recycling and prolong kinase activity    We report here c CBL mutations that mapped not only to the RING finger domain  but also to the TKB domain  proline rich domain and the C terminal region  but none mapped to the linker region as reported in the AML studies described above           In addition   mutants that we detected were found in different ethnic backgrounds  For example  SN HY  QE  W  were detected in Caucasians  Taiwanese and African Americans  respectively  The results point out not only the difference between lung cancer and other cancers  but also genetic polymorphism among different races in the same cancer  Interestingly  there is a large disparity between African American and other ethnic populations with lung cancer   We have previously shown that there was a low frequency of EGFR and MET mutation in African Americans as compared with Taiwanese and Caucasians   In this study  the number of African American samples analyzed was relatively fewer and we found  mutations that are unique to this ethnicity  It would now behoove us to further study the genetic alterations that can occur and determine the targeted therapeutics for African Americans  Our results provide evidence of the importance of c CBL in tumorigenesis and potential signaling  Our prediction  based on the AML data  would be that the VI RING finger domain mutation would affect the E activity  Also  it will be important to determine the binding partners for c CBL in the TKB domain and proline rich domain mutations  It has previously been shown that the TKB domain can bind to growth factor receptors and it will be important to determine the cross binding of these mutants to MET and or EGFR  It would also be important in the future to look at fluorescence in situ hybridization copy number changes in c CBL in lung cancer   c CBL plays an important role in down regulating RTK mediated signaling through K poly ubiquitination and subsequent downregulation of RTKs followed by lysosomal degredation   Mono ubiquitination or ubiquitinated with K linked chains of substrates by c CBL may lead to enhancement of biological and biochemical functions   The mutations that we analyzed in our studies all point out to the fact that the E activity of c CBL on EGFR is intact  the EGFR levels in the various mutants remain same   Multiple kinases  both RTKs and non RTKs could be acted upon by c CBL  including ERBs  PDGFR  FMS  MET  c Kit  VEGFR  FLT   RON  FGFR  IR  as well as SYK  FYN  LCK  FGR  LYN and c ABL   In lung cancers the relevant substrates of c CBL in terms of degradation or signal transduction are yet to be identified   The observation that c CBL somatic mutations  especially SN HY  QE and W  showed increased cell viability and cell motility that is in agreement with the physiological role for D cbl in the regulation of apoptosis and differentiation identified in Drosophila is very significant   It has been previously shown that activating c CBL mutation downregulates EGFR signaling and decreases cellular proliferation and migration in breast cancer cell lines   Although the role of c CBL in the negative regulation of RTKs is well substantiated  thereby suggesting that it is a natural tumor suppressor  studies in cancer cells have revealed both tumor suppressor and tumor promoting activities depending on the type of c CBL mutation and the number of alleles at the c CBL locus   In agreement with the above  the three c CBL mutants described here appear to have tumor growth and metastasis promoting properties  Although these mutants are outside of the RING finger or the linker region of c CBL  their downstream effects are significant so as to cause increased proliferation and migration  but the substrate affected by these mutations are not known yet  This raises the possibility that some of the cellular functions of c CBL are independent of its ubiquitin ligase activity  an area that we are currently investigating  The oncogenic nature of RTKs  addiction of cancers to growth signals and given the clustering of c CBL  EGFR and MET mutations  it is possible that the transforming effect of c CBL mutations is most likely a combinatorial effect of the three  We also show that LOH for c CBL was found in a significant number of samples that harbored MET or EGFR mutations  The fact that about   of lung tumor samples are likely to have c CBL mutations and an additional   are likely to harbor c CBL related LOH  makes c CBL a highly mutated molecule in lung cancer  Since LOH alone is not enough to cause a transforming event       associated mutation in the MET or EGFR locus or yet another RTK discussed above may play a role in carcinogenesis  We predict that this LOH in c CBL results in haploinsufficiency that downplays RTK ubiquitination leading to hyperactivity of the RTKs  However  whether this is sufficient cause for tumorigenesis remains to be determined  Consistent with our hypothesis is the fact that c cbl    mice have increased kinase activity in lymphocytes  but is not sufficient for tumor formation       c CBL LOH could also lead to increased expression of c CBL from the other allele to compensate for a loss of an allele  Alternately  there could be some form of synergy working with reduced c CBL levels and mutated receptors that exacerbate the phenotype of each alone   Previous studies from our lab and others have shown that East Asians with lung cancers have relatively high frequencies of gain of function of mutations in RTKs such as EGFR and MET   In a cohort of Japanese patients an activating MET mutation was identified in the splice region that deletes the juxtamembrane domain that is involved E activity of c CBL   This study also found that activation of MET is mutually exclusive of EGFR  KRAS and HER gene mutations   We failed to detect such mutations in significant numbers in lung tumor samples obtained from African Americans  and Caucasian  patients  One MET mutation was identified in each of the groups whereas  and  EGFR mutations were identified in the African American and Caucasian cohorts respectively  EGFR mutations have earlier been identified as one of the key mutations affecting lung adenocarcinoma patients in a comprehensive study of  patients   Our study encompasses different histologies of NSCLC  However  the published series did not find any mutations in c CBL or MET unlike our study that encompassed different subtypes of NSCLC  It is important to note  we have recently shown that MET mutations in lung cancer are in majority germline   We have reported earlier c CBL mutations in a small cohort of Taiwanese lung cancer samples   In our efforts to understand the ethnic differences in the lung oncogenome  we also looked at PAX transcription factors such as PAX and PAX that are highly expressed in lung cancers  however there was no preferential expression or mutations of the above genes in lung tumor samples of African Americans  In this study  we show relatively high frequency of c CBL mutations in lung cancers  especially in the large cell type among Caucasians and particularly among African Americans  We therefore propose c CBL as an efficacious target for lung cancers in African Americans that needs to be further substantiated  This is all the more important because the prognosis for African Americans with lung cancer  especially for men is much poorer compared to their Caucasian counterparts    In conclusion  the results presented in this study demonstrate that c CBL is frequently mutated or even lost in lung cancers  Our results support a role for c CBL mutants that are independent of its ubiquitination activity  Given the relatively high mutation rates in c CBL as well as RTKs such as MET and EGFR  it is likely that their combined effect could be synergistic in promoting tumorigenesis',\n",
       "       ...,\n",
       "       'The Runt related transcription factor  gene  belongs to a family of three evolutionarily conserved genes  that code for transcription factors involved in cell lineage differentiation during development   The RUNX gene  located in chromosome q   is crucial for the establishment of definitive hematopoiesis and the generation of hematopoietic stem cells in the embryo   RUNX  located in chromosome p  is essential for osteogenesis and RUNX  located in chromosome p   plays a major role in neurogenesis and thymopoiesis   Members of this family share a  amino acid region of high sequence homology  known as the  Runt domain   first identified in the Drosophila runt gene  In addition to this  Runt homology domain    there is a  amino acid sequence  VWRPY  that is   conserved at the C terminal end of each of the three gene products   The Runt domain is responsible for heterodimerization with the core binding factor b   to form a transcription factor  and for DNA binding   The RUNX gene spans  kb and consists of  exons with two distinct promoters  each followed by the initiation codon ATG  The RUNX protein is composed of  amino acids and contains several regions of major importance    These are the RHDs coded by exons   and located in the N terminal part   the transcription activation domain  coded by part of exon  and exon   the transcription inhibition domain  and the VWRPY sequence  All three are located in the C terminal region   The two promoters  which are  kb apart  generate several alternatively spliced mRNAs differing in their types of   and   untranslated regions  and their coding regions  Although at least  different RUNX mRNAs are generated  two splice variants are well characterized  The distal promotor driven transcript  RUNXb  codes for a  amino acid protein and the proximal promotor driven transcript  RUNXc  a  amino acid protein  The RHD is retained in both transcripts   The fulllength coding regions harbor the C terminal half of the protein  which regulates transcription   A third splice variant  RUNXa  codes for a protein that retains the RHD but lacks the TAD  This protein has a higher affinity for DNA binding and suppresses transcriptional activation of RUNXb   RUNX acts as a key regulator of hematopoiesis through the regulation of various hematopoietic genes  including growth factors   surface receptors   signaling molecules  and transcription activators    The RUNX protein acts as an activator or repressor of target gene expression depending upon the large number of transcription factors  coactivators and corepressors that interact with it  RUNX functions as an organizing protein that facilitates assembly of transcriptional activation or repression complexes   By recruitment of non DNA binding proteins as p CBP and histone acetyltransferase   it contributes to the activation of transcription of target genes  Upon recruitment of non DNA binding repressors such as mSinA  Groucho TLE and histone deacetylase   it represses transcription of target genes   RUNX translocations Some translocations involving RUNX have been shown to be frequent while others have only been reported in a few or single cases   At present   partner chromosome bands have been described  but the partner gene has only been identified in  translocations at the molecular level   Five translocations are known to be frequent in malignant hemopathies  They involve six different partner genes  The t translocation is a rare but recurrent translocation associated with therapy related acute myeloid leukemia myelodysplasia    It results in a RUNX PRDM  fusion gene  The t translocation is a recurrent rearrangement observed in patients with de novo or therapy related AML  therapyrelated MDS and chronic myeloid leukemia in the blastic phase  This translocation is very often included in a complex karyotype and the prognosis is poor  Three genes have now been shown to fuse with RUNX  These are RPLL  and MECOM    The t  first described by Rowley in    was shown to involve the RUNXT  gene in the early s   The RUNX RUNXT fusion gene is identified in some   of the patients with AML M subtype and is frequently associated with other cytogenetic abnormalities  Complete remission is achieved in the majority of patients  but relapse usually occurs within a few years   The t is the most common structural chromosomal abnormality in pediatric common  or B cell acute  lymphoblastic leukemia  accounting for approximately    of the cases  Complete remission is achieved in almost all the patients and the prognosis is very good  although late relapse is possible   At the molecular level  this translocation fuses the   region of ETV  with almost the entire coding region of RUNX   First described by Raimondi et al  in  in a case of AML subtype M   the t  has now been reported in  patients  including  with AML M   patients having had chemotherapy for a previous cancer   This translocation was shown to induce the formation of a new chimeric gene involving the RUNX and CBFAT  genes  There were  other partner genes or sequences identified in translocations associated with one or a few cases of malignant hemopathies   The fusion genes that were characterized will be described elsewhere  A total of  translocations were also seldom reported   In all these translocations  the RUNX gene was shown to be disrupted  presumably leading to the fusion of the RUNX gene with an as yet unidentified gene partner  However  in a few cases  these translocations involve a break point located in a band in which a partner gene has already been identified  This is true  for example  for translocations involving bands q  q and q  Whether these translocations involve an already known partner gene  or a different gene  remains to be determined  Unusual   complex rearrangements involving the RUNX gene Complex translocations involving three or four chromosomes and insertions have been reported in fewer than  cases   All but two of the translocations in which both genes were identified  lead to the formation of a RUNX RUNXT or a RUNX ETV fusion gene  Complex variant t translocations have been reported in    of cases with RUNX RUNXT fusion gene  the remaining patients having the classic t   Compilation of six studies  totaling  patients with RUNX RUNXT identified by real time PCR   real time quantitative PCR or RNA based PCR  demonstrates that  patients  have the usual t whereas the remaining  patients  have an unusual or complex rearrangement   Among these latter patients  nine  have a three way translocation  two  a four way translocation and seven  an insertion  A literature search allowed us to retrieve  complex translocations involving the RUNX gene and band q   The involvement of the RUNXT gene was not searched for in ten cases  it was assumed that these translocations were variant t translocations  Still  one cannot exclude the possibility that another gene located in band q was fused to RUNX   or that the fusion occurred between a gene located on the third chromosome and RUNX  The RUNX RUNXT fusion gene was identified in the remaining  translocations  There were  three way and six four way translocations  Most of the three way translocations could be recognized by conventional cytogenetics  However  in a few patients  the t variants were  masked  as they appeared to involve chromosome  and a chromosome other than chromosome   However  in these cases  fluorescent in situ hybridization   using dual color AML ETO probe and or RT PCR  confirmed the involvement of both genes and the formation of a fusion gene   All the chromosomes except chromosomes      and Y have been involved in complex translocations  Most of them were involved more than once  chromosomes  and  having been identified in five translocations each and chromosomes    and  in four translocations each  However  even among these chromosomes  no band was found to be nonrandomly broken  In three way translocations  up to three fusion genes can theoretically be generated  one fusing the   region of RUNX with the   region of RUNXT  the second fusing the   region of RUNXT with the   part of another gene and the third fusing the   part of this other gene with the   region of RUNX  Whether generation of these other fusion genes is associated with a more aggressive disease among patients with three way translocation than those with the classic t remains to be elucidated  Unfortunately  to the best of our knowledge  no group has tried to identify these other fusion genes  However  one cannot exclude that some apparently complex translocations could be the result of two or more consecutive translocations  In such cases a standard RUNX RUNXT fusion gene would be created  whereas the subsequent translocation would involve loci close to  but different from  the RUNX and RUNXT genes  A total of  chromosomal fragment insertions leading to RUNX RUNXT fusion were also retrieved from the literature   This includes the insertion of chromosome  material  into band q   or vice versa  the insertion of chromosomal material  into band q   All the insertions of material from chromosome  into chromosome  were cryptic  although one was suspected on G banded metaphases   Some of the insertions of chromosome  material into chromosome  were identified by conventional cytogenetics because they involved larger chromosomal fragments   Complex translocations involving chromosomes    and another chromosome have been rarely reported   In only three of them   FISH using dual color TEL AML probe and or RT PCR confirmed the presence of the fusion gene   Insertion is even rarer  only one case of insertion of q in band p with formation of the RUNX ETV fusion gene having been reported    Structure   function of the fusion genes The fusion genes can be classified into five categories according to their structure and  as a consequence  to their function   In addition  the same translocation can lead to different fusion products acting through different mechanisms  Indeed  alternate splicing transcripts have been reported in many fusion genes involving RUNX  leading to in frame fusion products with or without others leading to out of frame fusion products   Furthermore  most studies have focused on the consequences of the chimeric proteins on the transcriptional activity of RUNX  but few have analyzed the contributions of the fusion partners to leukemogenesis  However  there is now strong evidence that RUNX and its partner could cooperate to contribute to leukemogenesis using different pathways involving  for example  GATA and RAS   Most of the chimeric genes involving RUNX fuse the   part of the gene with the   part of the partner gene  The RHD  located in the   region of RUNX  is retained in these fusion genes  but not the TAD situated in the   part of the gene   Several translocations also lead to truncated fusion genes retaining the RHD but not the TAD of RUNX  These truncated RUNX gene products may compete with wildtype RUNX and act as dominant constitutive repressors inhibiting the function of the normal RUNX   The RUNX PRDM fusion gene  obtained as the result of the t translocation  retains the RHD of RUNX and almost the entire PRDM gene   Since the RUNX PRDM gene shares extensive homology with RUNX MECOM  it is likely that the RUNX PRDM protein acts in a similar way  as has been shown for the RUNX MECOM protein   The translocation also generates a truncated RUNX transcript containing RHD   The t translocation  described in a  year old male with AML by Nguyen et al   leads to the formation of a RUNX YTHDF  transcript that includes the RHD  but not the TAD of RUNX   The breakpoint occurs in intron  of the YTHDF gene that has six exons but is only translated from exons  to   Therefore  the fusion protein has a truncated RUNX protein without translating YTHDF  The t translocation generates an in frame chimeric RUNX ZNF  gene with the breakpoint located in intron  of the RUNX gene   As a result  the fusion protein retains the first  amino acids of RUNX  but only the first seven amino acids of the RHD  leading to a truncated and  presumably  nonfunctional RHD  This would prevent CBF b and DNA binding and transcriptional activation of target genes involved in hematopoietic differentiation  Cytogenetic analysis of leukemic cells of a patient with T cell acute lymphoblastic leukemia revealed a t translocation  Molecular analysis revealed three RUNX AFF  fusion transcripts  one in frame  one out of frame and one showing premature termination due to the insertion of an intronic sequence   The breakpoints occurred in intron  of RUNX and all three retained the RHD  The t translocation leads to the formation of  among others  a RUNX  MECOM transcript that includes the RHD of RUNX and the two zinc finger clusters of MECOM  The chimeric protein can accumulate PEBPb in the nucleus more efficiently than the wild type RUNX  a mechanism contributing to leukemogenesis   As a consequence  RUNX MECOM exerts a dominant negative effect over the wild type RUNX in transcription activation  an antiapoptotic effect  and a proliferation stimulation effect    The RUNX RPLL transcript  resulting from the t translocation  includes the RHD of RUNX and the last  codons of RPLL  The fusion mRNA is terminated by an out of frame stop codon shortly after the junction between RUNX and RPLL   It is likely that the RUNX RPLL protein acts as a negative inhibitor of the RUNX protein  The t translocation fuses the   region of the RUNX gene  to a  codon sequence of a novel gene named FGA    This rearrangement leads to a stop codon in the chimeric gene  resulting in a truncated RUNX protein possessing the RHD domain  but not the TAD  The t translocation  identified in one patient with acute myeloblastic leukemia  leads to the formation of an in frame RUNX SHD  transcript that includes the RHD of the RUNX gene and the full length coding sequence of the SHD gene   The cryptic t translocation leads to the formation of a RUNX USP fusion gene that includes the RHD  part of the TAD of RUNX and almost the entire open reading frame  of USP  gene  Unlike the other chimeric genes in which the breakpoint occurs   to the TAD  in the RUNX USP fusion gene it occurs within the TAD  however  it is most likely nonfunctional because it is disrupted   In the RUNX RUNXT protein product generated by the t translocation  the TAD of RUNX  which normally binds the transcriptional coactivators p CBP  is replaced by almost the entire RUNXT  As a consequence  the fusion protein recruits a corepressor complex involving N CoR  mSinA  SMRT and HDACs instead of the coactivators p CBP  Therefore  the RUNX RUNXT transcript functions as a transcriptional repressor of genes normally activated by RUNX   Furthermore  genes  such as TISb   under the downstream control of the fusion protein are overexpressed and contribute to leukemogenesis   The t translocation generates an in frame chimeric gene that encodes a protein made of the N terminal portion of RUNX  and the entire TRPS  The TRPS  gene encodes a nuclear transcription factor with ten zinc finger domains  including a GATA type zinc finger   Agerstam et al  described a t that generated a chimeric gene made of the first four exons of RUNX and repetitive sequences of a gene with unknown function  As a result  the fusion protein contained a truncated RHD and  amino acids from chromosome  ending with a stop codon   Therefore  this truncated RUNX protein may compete with wild type RUNX and act as a dominant constitutive repressor inhibiting the function of the normal RUNX  The t translocation generates two in frame chimeric transcripts from alternate splicing that encode proteins made of the N terminal portion of RUNX  and the C terminal of LPXN   a member of the paxillin superfamily of adaptor proteins   LPXN is preferentially expressed in hematopoietic cells and negatively regulates B cell receptor signaling   Out of frame fusions  in which the RHD of RUNX is retained but an early stop codon occurs in the LPXN gene  are also generated  Two RUNX MACROD  fusion genes owing to alternate splicing were identified following molecular analysis of a t  translocation   Both retain the RHD of RUNX and the full length coding sequence  but include exon  of MACROD  a gene for which the precise biological function remains to be elucidated  At the molecular level  the t translocation fuses the   region of ETV and the   region of RUNX   The ETV RUNX fusion gene retains the RHD and the TAD of RUNX as well as the promoter and the central repression domain of ETV  The ETV RUNX fusion protein retains the ability to bind the RUNX target sequences and has the potential to function as a HDAC dependent repressor  causing deregulation of the RUNX target genes   Molecular analysis of a t translocation revealed an out of frame fusion of RUNX with the CPNE  gene  resulting in translational termination after two amino acids of Copine VIII and  thus  a truncated RUNX protein  This truncated protein contained the RHD but not the TAD  the breakpoint having occurred in intron  of the RUNX gene   The significant homology between CBFAT  fused to RUNX in the t translocation  and RUNXT suggests that both RUNX CBFAT and RUNX RUNXT fusion genes have similar activity   The RUNX CBFAT protein is an altered transcriptional corepressor capable of recruitingThe t translocation fuses the   end of the RUNX gene  to a  nucleotide sequence on chromosome  termed AMP    This rearrangement leads to an out of frame fusion gene  resulting in a truncated RUNX protein bearing the RHD domain  but not the TAD  The rare t translocation leads to the formation of an in frame RUNX  PRDX transcript that includes the RHD of the RUNX gene  The PRDX gene  a member of the peroxiredoxin family  has a peroxidase activity  but is also involved in other cellular functions  including cell proliferation and differentiation   Chan et al  described a patient with myelodysplastic syndrome associated with a t  identified in conventional cytogenetics that was demonstrated  by FISH and molecular studies  to be a three way translocation  t  leading to a RUNX ZFPM fusion gene   The ZFPM gene  encodes a protein that retains the RHD of RUNX and the zinc fingers that interact with GATA proteins and the CtBP  interaction domain of ZFPM  Two patients presenting with chronic myelomonocytic leukemia associated with a cryptic inv were found to have a USP  RUNX fusion transcript  In both cases  the transcript did not have an open reading frame and was made of the   part of the USP  gene and the   part of RUNX  Furthermore  the RHD coded by exons  to  of the RUNX gene is not preserved in the fusion protein   Identification of two new partners of the RUNX gene Bone marrow and or peripheral blood of patients with myeloid leukemia referred to the cytogenetics laboratory of the University Hospital in Brest are usually analyzed by FISH using bacterial artificial chromosome  clones in search of fusion genes involving the RUNX gene  This approach allowed us to identify a new partner gene in one case and to demonstrate in another case the complexity of rearrangements involving band q in which three RUNX partners have already been identified   Analysis of a t Conventional cytogenetic analysis of peripheral blood cell culture of a woman with chronic myelomonocytic leukemia   a myelodysplastic myeloproliferative neoplasm  identified a  XX   karyotype in seven of the  cells studied  However  FISH analysis using the dual color TEL AML probe  showed disruption of the AML probe and localization of part of it on a small chromosome  As abnormalities of chromosome  are recurrently observed in CMML  p and q subtelomeric probes  were applied  The analysis showed the q subtelomere to be translocated to the long arm of chromosome   signing the presence of a cryptic translocation involving bands q and q  t  Further FISH analyses were performed with BAC clones located on q and a clone  covering the RUNX gene   Clones RP L and RP L  normally located in band q   were found to be split by the breakpoint  Hybridization with RP L and RP G showed colocalization of both signals  as the result of a probable fusion gene  BAC clones RP L and RP L contain the ZFP  gene  the protein coded by this gene could be involved in transcription regulation  Sakamoto et al  showed that the ZFP protein interacted with the intracellular domain of NOTCH  a signaling protein having a major role in mesenchymal cell differentiation   Furthermore  the expression of ZFP is under the control of RUNX  belonging to the same gene family  Analysis of a t A complex translocation involving five chromosomes  which included both chromosomes   t  was identified in the bone marrow cells of a  yearold patient with acute myeloblastic leukemia without maturation  at relapse following bone marrow trans  at relapse following bone marrow transplantation   FISH analysis using the dualcolor TEL AML probe showed disruption of the AML probe and translocation of part of it on chromosome   in band q  Further FISH analyses were performed with BAC clones located on q  some of them covering the MECOM complex locus and RPLL gene already known to be fused with RUNX in  translocations  and a clone  covering the RUNX gene  In this case  the breakpoint occurred between clones RP P and RP C at q  eight megabases distal to the MECOMRPLL cluster  therefore arguing for the involvement of another gene  Three genes have been mapped in this interval between RP P and RP C  including WIG  a wild type p induced gene   This gene could be a good candidate to fuse with RUNX as it is located in the nucleus and contains three zinc finger domains and a putative nuclear localization signal  Unfortunately  no remaining material was available for further cytogenetic and molecular studies  This case illustrates the importance of always identifying the partner gene even when RUNX is involved in a  recurrent translocation with known partner gene   Conclusion Most of the translocations involving RUNX lead to the formation of a fusion transcript made of the   region of RUNX  including the RHD but not the TAD  fused to the   region of a partner gene  the breakpoints being concentrated in introns  and  of the RUNX gene   One exception is the RUNX  ETV fusion gene  in which the   sequences of RUNX  including RHD and TAD  are fused to the   region of ETV  including its promotor  All the translocations that retain RHD but remove TAD have a leukemogenic effect by acting as dominant negative inhibitors of wild type RUNX in transcription activation  mimicking the normal  but rare  RUNXa splice variant which has higher affinity for DNA binding and suppresses the transcriptional activation of RUNXb  Future perspective Given the crucial role of RUNX for the establishment of definitive hematopoiesis and generation of hematopoietic stem cells  it is expected that future studies will try to better understand its implication in normal hematopoiesis  Thus  this will provide more data on the leukemogenic effects of fusion genes involving RUNX and  possibly  give useful information on the phenotypic heterogeneity of leukemia associated with RUNX abnormalities  Much will be learned when currently unidentified genes are discovered  putting more insight into the mechanisms by which these partner genes and RUNX may cooperate in contributing to leukemogenesis using several different pathways  HDACs  thus repressing the expression of RUNX target genes     Executive summary Normal function   structure of the RUNX gene n\\tRUNX has a crucial role in establishment of definitive hematopoiesis and generation of hematopoietic stem cells in the embryo  n\\tRUNX spans  kb and consists of  exons with two distinct promoters generating at least  different spliced mRNAs  n\\tRUNX acts as a key regulator of hematopoiesis through the regulation of various hematopoietic genes  n\\tThe RUNX protein has two major domains of biological significance  a Runt homology domain and a transcription activation domain  n\\tThe RUNX protein acts as activator or repressor of target gene expression  Translocations involving the RUNX gene n\\tThere are  partner chromosome bands involved in translocations with the RUNX gene  which have been described  n\\tA few translocations are recurrent  but the majority have been reported in a few or single cases  n\\tComplex translocations involving three or four chromosomes and insertions have been reported rarely   Fusion genes involving the RUNX gene   their biological consequences n\\tA total of  partner genes involved in fusion with the RUNX gene have been characterized  n\\tAlternative splicing transcripts have been reported in many fusion genes  leading to different fusion products acting through different mechanisms  n\\tAlmost all the translocations retain the Runt homology domain but remove the transcription activation domain  n\\tThey have a leukemogenic effect by acting as dominant negative inhibitors of wild type RUNX in transcription activation   Table   Chromosomal translocations involving the RUNX gene and an identified partner gene  Translocation Partner gene Disease No  cases Ref  t PRDM MDS  AML    t YTHDF AML M   t ZNF AML M   t AFF T cell ALL   t MECOM MDS  AML  bCML    t RPLL MDS  AML  bCML    t FGA T cell ALL   t SHD AML M   t USP AML M   t RUNXT AML M    t ZFPM MDS   t TRPS AML M   t Gene of unknown function AML   t LPXN AML   t MACROD AML   t ETV B cell ALL    t CPNE AML M   t CBFAT AML    t AMP AML   t ZFP CMML  This study t PRDX AML M   inv USP CMML     Several dozen  of cases  ALL  Acute lymphoblastic leukemia  AML  Acute myeloid leukemia  bCML  Chronic myeloid leukemia in blastic phase  CMML  Chronic myelomonocytic leukemia  MDS  Myelodysplastic syndrome    Table   Translocations with unknown partner gene  Translocation Disease No  of cases Ref  t AML M   t AML M   t AML Mb   t MDS   t AML M   t AML M   t AML M   t AML M   t ALL   t AML M   t AML M   t T cell ALL   t AML M   t AML  AML M  CMML   t MDS   t ALL   t AML M   t AML M    t AML   t AML M   t AML M   t AML M   t AML M   t AML M   t AML   t MDS   t AML   t AML  AML M   t AML   t MDS  AML M   t bCML   t AML M   t bCML   ALL  Acute lymphoblastic leukemia  AML  Acute myeloid leukemia  bCML  Chronic myeloid leukemia in blastic phase  CMML  Chronic myelomonocytic leukemia  MDS  Myelodysplastic syndrome   Table   Complex and unusual rearrangements involving the RUNX gene  Disease Chromosomal rearrangement Fusion gene Ref  AML M t RUNX    AML M t RUNX    AML M t RUNX    AML M t RUNX    AML M t RUNX    AML t RUNX    AML t RUNX    AML M t RUNX    AML M t RUNX    AML M t RUNX    AML M t RUNX RUNXT  AML M t RUNX RUNXT  AML M t RUNX RUNXT  AML M t RUNX RUNXT  AML M t RUNX RUNXT  AML M t RUNX RUNXT  AML M t RUNX RUNXT  AML M t RUNX RUNXT  AML t RUNX RUNXT  AML M t RUNX RUNXT  AML M t RUNX RUNXT  AML M t RUNX RUNXT  AML M t RUNX RUNXT  AML M t RUNX RUNXT  AML M t RUNX RUNXT  AML M t RUNX RUNXT  AML M t RUNX RUNXT  AML M t RUNX RUNXT  AML M t RUNX RUNXT  AML M t RUNX RUNXT  AML M t RUNX RUNXT  AML M t RUNX RUNXT  AML M t RUNX RUNXT  AML t RUNX RUNXT  AML M t RUNX RUNXT  AML M t RUNX RUNXT  AML M t RUNX RUNXT  AML M t RUNX RUNXT  AML M t RUNX RUNXT  AML M t RUNX RUNXT  AML M t RUNX RUNXT  AML M t RUNX RUNXT  AML M t RUNX RUNXT  AML t RUNX RUNXT  AML M t RUNX RUNXT  AML M t RUNX RUNXT  AML M t RUNX RUNXT  AML t RUNX RUNXT  AML M t RUNX RUNXT  AML M t RUNX RUNXT  AML t RUNX RUNXT   Table   Complex and unusual rearrangements involving the RUNX gene   Disease Chromosomal rearrangement Fusion gene Ref  AML M t RUNX RUNXT  AML M t RUNX RUNXT  AML M t RUNX RUNXT  AML M ins RUNX RUNXT  AML M ins RUNX RUNXT  AML M ins RUNX RUNXT  AML M ins RUNX RUNXT  AML M ins RUNX RUNXT  AML M ins RUNX RUNXT  AML M ins RUNX RUNXT  AML M ins RUNX RUNXT  AML M ins RUNX RUNXT  AML M ins RUNX RUNXT  AML M ins RUNX RUNXT  AML M ins RUNX RUNXT  AML M ins RUNX RUNXT  AML M ins RUNX RUNXT  AML M ins RUNX RUNXT  MDS t RUNX  ZFPM  AML M t RUNX unidentified gene  pre B ALL t ETV RUNX  pre B ALL t ETV RUNX  pre B ALL t ETV RUNX  pre B ALL ins ETV RUNX  ALL  Acute lymphoblastic leukemia  AML  Acute myeloid leukemia  MDS  Myelodysplastic syndrome   Table   Structure of the fusion genes involving RUNX  Translocation Fusion genes In frame fusion genes retaining the RHD but not the TAD of RUNX t RUNX PRDM t RUNX AFF t RUNX MECOM t RUNX SHD t RUNX USP t RUNX RUNXT t RUNX TRPS t RUNX LPXN t RUNX MACROD t RUNX  CBFAT t RUNX PRDX t RUNX ZFPM Truncated fusion genes retaining part of the RHD but not the TAD of RUNX t RUNX ZNF t RUNX unknown gene Truncated fusion genes retaining the RHD but not the TAD of RUNX t RUNX PRDM t RUNX YTHDF t RUNX AFF t RUNX RPLL t RUNX FGA t RUNX LPXN t RUNX CPNE t RUNX AMP Fusion gene not retaining the RHD of RUNX inv USP RUNX In frame fusion gene retaining the RHD   the TAD of RUNX t ETV RUNX RHD  Runt homology domain  TAD  Transcription activation domain',\n",
       "       'The RUNX AML gene is the most frequent target for chromosomal translocation in leukemia  In addition  recent studies have demonstrated point mutations in the RUNX gene as another mode of genetic alteration in development of leukemia  Monoallelic germline mutations in RUNX result in familial platelet disorder predisposed to acute myelogenous leukemia   Sporadic point mutations are frequently found in three leukemia entities  AML M subtype  MDS AML  and secondary  MDS AML  Therapy related leukemias resulting from anticancer treatments are not uncommon  and the incidence of RUNX point mutations appears comparable to the incidence of the t AML M subtype and the inv AML MEo subtype  Half of the point mutations in M cases are biallelic  although the frequency varies with ethnicity  Most of the RUNX mutations are clustered in the Runt domain and result in defective DNA binding but active beta subunit binding  which is consistent with three dimensional structural findings and may explain the dominant inhibitory effects  Unlike the classical tumor suppressor genes requiring biallelic inactivation  haploinsufficient RUNX is apparently leukemogenic  However  RUNX abnormalities per se are insufficient to cause full blown leukemia  Intensive investigation of cooperating genetic alterations should elucidate leukemic mechanisms   Keywords  RUNX  AML  PEBP  point mutation  familial leukemia Top of page Introduction Since the discovery of the Philadelphia chromosome  t  in chronic myelogenous leukemia in   leukemia researchers and physicians have paid much attention to the investigation of chromosomal translocations and resultant chimeric genes  It was also the case for RUNX AML related leukemia   Miyoshi et al   identified AML as a gene located on the breakpoint of chromosomal translocation t in   Since then  leukemia research on AML RUNX has been exclusively concentrated on investigating t and resultant AML ETO  t  RUNX EVI  t  TEL RUNX  and inv  PEBPbeta MYH  In   Osato et al   and Song et al  reported a RUNX point mutation in sporadic and familial myeloid leukemia  respectively  Subsequent studies have identified point mutations in acute myeloid leukemia   predominantly in the M subtype  in different ethnic groups   Until recently  mutations were believed to cluster within the Runt domain  However  mutations in the C terminal region  outside the Runt domain  have also been identified  predominantly in MDS AML   Finally  RUNX point mutations turned out to occur with comparable frequency to the first mode of RUNX alterations  namely  t  inv  and t  in secondary MDS AML   Accordingly  the RUNX point mutation is fully recognized as the second mode of leukemogenesis in RUNX leukemia  In this review  I summarize the currently available cases of point mutations and discuss the leukemogenesis mediated by RUNX    status   Top of page Frequency and distribution of the point mutations The RUNX gene consists of  exons  and two promoters followed by the distinct initiation codon in exon  and exon   The proximal promoter driven transcript starts in exon  and encodes  amino acids    whereas the distal promoter driven transcript from exon  encodes  aa   Full length  contains an additional  aa instead of  aa from the full length  in its N terminal end  Thus  both full lengths share exons    B  and  and nearly the complete coding sequence  The Runt domain  a highly conserved  aa protein motif responsible for both DNA binding and heterodimerization with the beta subunit PEBPbeta CBFbeta spans aa   in full length  and is located on exons    All RUNX mutation screening studies covered exons    and most included intron sequences flanking the exon  A few studies screened exons  and  as well  and half of the studies did not cover exons   encoding the C terminal moiety where the transactivation domain spanning aa   is located  All types of human leukemia  except chronic lymphocytic leukemia   have been extensively investigated  In addition  a significant number of cell lines  including those established from solid tumors  as well as healthy volunteers were also included in the mutation screenings    A total of  cases out of the  tested had a RUNX alteration including a point mutation and gene deletion  Most of the nucleotide changes resulted in amino acid changes with clear functional defects as shown in Figure   However  some mutations failed to show any apparent functional alterations  and the nucleotide change may have been constitutive or leukemia specific  Nucleotide changes that do not lead to amino acid change  may generate an abnormal splicing donor or acceptor site  resulting in a nonfunctional protein  These enigmatic cases are listed as  variation  in Table   The numbers of mutations in Table  include variation cases  Most of the mutations are clustered within the Runt domain   N terminal region     Runt domain     C terminal region     The total number of mutations  differs  because the compiled mutants in Figure  include  cases from FPD AML  which are not listed in Table   Conversely  gene deletion is not included in Figure   while a biallelic case in Table  provides two distinct mutations simultaneously   Figure   Figure    Unfortunately we are unable to provide accessible alternative text for this  If you require assistance to access this image  please contact help nature com or the author Distribution and functional consequences of RUNX gene mutations  The mutations  and disease subtypes are compiled to the left of the corresponding amino acid indicated at the center  Amino acids in the Runt domain in dark green in the green bar representing exons are listed in a consecutive manner  while those outside the Runt domain are not  Missense  termination  and frameshift mutations are highlighted by yellow  dark gray  and gray  respectively  The functional consequences  DNA binding  beta subunit binding  and dominant negative effects are listed on the right    indicates function retention  Normal and hypernormal functions are in pink and red  respectively  Dominant negative effects are shown in light blue or dark blue in the right end column  The strong cases are in dark blue  Full figure and legend   Table    Frequency of mutations in the RUNX AML gene in various cancer patients and healthy volunteers  Table    Frequency of mutations in the RUNX  AML gene in various cancer patients and healthy volunteers   Unfortunately we are unable to provide accessible alternative text for this  If you require assistance to access this image  please contact help nature com or the author Full table  Top of page Functional consequence  haploinsufficiency Entire gene deletions and early truncated type mutations  were found with significant frequency  Most of the missense mutations also lost their DNA binding and transcriptional activities   Since the majority of cases are heterozygous  haploinsufficiency appears to be a basis for pathogenesis  This notion is most strongly supported by the whole gene deletion in FPD AML  RUNX    status is likely to provide the cells with a growth advantage to overcome  selective pressure  in the subject  However  if this were the only mechanism  mutations would be distributed throughout the gene without  hot spots   Therefore  clustering in the Runt domain suggests that a mechanism other than genuine haploinsufficiency is important for leukemogenesis   Top of page Dominant negative effect by the retained bold italic beta binding activity In the Runt domain  eight regions or  hot spots  were affected more than twice   Two different mechanisms are probably at least partly responsible  First  the primary sequence of particular sites seems to be preferably targeted by the mutation mechanism  R  R  and R are encoded by triplets containing CpG dinucleotides  which are extremely prone to transition type mutations by a mechanism facilitated by methylation of the cytidine residue   Second  certain mutation sites are considered to provide cells a stronger growth advantage under  selective pressure   Mapping of the missense mutations strongly supports this notion  Three regions recurrently affected by missense mutations exclusively correspond to three loop containing regions responsible for DNA binding  betaP loop  betaE  loop  and betaP loop    Consequently  these mutants are defective only in DNA binding  and not in packing or heterodimerization   which may explain the dominant negative effects of the RUNX mutants  In cotransfection assays  missense and some nonsense mutants showed inhibitory effects towards transactivation mediated by wild type RUNX  These mutants bound to PEBPbeta CBFbeta more efficiently than wild type RUNX   In addition  PEBPbeta protects RUNX from ubiquitin proteasome mediated degradation   Thus  preferential binding of mutant RUNX proteins to PEBPbeta may in turn increase protein half life and hence further augment the dominant negative effects  In the right end column of Figure   a general correlation can be seen between the retained beta binding activity and the inhibitory potential of each mutant  This correlation is more clearly indicated by the comparison of the distribution of missense mutations between RUNX and RUNX   Germline mutation of RUNX causes the bone disease cleidocranial dysplasia    The missense mutations of RUNX distributed throughout the Runt domain affected the overall folding architecture in the cases marked by asterisks in Figure   resulting in impairment of both DNA  binding and heterodimerization activity   The mutations underlined in Figure  showed defects in beta binding alone  In clear contrast  no such mutations resulting in defective packing or beta binding have been found in RUNX so far    but functional analyses have not yet been performed  No suspected case with defect in beta binding alone has been reported   Conversely  mutations mapping to the first DNA interacting region  betaP loop   are seen in RUNX but not in RUNX    These RUNX mutants  RC  KN  and KE showed strong affinity for the beta subunit and hence a strong dominant negative effect  suggesting that haploinsufficiency is a dominant mechanism in RUNX mutants  whereas a dominant negative effect is important in RUNX mutants  Sequestration of the beta subunit may be one important mechanism of the dominant negative effect caused by the RUNX mutants   Figure   Figure    Unfortunately we are unable to provide accessible alternative text for this  If you require assistance to access this image  please contact help nature com or the author Structural mappings of the missense mutations detected in leukemia or the bone disease CCD  The RUNX mutants in leukemia  are mapped exclusively to the three regions involved in DNA binding  whereas the RUNX mutations in CCD  are distributed all over the Runt domain except for the loop   Mutations associated with packing  and beta binding  defects are found only in CCD  Adapted from Nagata et al    Full figure and legend   Top of page The lower the RUNX activity  the higher the leukemogenicity How does the dominant negative effect mentioned above relate to leukemogenesis  Although the limited number of individuals and minimal clinical information evaluated prohibit a robust conclusion  families with mutations acting simply via haploinsufficiency have a lower incidence of leukemia than families with mutations acting in a dominant negative fashion  In two of the largest FPD AML pedigrees  leukemia occurred at a markedly higher rate in the family with the strong dominant negative KE mutation  compared to the pedigree with a complete deletion of RUNX     Thus  low RUNX activity appears to correlate with high leukemogenicity  The hypothesis may be expanded to some extent to leukemogenesis induced by chimeric genes  The majority of current findings indicate that a major function of chimeric genes is a transdominant inhibition of wild type RUNX  resulting in severely decreased RUNX activity in affected cells  Currently  more than  partner genes fused with RUNX have been identified in chromosomal translocations  Although each chimeric gene may have its own leukemogenic mechanism  RUNX insufficiency may be a common underlying mechanism in RUNX leukemias   Table    RUNX mutations in familial leukemia  FPD AML  Table    RUNX mutations in familial leukemia  FPD  AML   Unfortunately we are unable to provide accessible alternative text for this  If you require assistance to access this image  please contact help nature com or the author Full table  Acquired trisomy  underscores this notion  RUNX    mutant alleles appear to be duplicated frequently  This finding may also lend supporting evidence to the hypothesis that reduced RUNX activity increases leukemogenic status  This argument will be discussed later   Top of page Mutation status   disease phenotype in sporadic leukemias Biallelic mutation and AML M subtype RUNX point mutations were most frequently found in the AML M subtype   In addition   of the   M cases showed the biallelic type of mutation   Conversely  only two of the  RUNX mutation cases in other leukemia subtypes were the biallelic type  Therefore  it is clear that RUNX point mutations of the biallelic type are tightly associated with the AML M subtype  Mice heterozygous for the RUNX MTG or PEBPbeta MYH chimeric genes showed defects in definitive hematopoiesis similar to Runx    mice  suggesting that the chimeric genes can act as strong transdominant inhibitors of the remaining normal Runx allele   Therefore  the leukemogenic potential of biallelic RUNX mutants is considered equivalent to those of chimeric genes  On the other hand  t  inv  and biallelic point mutations are related to the distinct M  MEo  and M subtypes of myeloid leukemias  respectively  The different disease phenotypes are quite interesting  The contribution of each partner gene  thought to be considerable  may also be explained by the different residual activities of RUNX in the affected adult hematopoietic cells  Conditional Runx knockout mice showed apparent defects in hematopoiesis   whereas hematopoiesis in conditional heterozygous RUNX ETO knockin mice was nearly normal   suggesting that Runx    induces more severe effects than the RUNX ETO chimeric gene in adult hematopoiesis  The M subtype shows minimally differentiated morphology  while the M subtype is characteristically well differentiated  Therefore  the differing residual activities of RUNX seem to be important  In addition  influence on the activities of other RUNX family genes must be considered because RUNX MTG or PEBPbeta  MYH chimeric genes could inhibit RUNX and RUNX as well  while RUNX    status is a defect in RUNX alone  Careful study will resolve this intriguing issue  Table    RUNX mutations in AML M subtype  Table    RUNX mutations in AML M subtype   Unfortunately we are unable to provide accessible alternative text for this  If you require assistance to access this image  please contact help nature com or the author Full table  Of note  the biallelic type mutation is strictly confined to the status with two copies of RUNX  Biallelic mutation in trisomy  is not associated with the AML M subtype  but is associated with myeloid malignancies other than M  This phenotypic difference will be discussed later   Entire distribution including the C terminal moiety and MDS AML secondary leukemia The RUNX point mutations are also frequently found in MDS and subsequent overt leukemia    The mutations are characteristically distributed  as they were also detected in the C terminal region which is rarely targeted in other types of leukemias   Since this distribution was only recently identified  C terminal mutations were overlooked in a number of previous studies  Half of the studies did not investigate exons B and  in the C terminal region  With regard to MDS AML  the incidence covering both exons   and exons   were evaluated less frequently than those of exons   alone   Thus  C terminal mutations may be underestimated in MDS AML  However  C terminal mutations still seem rare in AML  since C terminal mutations were not identified in the study with a relatively large number of cases  Therefore  RUNX point mutations including the C terminal moiety appear to be found predominantly in MDS AML  Table    RUNX mutations in MDS  Table    RUNX mutations in MDS   Unfortunately we are unable to provide accessible alternative text for this  If you require assistance to access this image  please contact help nature com or the author Full table  Harada et al   from Hiroshima  one of the two cities suffered from atomic bomb  reported two further implications of RUNX mutations in secondary MDS or AML   including therapy related and atomic bomb related MDS AML  Firstly  the relationship to radiation seems important in terms of the social issue  In addition to the victims in the above two cities  an estimated   US soldiers and a significant number of local residents and passengers have been irradiated in the aboveground nuclear weapon test sites at Nevada  Bikini atoll  and others in the late s to s  Many of these people are thought to have succumbed to MDS AML caused by RUNX point mutation  Secondly  the involvement in therapy related MDS AML seems to have more general impact  With the improvement of treatments for primary cancer  the incidence of therapy related MDS AML continues to increase  Therapy related MDS AML accounts for roughly    of all new cases of MDS and AML in standard hospitals  Therefore  the incidence of secondary MDS AML with RUNX mutation is comparable to those of t AML M subtype and inv AML MEo subtype   Considering its poor prognosis  clinical implication of RUNX point mutation in secondary MDS AML would be rather more significant than we expected   Table    Estimated frequency of leukemia with RUNX point mutation  Table    Estimated frequency of leukemia with RUNX point mutation   Unfortunately we are unable to provide accessible alternative text for this  If you require assistance to access this image  please contact help nature com or the author Full table  Leukemic incidence rates were based on several textbooks and public databases from CDC  WHO  and National Cancer Institute   CLL accounts for     which may explain the relatively low incidences of t  inv  and t   Acquired  trisomy  and non MO myeloid malignancy Preudhomme et al   first reported the relationship between trisomy  and RUNX point mutation   Sporadic cases of myeloid but not lymphoid malignancy with trisomy  were associated with a high frequency of RUNX point mutations  Taketani et al   subsequently surveyed RUNX mutation in congenital cases of trisomy   that is  the Down syndrome related hematopoietic diseases  However  point mutation in congenital cases appears to be rare except for the enigmatic HN variation  which has nearly normal function  Allelic analysis of sporadic cases with acquired trisomy  showed the following combination    mutant vs   wild type  The co occurrence of mutation and trisomy  should be associated with the allelic combination with one mutant and two WT combination  However  this combination has never been found  These findings suggest that trisomy  increases the copy number of the mutant allele  which already exists in the status of two copies of RUNX  In other words  trisomy  is thought to be a secondary change to RUNX    status  Trisomy  as a second hit will be discussed later  Table    Trisomy  and RUNX mutation  Table    Trisomy  and RUNX mutation   Unfortunately we are unable to provide accessible alternative text for this  If you require assistance to access this image  please contact help nature com or the author Full table  As mentioned earlier  biallelic RUNX mutations are tightly liked to the AML M subtype  However  interestingly  acquired   carrying a RUNX point mutation has never been observed in the AML M subtype  even in cases with biallelic mutations   This  discrepancy  is not explained by the differing residual activities of RUNX  as half of the M cases with RUNX mutation are monoallelic  Leukemic phenotype may be determined by an as yet undetermined mechanism   Top of page Familial leukemia  FPD AML Heterozygous germline mutation in the RUNX gene causes familial leukemia  familial platelet disorder with predisposition to acute myelogenous leukemia   A total of  pedigrees have been identified so far  mainly in individuals of European descent   FPD alone  before the onset of AML  is not fatal but significantly inhibits blood clotting  which could be life threatening after surgery  injury  dental treatment  or childbirth  The incidence of leukemia among affected individuals varies from  to     and is highest in familial leukemia syndromes including Fanconi anemia  Bloom syndrome  and Kostman syndrome  The affected subjects seem to develop myeloid malignancies throughout their life span  but not later in life as described in some previous reports  Leukemia developed before the age of  years in four of  cases  These features of FPD AML were compared with a similar dominant hereditary disease  Huntington disease   The subjects of HD are healthy until disease onset at   years of age and die   years later  In addition to the low penetrance and or low severity of symptoms  late onset is a key factor for hereditary disease  since patients often have children before disease symptoms appear  On the other hands  FPD AML seems detrimental to the subjects in any factors of hereditary disease  How can such a pedigree full of drawbacks exist  FPD AML may persist as a disease due to the easy mutability of RUNX  As discussed before  RUNX indeed contains easy mutable primary sequences  triplets containing CpG dinucleotides extremely prone to transition type mutations facilitated by methylation of the cytidine residue   This mechanism could work particularly well in germ cells  The FPD AML pedigree may readily disappear in the human population  while a  de novo  pedigree could be continually generated with comparable frequency  Consistent with this argument  of CCD cases caused by RUNX mutation  approximately half are the  de novo  and not the familial type   Irradiation readily generates CCD like phenotypes in mice due to Runx alterations  RUNX shares the above noted easy mutable sequences  In the case of RUNX  healthy British subjects have two kinds of variations  LS and Ssyn with  and   frequency  respectively   Japanese subjects also have several recurrent enigmatic variations  GR  Isyn  and HN  Interestingly  GR was identified in both leukemia patients and healthy volunteers   These variations  probably including polymorphisms  might reflect the above noted easy mutability of RUNX   As mentioned above  FPD AML is extremely rare worldwide  Therefore  this clinical entity had been overlooked by physicians until recently  Bone marrow transplantaion was carried out from an affected sibling with mild FPD symptoms to a patient suffering from AML   The recipient succumbed to leukemia again  clearly from the hematopoietic cells derived from the donor   Top of page Genetic alterations cooperating with RUNX    status As mentioned earlier  RUNX    status is frequently found in the AML M subtype  Thus  RUNX may appear to act as a classical tumor suppressor gene  requiring inactivation of two alleles according to Knudson s two hits theory  However  biallelic inactivation is not required in all cases of leukemogenesis  in fact  the overall frequency of biallelic cases is relatively low   Biallelic cases of FPD AML have never been reported  Therefore  RUNX    per se appears to be leukemogenic  The dominant negative effect associated with missense mutations might play a role in leukemogenesis  but not to the same extent as gene inactivation  In any case  heterozygous mutation appears to be a step towards leukemogenesis  Therefore  a mutation in the wild type RUNX on the remaining allele is considered to be a second hit in addition to RUNX    status  Furthermore  acquired but not congenital trisomy  is also thought to be a second hit  As discussed earlier  the link between   and RUNX mutation is interpreted as follows  a mutation takes place in the two copy status and the mutant allele is duplicated in accordance with    In both RUNX    and   cases  acquirement of another mutant allele is therefore thought to be an additional genetic alteration cooperating with RUNX    status    Figure   Figure    Unfortunately we are unable to provide accessible alternative text for this  If you require assistance to access this image  please contact help nature com or the author Leukemogenesis from Runx     Biallelic mutations of RUNX  are tightly liked to the AML M subtype  However  acquired   carrying a RUNX point mutation  has never been observed in M  despite cases with biallelic mutations  Mutation likely takes place in the two copy status  and the mutant allele is duplicated in accordance with    resulting in myeloid malignancies other than M  FLT  mutations were frequently found in M cases with RUNX point mutations  These genetic alterations are mapped based on such observations  Full figure and legend   In line with    cytogenetics helps to identify second hits  Patients with RUNX point mutations show the following concurrent abnormalities   cases showed              Loss of chromosome  is suspected to be a strong predisposing factor for the second hit in RUNX      FLT  mutations were frequently observed even in monoallelic RUNX mutations in M patients    Hence  the FLT  mutation is also a cooperating genetic alteration   FLT  mutation results in a constitutively active form known to stimulate cell proliferation  Therefore  differentiation blockade by RUNX alteration and growth stimulation by FLT mutation would synergistically contribute to a substantial fraction of AML M subtype cases  Interesting enough  FLT  mutations are known to be rare in the other RUNX leukemias  t and inv  Certain RUNX leukemias seem to cooperate with distinct second hits  Elucidation of this mechanism might shed light on development of RUNX leukemias   Top of page Current problems Region for mutation screening As discussed previously  half of the screening studies did not cover C terminal exons B and  and therefore overlooked C terminal mutations in MDS AML  C terminal screening is unlikely to reveal information about AML but is recommended  Likewise  exons  and  might be affected in particular types of leukemia  Theoretically  the promoter and untranslated regions could also be targeted  Variations Significant numbers of variations are reported  as shown in Table  and Figure   Functional analyses have not yet been performed in all the cases  Each variation must be studied to determine if it is a mutation or a polymorphism  Analyses of polymorphisms might help to characterize susceptibility to leukemia  Dominant negative effect by C terminal mutants in MDS AML In the reporter assay  all C terminal mutants found in MDS AML showed inhibitory effects in a transdominant manner  like the missense mutants in the Runt domain   However  similar C terminal RUNX mutations in CCD patients retain partial transactivation activity and show much milder phenotypes compared to the standard CCD phenotypes  which are caused by simple gene deletions   This discrepancy may be due to the distinct cell context or promoter used for the assay  However  a correlation with MDS but not AML might indicate that C terminal mutants have milder inhibitory effects than Runt domain mutants  The biological assay  but not the reporter assay  may address this issue  Cooperative genetic alterations FLT  mutation and   are thought to be second hits in MDS AML with RUNX mutations  High frequency of chromosome  loss suggests inactivation of the unknown gene on chromosome  plays a synergistic role with RUNX mutations  However  the overall frequency of these mutations does not account for all cases  Furthermore  to induce a leukemia  three to five genetic alterations are considered to be required based on current evidence  Therefore  a number of genetic alterations must be identified to understand the whole leukemogenic mechanism of RUNX leukemia  My laboratory recently established a mouse system that appears to be quite useful  Myeloid leukemia prone mice mimicking FPD AML were generated by backcrossing Runx    onto the BXH strain  in which retroviral insertional mutagenesis occurs spontaneously with the eventual development of myeloid leukemia in over   of the cases   The proviral integration sites in the leukemias provide powerful genetic tags for disease gene identification  Therefore  collaborating oncogenes with Runx    would be readily detected  Loss of function of RUNX is widely believed to be the common underlying mechanism of RUNX leukemia  RUNX    status is considered to be the minimum alteration required for several forms of RUNX leukemias  Therefore  the mouse model for FPD AML mentioned above should be useful for searching for cooperative genetic alterations in the other RUNX leukemias  t and inv  These genetic alterations identified by the retroviral tagging system might explain the leukemogenic mechanisms not only of RUNX point mutations but also of other RUNX leukemias  Top of page Future directions Why is RUNX    status leukemogenic  This question seems to be the most fundamental question not only for RUNX point mutation but also all RUNX leukemias  Determining the means by which RUNX    is leukemogenic may not be possible by the currently used approaches to evaluate differentiation  proliferation  and apoptosis at the cellular level and transcriptional control at the molecular level  A novel approach is warranted  For example  alteration of  stemness  should be investigated in RUNX    status  Stemness could provide cells with a time window sufficient for sequential genetic alterations  Intrinsic molecular functions relevant to leukemogenesis  other than transcriptional control  might be identified by novel approaches',\n",
       "       'The most frequent mutations associated with leukemia are recurrent somatic chromosomal translocations or inversions  many of which involve the polyomavirus enhancer binding protein or core binding factor transcriptional regulation complex   Several translocations involve the α subunit of this complex  the RUNX gene  on chromosome q    t  and t   Additionally  the β subunit of the complex  PEBPβ also called CBFβ  is disrupted in inv  An abundance of evidence points to the existence of genes that predispose to hematologic malignancies  However  large multiple generation families with hematologic malignancies alone are rare  Only  loci for familial hematologic malignancies have been identified to date   on chromosome q  and the other on q   These loci contain RUNX andPEBPβ CBFβ  respectively Studies of families that demonstrate single gene inheritance for leukemia predisposition should help to identify the genes and mechanisms involved in the first steps of leukemia development  The autosomal dominant familial platelet disorder  AML  is a good model to validate this hypothesis because  in addition to developing thrombocytopenia  patients show a propensity for progression to myelodysplasia and acute myeloid leukemia   Affected individuals within the same family may present with variable clinical severity at varying ages  FPD AML was linked to q      including the region that contains RUNX  and germline RUNX mutations were subsequently identified in  pedigrees Heterozygous missense mutations and biallelic nonsense or frameshift mutations in RUNX were also identified in sporadic leukemias  and other point mutations have been characterized in patients with M AML and various myeloid malignancies  including some with acquired trisomy and tetrasomy of chromosome   While missense mutations abolish the DNA binding and transactivation activities of RUNX  they do not affect heterodimerization with PEBPβ CBFβ  In addition  these mutant proteins inhibit wild type RUNX in cotransfection studies  The evolutionarily conserved  amino acid Runt domain  present in most of RUNX isoforms  is involved in DNA binding and heterodimerization with PEBPβ CBFβ  Heterodimerization of RUNX with PEBPβ CBFβ promotes DNA binding by stabilizing the interaction of the complex with the DNA   All RUNX point mutations described in FPD AML patients and most RUNX point mutations in patients with sporadic leukemias  are within the Runt domain and alter amino acids essential for DNA binding and heterodimerization   Longer RUNX proteins also contain a transactivation domain and domains involved in self regulation  protein protein interaction  and cellular localization  RUNX regulates the activity of several important hematopoietic genes  such as macrophage colony stimulating factor  receptor  T cell receptor α and β  granulocyte macrophage  CSF myeloperoxidase  and neutrophil elastase by binding to a core sequence  found in their promoters or enhancers It has been proposed that haploinsufficiency for RUNX is responsible for FPD AML  However  hyperactivating  inhibitory  and loss of function RUNX point mutations have all been reported in sporadic leukemia  This diversity in the mechanisms of pathogenesis of sporadic AML RUNX mutations suggested that FPD AML RUNX mutations warranted further functional studies  Here we describe  new RUNX mutations in FPD AML and potential mechanisms involved in the pathogenesis of the disease based on in vitro studies of  RUNX FPD AML mutations Materials and methodsSample collection and genomic DNA isolationOur FPD AML pedigrees are shown in Figure  Pedigree  has been extensively described  Fourteen members of the family have a hemorrhagic diathesis characterized by a long bleeding time  abnormal platelet aggregation  and decreased numbers of platelet dense bodies  Among the affected individuals  there were  documented cases of myeloblastic leukemia   documented cases of myelomonoblastic leukemia  and  cases of leukemia by history Fig   Download figureOpen in new tabDownload powerpointFig    The FPD AML pedigrees unique to this study All  pedigrees show linkage to FPD AML locus   Half filled symbols represent individuals who had a bleeding disorder and  where studied  a platelet defect as well  and completely filled symbols represent affected individuals who developed leukemia Pedigree   has  affected individuals from  generations with a clinical history of prolonged bleeding  Constitutional thrombocytopenia was found in  members   of whom had extensive platelet evaluations documenting aspirinlike aggregation defects and dense granule abnormality demonstrable by electron microscopic morphology and quinacrine fluorescence  Three individuals developed AML  and a fourth was said to have died of  pernicious anemia  Pedigree  consists of  affected individuals from  generations of a family with a bleeding disorder characterized by impaired platelet aggregation associated with decreased numbers and contents of both platelet dense and α granules  as well as a unique platelet phospholipid defect and decreased α  adrenergic receptors   The platelet counts in this family are at or slightly below the lower limit of normal   Two members of the family developed AML  and another was reported to have died of leukemia Blood samples were obtained from healthy and affected individuals after obtaining informed consent and in accordance with institutional guidelines for human subjects  DNA was isolated from blood leukocytes using standard protocols Linkage analysisGenotype information was generated with polymerase chain reaction  amplification of chromosome  microsatellite markers from the Genome Database   Linkage analysis was as previously described   At least  microsatellite repeat polymorphisms in the FPD AML critical region of chromosome q \\u2009 were genotyped in each family Mutation analysesAll  exons of RUNX were amplified from genomic DNA using primers designed to the flanking region of each exon  Single strand conformational polymorphism analysis was performed on PCR fragments of less than  base pairs using a Genephor apparatus and the GeneGel   kit   Exons showing conformational changes were directly sequenced in both directions using BigDye terminator cycle sequencing  and analyzed on an ABI sequencer RT PCRFirst strand complementary DNA  was synthesized from patient  and control RNA using Superscript II reverse transcriptase  PCR was performed using primers AMLRTF  in exon  of AMLb and AMLRTR  in exon  as reverse primer on these double stranded cDNAs  cloned into a TA vector  and sequenced  A cryptic splice site was observed in one clone  Primers covering the cryptic splice junction of exon  and   and within the nucleotides of exon  excluded from the mutant transcript  were designed and used with AMLRTR to amplify a PCR product on patients  cDNA  no mutant transcript was observed in control cDNA Fig   Download figureOpen in new tabDownload powerpointFig    Mutation analysis in FPD AML pedigrees Electropherograms from affected and control individuals showing the A G substitution leading to the KE missense mutation in pedigree    the C A substitution resulting in a nonsense mutation YX in pedigree    and the deletion of an A at the   position of the splice donor site of intron  in pedigree    Arrows indicate the point of the mutations   Schematic representation of the cryptic splice site within exon  used in the mutated transcript in pedigree  and the RT PCRs used to analyze the splicing  A  B  and C represent forward primers used in the RT PCR experiment  Primer A is specific to the mutated form  and primer B is specific to the wild type form  Primer C was used as control  The same reverse primer was used in each reaction  The  nucleotides of exon  excluded from the mutant transcript are indicated by the black box   RT PCR on patient and control RNA as described in panel D  The primer specific to the mutated form does not show any amplification on control RNA   Sequence analysis of the mutant exon  exon  junction Electrophoretic mobility shift assayAmino acids  to  of RUNX  including the Runt domain  were expressed in Escherichia coli  purified  and subjected to electrophoretic mobility shift assay  essentially as described previously  The wild type expression plasmid was previously described  and mutant RUNX plasmids were obtained by site directed mutagenesis on wild type pQE RUNX using Strategene s  quick change  method according to the manufacturer s instruction manual   For KfsX an alternative vector  pQE  was used instead to express it in fusion with a more bulky N terminal appendage containing hexahistidines and dihydrofolate reductase  Mutant constructs were confirmed by sequencing Affinity assay of RUNX PEBPβ CBFβ associationThe heterodimerization activity of RUNX mutants impaired in DNA binding was assayed by MagExtractor kit   Briefly  the hexahistidine tagged RUNX fragment  was incubated with tagless PEBPβ CBFβ  and mixed with nickel coated magnetic beads  The mixture was successively washed with buffers containing  mM imidazole and eluted by the buffer supplied by the manufacturer  Proteins in each fraction were analyzed as previously described Subcellular localizationFor functional studies of mutated RUNX proteins  the wild type RUNX sequence  was inserted into the pEF BOS mammalian expression plasmid  Construction of RUNX mutants was carried out using the megaprimers method  and the resultant plasmids were transfected into NIHT or REF  cells using a nonliposomal transfection reagent FuGENE   Immunofluorescence labeling of RUNX and PEBPβCBFβ and microscopy were as previously described  For each transfection  cells were visualized Transactivation assaysThe luciferase reporter plasmid  and the effector plasmid   containing the  amino acid isoform of RUNX with the mutations to be studied  were transfected at a fixed ratio  into U  HL  and Jurkat cells  Cell extracts were prepared  hours after transfection and assayed essentially as described  To study the inhibitory effect of mutant RUNX proteins on wild type RUNX reporter gene activation    μg pM CSF R luc and   to   μg pEF constructs expressing either RUNX with or without mutations or PEBPβ MYH were cotransfected into U  HL  and Jurkat cells  The total amount of DNA transfected was kept constant  by supplementing appropriate amounts of the backbone pEF plasmid  thereby avoiding potential artifacts due to unbalanced DNA dosages Partial rescue from repression was obtained by transfecting varying doses of PEBPβCBFβ with a repressive dose of wild type and mutant proteins ResultsLinkage and mutation analyses of the  new pedigreesEach of the  families demonstrated features typical of FPD AML  A candidate region genetic linkage analysis provided maximum  point log odds ratio scores of       and    with markers DS in pedigree   IFNAR in pedigree   and DS in pedigree   Extended haplotype analysis of each family indicated an approximate  megabase common nonrecombinant interval on human chromosome q  containing RUNX in affected individuals Pedigree  showed an A G substitution in exon  resulting in a missense mutation  KE   The mutation segregates with the disorder in the family and was absent in  unrelated control chromosomes  Pedigree  showed a one base deletion in the splice donor site of intron   IVS\\u2009 \\u2009delA   The mutation segregates with the disorder in the family and was absent in  unrelated control chromosomes  Reverse transcriptase  PCR on RNA from affected individuals showed the use of a cryptic donor splice site  not used in control RNA   nucleotides upstream of the normal splice site   The novel transcript generated by the use of the cryptic splice site results in a frameshift after amino acid   addition of  unrelated residues  and termination at codon    Pedigree  showed a C A substitution in exon B resulting in a nonsense mutation  YX   The mutation segregated with the disease in all family members tested Effects of RUNX FPD AML mutations on DNA binding and heterodimerization activities of the Runt domainWe examined the function of the Runt domain for  new  and  described  FPD AML point mutations  On EMSA analyses  RQ  RX  and RX alone showed barely detectable DNA binding but produced a supershift band of increased intensity in the presence of PEBPβ CBFβ  indicating that heterodimerization of these mutants with PEBPβ CBFβ still occurs   KE  RfsX  RQ  and KfsX showed no detectable DNA binding in the presence or absence of PEBPβ CBFβ  An affinity assay of these mutants for heterodimerization with PEBPβ CBFβ revealed that both missense mutants retained the ability to heterodimerize whereas both frameshift mutants did not  Fig   Download figureOpen in new tabDownload powerpointFig    Alterations in the DNA binding and heterodimerization activities of the Runt domain of FPD AML RUNX mutations  Partial RUNX proteins  were subjected to EMSA in the presence  and absence  of PEBPβ CBFβ  WT indicates wild type RUNX  The position of the RUNX and RUNX CBFβ complexes with the DNA are indicated   Affinity assay of the indicated partial RUNX proteins with PEBPβ CBFβ  M indicates molecular weight marker as indicated both to the left and right  β  PEBPβ CBFβ  A  input RUNX protein  W  unbound proteins in washed fractions  E  bound proteins eluted at  mM imidazole  The bands marked with arrows indicate the β subunits associated with RUNX proteins Subcellular localization of the FPD AML RUNX mutants and colocalization with PEBPβ CBFβIn RUNX a nuclear localization signal  is present at amino acids  to  at the end of the Runt domain  RUNX wild type protein is localized in the nucleus with a diffuse pattern and is excluded from the nucleoli   Transfection of mutant full length cDNAs into NIHT cells and immunostaining the expressed RUNX proteins revealed that the frameshift  and nonsense  mutants are localized almost exclusively in the cytoplasm  Consistent with the importance of basic amino acids in the NLS  RQ and RQ  showed cytoplasmic localization in   and   of observed cells  respectively  Fig   Download figureOpen in new tabDownload powerpointFig    Subcellular localization of mutant RUNX proteins and colocalization with PEBPβ CBFβ  Subcellular localization of the indicated RUNX proteins was detected by immunofluorescence staining with anti αB  A schematic of the RUNX protein is shown with the Runt and transactivation domains indicated  The sequence of the NLS is indicated   Double staining for mutant RUNX and PEBPβ CBFβ reveals that the subunits are colocalized in the cytoplasm within speckled dotlike structures for the missense mutants and a diffuse staining pattern for the frameshift and nonsense mutations  All examples in this figure are in NIHT cells  Original magnification     While the KE and RQ mutations are not within the known NLS  both mutant proteins showed reduced nuclear localization  KE showed cytoplasmic localization in   of transfected cells  in contrast to the normal nuclear localization seen in KN   This observation may suggest that the KE mutation  results in greater loss of function than the KN mutation   The RQ mutation is at a site involved in DNA binding and was expected to affect DNA binding rather than nuclear localization  However  cytoplasmic localization was observed in   of cells  The observed defect in nuclear localization in KE and RQ is consistent with earlier studies and may point to the existence of an additional unidentified N terminal domain critical for nuclear localization In contrast to RUNX  PEBPβ CBFβ is normally localized in the cytoplasm before heterodimerization and can enter the nucleus only with RUNX  However  the regulation of the heterodimerization remains unknown  and it has been shown that both proteins may have different cellular localizations when transfected in the same cells  The missense mutants showed speckled dotlike distribution in the cytoplasm  markedly different from the diffuse pattern of the truncated or nonsense mutants  Double staining for the α and β subunits revealed that they colocalized within these speckled patterns  Transactivation abilities of the FPD AML RUNX mutantsThe transactivation potential of the mutant RUNX proteins was measured using a reporter construct based on the M CSF receptor promoter as a myeloid specific RUNX target  Wild type RUNX demonstrated more than  fold transactivation compared with the FPD AML mutants  which showed no significant transactivation with or without PEBPβ CBFβ   Despite the retention of DNA binding activity and C terminal activation domains  RQ failed to transactivate  Several mutants showed persistently lower luciferase activities than the mock transfected plasmid  indicating that these mutants could hinder the function of endogenous RUNX in U cells  To allow measurement of inhibition of the wild type transactivational activities caused by the mutant proteins  the reporter construct and a fixed amount of nonsaturating wild type RUNX plasmid were cotransfected with varying amounts of FPD AML mutant plasmids encoding mutant proteins with DNA and or PEBPβ CBFβ binding activity  PEBPβ CBFβ MYH and KN repressed the M CSF receptor promoter as previously reported   With the exception of RX  all of the mutants showed varying degrees of interference with the wild type protein function in all cell lines tested   Most notably  KE and RQ showed potent inhibition similar to that of PEBPβ CBFβ MYH   The results of cotransfection reporter assays using frameshift mutants are not reproducible within or between cell lines Fig   Download figureOpen in new tabDownload powerpointFig    Transactivation of the M CSF receptor promoter by exogenously expressed RUNX proteins  Cells were transfected with a luciferase reporter plasmid and the indicated RUNX expression constructs  Luciferase activities were measured and presented as the fold increase relative to the control transfected with the backbone expression vector   The wild type RUNX and missense RUNX mutants were coexpressed in varying doses as indicated  Luciferase activities are expressed as fold changes relative to the activity observed at the standard dose  of the wild type RUNX alone   PEBPβ CBFβ was transfected in varying doses with a repressive dose of wild type and mutant RUNX proteins  In panels A C  each value represents the mean of  separate experiments  The luciferase activity of the y axis in panel C is shown as relative luciferase units   Deviations of the measurements are given by thin vertical bars  The chimeric protein PEBPβ CBFβ MYH and the sporadic mutation KN were used as controls in each experiment  All examples in this figure are in U cells Because all inhibitory mutants maintained the ability to heterodimerize and all colocalized with PEBPβ CBFβ  sequestration of PEBPβ CBFβ by mutant RUNX proteins may be the major inhibitory mechanism  To test this hypothesis  PEBPβ CBFβ was cotransfected in increasing amounts together with inhibitory amounts of mutant RUNX  resulting in the restoration of near  normal reporter gene activity  DiscussionFive RUNX point mutations and a deletion of the entire RUNX gene have previously been described in FPD AML  Here we report  new RUNX point mutations associated with the disease  YX  in pedigree   is at the beginning of the transactivating domain and removes a part of the negative regulatory region for DNA binding   It is the first described familial RUNX mutation outside the Runt domain  As such  the studies described in this paper are not appropriate to test the functional implications of this mutation  Functional analyses of  mutants  were performed to investigate the mechanisms that contribute to FPD AML  particularly the propensity to develop AML  The capacity to bind DNA was reduced or abolished in all mutant proteins as expected from the sites of substitutions in the Runt domain  The missense and nonsense mutant proteins  retained the capacity to heterodimerize with PEBPβ CBFβ  The frameshift mutant proteins  which lack a substantial part of the Runt domain  failed to heterodimerize  as has been reported in a sporadic RUNX AML nonsense mutation SX  These observations are consistent with the known crystal structure and predicted binding domain of mouse Runx  The positions of the  missense mutations are not critical for the structural integrity of the dimerization site  whereas the frameshift mutant proteins are predicted to fold abnormally and lack Runt domain residues required for heterodimerization Do the mechanisms of pathogenesis of sporadic leukemias provide any clues to the mechanisms of pathogenesis in FPD AML  In many sporadic hematologic malignancies  the same chromosomal rearrangements consistently appear  supporting the hypothesis that these events are a prerequisite for tumor induction  The somatic mutations acquired in leukemia by translocations and inversions are heterozygous in affected cells  normal copies of the translocated genes are still present  Consequently  the mechanism of disease pathogenesis may include  a gain of function by chimeric fusion proteins   haploinsufficiency   a dominant negative effect of the chimeric fusion proteins  and or  a combination of these mechanisms  However  it is unlikely that the mutations described here have a gain of function activity Haploinsufficiency of RUNX proteinsThe importance of gene dosage to the normal function of CBF transcriptional regulators is seen in the autosomal dominant human bone disorder  cleidocranial dysplasia  caused by mutations inRUNX   Runx    mice show skeletal defects similar to those in cleidocranial dysplasia  RUNX mutations may be heterozygous missense mutations  insertions  and deletions in either the Runt DNA binding domain or more C terminal domains responsible for transactivation  RUNX point mutations are predicted to affect the folding and stability of RUNX  resulting in haploinsufficiency of RUNX and defects in bone formation   Complete deletion of RUNX results in haploinsufficiency in FPD AML  The  FPD AML splice site mutations resulting in frameshifts RfsX and KfsX cause loss of DNA binding  heterodimerization  nuclear localization  and transactivation functions and are also likely to act through simple haploinsufficiency Dominant negative effects of mutant RUNX proteins Translocations involving RUNX produce chimeric proteins  most notably RUNX AML ETO in AML t and TEL RUNX AML in ALL t  These chimeric proteins both retain the entire Runt domain and have been shown to interfere with transactivation by the normal RUNX in a dominant negative manner   RUNX AML ETO and related fusion proteins have been shown to form complexes with PEBPβ CBFβ and other nuclear proteins more efficiently than wild type RUNX   Moreover  naturally occurring isoforms of RUNX proteins that contain only the Runt domain can also suppress transactivation by full length RUNX  Deletion of a negative regulatory domain for heterodimerization in the C terminal region of full length RUNX  results in more efficient heterodimer formation  Thus  chimeric RUNX proteins resulting from translocations and the isoforms of RUNX without the transactivation domain may act in the same manner by dimerizing with PEBPβ CBFβ more efficiently than the wild type Cotransfection of equal amounts of wild type and mutant plasmids closely mimics the in vivo situation of one normal and one mutant allele  All FPD AML RUNX missense and nonsense mutants studied  except RX  show a decrease in their transactivation capacities and an inhibitory effect on wild type RUNX upon cotransfection and thus also seem to act as dominant negative inhibitors  Both the missense and nonsense mutants still interact with PEBPβ CBFβ  and both nonsense mutants studied here lack the negative regulatory domain for heterodimerization  RQ and the  nonsense mutant proteins maintain some degree of DNA binding  although the DNA binding of the nonsense mutations may be irrelevant in vivo due to their cytoplasmic localization resulting from a direct disruption of the NLS  The missense mutants  while appropriately translocated to the nucleus most of the time  also show some cytoplasmic staining  The inhibitory nature of these mutants is likely to be due to competition with wild type for dimerization with PEBPβ CBFβ and its sequestration  In the case of RX  this must occur in the cytoplasm  whereas for the missense mutations it may occur in both the nucleus and cytoplasm  RX  which differs from RX by only  amino acids  did not show significant repression of the wild type RUNX in the cotransfection assay and may be considered as haploinsufficient  Structural analysis of the C terminal end of the Runt domain by nuclear magnetic resonance suggested that this is a region of conformational flexibility  RX  which lacks  amino acids in this region  might show decreased flexibility and increased affinity for PEBPβ CBFβ over RX  Alternatively  the lack of inhibition by RX could be due to technical difficulties inherent in the assay  as seen by the inconclusive results observed for the frameshift mutations One of the notable features of the FPD AML RUNX missense mutants studied here is the colocalization of the RUNX and PEBPβ CBFβ in cytoplasmic speckles  where sequestration of PEBPβ CBFβ could occur  There is a correlation between colocalization  PEBPβ CBFβ interaction in vitro  and the level of the inhibitory effect  Because no such pattern was seen with the nonsense and frameshift mutants prematurely terminated within the Runt domain  the presence of the C terminal region must be required for aggregation of RUNX into the speckled cytoplasmic patterns  This raises the possibility that the speckled structures could be the result of C terminal associating molecules  which could also be sequestered in the cytoplasm It was recently shown that dimerization with PEBPβ CBFβ protects RUNX from ubiquitin proteasome mediated degradation  and substitution of K to R was shown to increase the half life of RUNX as lysine residues are ubiquitylated  Preferential binding of mutant RUNX proteins to PEBPβ CBFβ may increase their half lives  KE may be the most potent nonchimeric mutant inhibitor of wild type RUNX described so far because K is a residue important for ubiquitinylation Mice heterozygous for the RUNX AML ETO or PEBPβ CBFβ MYH knocked ins showed defects in definitive hematopoiesis similar to those observed in Runx    or PEBPβ CBFβ   mice   Thus  expression of strong dominant negative inhibitors of RUNX seems to produce an embryonic lethal phenotype  The dominant transmission of RUNX mutations in FPD AML implies that strong dominant negative RUNX mutations would also be expected to be lethal  Another study recently showed dominant negative mutations inCEBPA encoding the C EBPα transcription factor in sporadic AML  However the dominant negative effects of the predominantly frameshift mutations described in this study do not have an inhibitory effect by themselves but by increased formation of a secondary protein product by use of a downstream initiating methionine Progression to leukemia in FPD AMLHaploidy of RUNX is clearly sufficient for normal development but possibly insufficient for tumor suppression  Runx   mice show a complete lack of definitive hematopoietic cells in the fetal liver  with death occurring from hemorrhages in the central nervous system at   days after coitus Runx    mice demonstrate minimal changes in phenotype   although closer hematologic analysis has revealed a trend for bone marrow progenitor cells to have increased sensitivity to G CSF  possibly reflecting a propensity to develop myelogenous leukemia The presence of biallelic RUNX mutations in sporadic leukemias may indicate that RUNX functions as a classical tumor suppressor gene   However  in other cases of sporadic leukemia only monoallelic RUNX mutations were described   Also  in FPD AML leukemic patients one each from the families with the complete RUNX deletion and RX  analysis of leukemic cells failed to detect RUNX mutations or deletion of the second RUNX allele Two tumor suppressor genes  have been shown to be haploinsufficient for tumor suppression in hemizygous mice where the potential complication of dominant negative mutations can be excluded   A revision of the Knudson model has been proposed to accommodate haploinsufficiency  The normal level of biologic activity of a tumorigenic gene is tightly controlled  and an increase or decrease in this activity leads to increased tumor susceptibility  In a multiprotein complex or molecular cascade  hemizygous loss of each of   partners or molecules in the same pathway may be almost as tumorigenic as homozygous loss of any one partner While mutations in early acting genes such as RUNX predispose to development of hematologic malignancies  the affected lineage and consequent type of malignancy may depend upon which genes subsequently sustain downstream  hits  from additional somatic mutation  For example  combined positivity for antigens CD  C KIT  and HLA DR characterizes the CBF leukemias AML M   and AML MEo    C KIT  is a transmembrane tyrosine kinase receptor for stem cell factor  which is required for normal hematopoiesis  Mutations in C KIT have been described in various sporadic hematologic malignancies and other diseases   For example  mutations of residue D in exon  of C KIT were detected in  of  patients with either t AML M or inv AML MEo It is difficult to make a robust correlation between the type of mutation and the proportion of patients who develop leukemia in FPD AML families  because of the limited number of individuals identified and a lack of detailed clinical information  It would appear  however  that families with mutations acting simply via haploinsufficiency  show a smaller proportion of affected individuals who develop leukemia than do families transmitting mutations that may act in a dominant negative fashion   This is consistent with the hypothesis that a second mutation has to occur in RUNX or other genes to cause leukemia among individuals harboring an inheritedRUNX mutation  and these mutations are more likely to occur in the individuals with lower biologic activity of RUNX  For example  in  of the largest FPD AML pedigrees  a markedly higher rate of leukemia is seen in the family with strong predicted dominant negative KE mutation  compared with the pedigree with a complete deletion of RUNX  Fig   Download figureOpen in new tabDownload powerpointFig    A summary of the FPD AML RUNX mutants and the studies presented in this paper The FPD AML RUNX mutants are listed in descending order of PEBP CBF activity  The Runt and transactivation domains are indicated on the mutant proteins as appropriate  Pedigrees  to  are from Song et al  and A and A are described in this paper  The results of the EMSA   affinity assay   subcellular localization  and transactivation studies are summarized  The ratio of leukemic FPD individuals to FPD affected individuals in each pedigree is indicated on the right  In the largest FPD AML pedigrees  a markedly higher rate of leukemia is seen in the family with strong predicted dominant negative KE mutation  compared with the pedigree with a complete deletion of RUNX  Thus  we propose that the less functional PEBP CBF transcriptional regulation complex present in a hematopoietic cell due to variable inhibitory effects of heterozygous mutations  or mutations of both alleles  the higher the propensity to develop leukemia  This mechanism is valid for FPD AML or sporadic leukemia patients  Clearly  biallelic mutations would be more prone to leukemia development although additional genetic changes may still be required in other genes  Analyses of additional FPD AML and sporadic leukemia cases with additional clinical and molecular data  including mutation analyses of genes coding for partners of PEBP CBF or molecules in the same tumorigenic pathway  will help provide evidence for this hypothesis  The RUNX AML gene is the most frequent target for chromosomal translocation in leukemia  In addition  recent studies have demonstrated point mutations in the RUNX gene as another mode of genetic alteration in development of leukemia  Monoallelic germline mutations in RUNX result in familial platelet disorder predisposed to acute myelogenous leukemia   Sporadic point mutations are frequently found in three leukemia entities  AML M subtype  MDS AML  and secondary  MDS AML  Therapy related leukemias resulting from anticancer treatments are not uncommon  and the incidence of RUNX point mutations appears comparable to the incidence of the t AML M subtype and the inv AML MEo subtype  Half of the point mutations in M cases are biallelic  although the frequency varies with ethnicity  Most of the RUNX mutations are clustered in the Runt domain and result in defective DNA binding but active beta subunit binding  which is consistent with three dimensional structural findings and may explain the dominant inhibitory effects  Unlike the classical tumor suppressor genes requiring biallelic inactivation  haploinsufficient RUNX is apparently leukemogenic  However  RUNX abnormalities per se are insufficient to cause full blown leukemia  Intensive investigation of cooperating genetic alterations should elucidate leukemic mechanisms   Keywords  RUNX  AML  PEBP  point mutation  familial leukemia Top of page Introduction Since the discovery of the Philadelphia chromosome  t  in chronic myelogenous leukemia in   leukemia researchers and physicians have paid much attention to the investigation of chromosomal translocations and resultant chimeric genes  It was also the case for RUNX AML related leukemia   Miyoshi et al   identified AML as a gene located on the breakpoint of chromosomal translocation t in   Since then  leukemia research on AML RUNX has been exclusively concentrated on investigating t and resultant AML ETO  t  RUNX EVI  t  TEL RUNX  and inv  PEBPbeta MYH  In   Osato et al   and Song et al  reported a RUNX point mutation in sporadic and familial myeloid leukemia  respectively  Subsequent studies have identified point mutations in acute myeloid leukemia   predominantly in the M subtype  in different ethnic groups   Until recently  mutations were believed to cluster within the Runt domain  However  mutations in the C terminal region  outside the Runt domain  have also been identified  predominantly in MDS AML   Finally  RUNX point mutations turned out to occur with comparable frequency to the first mode of RUNX alterations  namely  t  inv  and t  in secondary MDS AML   Accordingly  the RUNX point mutation is fully recognized as the second mode of leukemogenesis in RUNX leukemia  In this review  I summarize the currently available cases of point mutations and discuss the leukemogenesis mediated by RUNX    status   Top of page Frequency and distribution of the point mutations The RUNX gene consists of  exons  and two promoters followed by the distinct initiation codon in exon  and exon   The proximal promoter driven transcript starts in exon  and encodes  amino acids    whereas the distal promoter driven transcript from exon  encodes  aa   Full length  contains an additional  aa instead of  aa from the full length  in its N terminal end  Thus  both full lengths share exons    B  and  and nearly the complete coding sequence  The Runt domain  a highly conserved  aa protein motif responsible for both DNA binding and heterodimerization with the beta subunit PEBPbeta CBFbeta spans aa   in full length  and is located on exons    All RUNX mutation screening studies covered exons    and most included intron sequences flanking the exon  A few studies screened exons  and  as well  and half of the studies did not cover exons   encoding the C terminal moiety where the transactivation domain spanning aa   is located  All types of human leukemia  except chronic lymphocytic leukemia   have been extensively investigated  In addition  a significant number of cell lines  including those established from solid tumors  as well as healthy volunteers were also included in the mutation screenings    A total of  cases out of the  tested had a RUNX alteration including a point mutation and gene deletion  Most of the nucleotide changes resulted in amino acid changes with clear functional defects as shown in Figure   However  some mutations failed to show any apparent functional alterations  and the nucleotide change may have been constitutive or leukemia specific  Nucleotide changes that do not lead to amino acid change  may generate an abnormal splicing donor or acceptor site  resulting in a nonfunctional protein  These enigmatic cases are listed as  variation  in Table   The numbers of mutations in Table  include variation cases  Most of the mutations are clustered within the Runt domain   N terminal region     Runt domain     C terminal region     The total number of mutations  differs  because the compiled mutants in Figure  include  cases from FPD AML  which are not listed in Table   Conversely  gene deletion is not included in Figure   while a biallelic case in Table  provides two distinct mutations simultaneously   Figure   Figure    Unfortunately we are unable to provide accessible alternative text for this  If you require assistance to access this image  please contact help nature com or the author Distribution and functional consequences of RUNX gene mutations  The mutations  and disease subtypes are compiled to the left of the corresponding amino acid indicated at the center  Amino acids in the Runt domain in dark green in the green bar representing exons are listed in a consecutive manner  while those outside the Runt domain are not  Missense  termination  and frameshift mutations are highlighted by yellow  dark gray  and gray  respectively  The functional consequences  DNA binding  beta subunit binding  and dominant negative effects are listed on the right    indicates function retention  Normal and hypernormal functions are in pink and red  respectively  Dominant negative effects are shown in light blue or dark blue in the right end column  The strong cases are in dark blue  Full figure and legend   Table    Frequency of mutations in the RUNX AML gene in various cancer patients and healthy volunteers  Table    Frequency of mutations in the RUNX  AML gene in various cancer patients and healthy volunteers   Unfortunately we are unable to provide accessible alternative text for this  If you require assistance to access this image  please contact help nature com or the author Full table  Top of page Functional consequence  haploinsufficiency Entire gene deletions and early truncated type mutations  were found with significant frequency  Most of the missense mutations also lost their DNA binding and transcriptional activities   Since the majority of cases are heterozygous  haploinsufficiency appears to be a basis for pathogenesis  This notion is most strongly supported by the whole gene deletion in FPD AML  RUNX    status is likely to provide the cells with a growth advantage to overcome  selective pressure  in the subject  However  if this were the only mechanism  mutations would be distributed throughout the gene without  hot spots   Therefore  clustering in the Runt domain suggests that a mechanism other than genuine haploinsufficiency is important for leukemogenesis   Top of page Dominant negative effect by the retained bold italic beta binding activity In the Runt domain  eight regions or  hot spots  were affected more than twice   Two different mechanisms are probably at least partly responsible  First  the primary sequence of particular sites seems to be preferably targeted by the mutation mechanism  R  R  and R are encoded by triplets containing CpG dinucleotides  which are extremely prone to transition type mutations by a mechanism facilitated by methylation of the cytidine residue   Second  certain mutation sites are considered to provide cells a stronger growth advantage under  selective pressure   Mapping of the missense mutations strongly supports this notion  Three regions recurrently affected by missense mutations exclusively correspond to three loop containing regions responsible for DNA binding  betaP loop  betaE  loop  and betaP loop    Consequently  these mutants are defective only in DNA binding  and not in packing or heterodimerization   which may explain the dominant negative effects of the RUNX mutants  In cotransfection assays  missense and some nonsense mutants showed inhibitory effects towards transactivation mediated by wild type RUNX  These mutants bound to PEBPbeta CBFbeta more efficiently than wild type RUNX   In addition  PEBPbeta protects RUNX from ubiquitin proteasome mediated degradation   Thus  preferential binding of mutant RUNX proteins to PEBPbeta may in turn increase protein half life and hence further augment the dominant negative effects  In the right end column of Figure   a general correlation can be seen between the retained beta binding activity and the inhibitory potential of each mutant  This correlation is more clearly indicated by the comparison of the distribution of missense mutations between RUNX and RUNX   Germline mutation of RUNX causes the bone disease cleidocranial dysplasia    The missense mutations of RUNX distributed throughout the Runt domain affected the overall folding architecture in the cases marked by asterisks in Figure   resulting in impairment of both DNA  binding and heterodimerization activity   The mutations underlined in Figure  showed defects in beta binding alone  In clear contrast  no such mutations resulting in defective packing or beta binding have been found in RUNX so far    but functional analyses have not yet been performed  No suspected case with defect in beta binding alone has been reported   Conversely  mutations mapping to the first DNA interacting region  betaP loop   are seen in RUNX but not in RUNX    These RUNX mutants  RC  KN  and KE showed strong affinity for the beta subunit and hence a strong dominant negative effect  suggesting that haploinsufficiency is a dominant mechanism in RUNX mutants  whereas a dominant negative effect is important in RUNX mutants  Sequestration of the beta subunit may be one important mechanism of the dominant negative effect caused by the RUNX mutants   Figure   Figure    Unfortunately we are unable to provide accessible alternative text for this  If you require assistance to access this image  please contact help nature com or the author Structural mappings of the missense mutations detected in leukemia or the bone disease CCD  The RUNX mutants in leukemia  are mapped exclusively to the three regions involved in DNA binding  whereas the RUNX mutations in CCD  are distributed all over the Runt domain except for the loop   Mutations associated with packing  and beta binding  defects are found only in CCD  Adapted from Nagata et al    Full figure and legend   Top of page The lower the RUNX activity  the higher the leukemogenicity How does the dominant negative effect mentioned above relate to leukemogenesis  Although the limited number of individuals and minimal clinical information evaluated prohibit a robust conclusion  families with mutations acting simply via haploinsufficiency have a lower incidence of leukemia than families with mutations acting in a dominant negative fashion  In two of the largest FPD AML pedigrees  leukemia occurred at a markedly higher rate in the family with the strong dominant negative KE mutation  compared to the pedigree with a complete deletion of RUNX     Thus  low RUNX activity appears to correlate with high leukemogenicity  The hypothesis may be expanded to some extent to leukemogenesis induced by chimeric genes  The majority of current findings indicate that a major function of chimeric genes is a transdominant inhibition of wild type RUNX  resulting in severely decreased RUNX activity in affected cells  Currently  more than  partner genes fused with RUNX have been identified in chromosomal translocations  Although each chimeric gene may have its own leukemogenic mechanism  RUNX insufficiency may be a common underlying mechanism in RUNX leukemias   Table    RUNX mutations in familial leukemia  FPD AML  Table    RUNX mutations in familial leukemia  FPD  AML   Unfortunately we are unable to provide accessible alternative text for this  If you require assistance to access this image  please contact help nature com or the author Full table  Acquired trisomy  underscores this notion  RUNX    mutant alleles appear to be duplicated frequently  This finding may also lend supporting evidence to the hypothesis that reduced RUNX activity increases leukemogenic status  This argument will be discussed later   Top of page Mutation status   disease phenotype in sporadic leukemias Biallelic mutation and AML M subtype RUNX point mutations were most frequently found in the AML M subtype   In addition   of the   M cases showed the biallelic type of mutation   Conversely  only two of the  RUNX mutation cases in other leukemia subtypes were the biallelic type  Therefore  it is clear that RUNX point mutations of the biallelic type are tightly associated with the AML M subtype  Mice heterozygous for the RUNX MTG or PEBPbeta MYH chimeric genes showed defects in definitive hematopoiesis similar to Runx    mice  suggesting that the chimeric genes can act as strong transdominant inhibitors of the remaining normal Runx allele   Therefore  the leukemogenic potential of biallelic RUNX mutants is considered equivalent to those of chimeric genes  On the other hand  t  inv  and biallelic point mutations are related to the distinct M  MEo  and M subtypes of myeloid leukemias  respectively  The different disease phenotypes are quite interesting  The contribution of each partner gene  thought to be considerable  may also be explained by the different residual activities of RUNX in the affected adult hematopoietic cells  Conditional Runx knockout mice showed apparent defects in hematopoiesis   whereas hematopoiesis in conditional heterozygous RUNX ETO knockin mice was nearly normal   suggesting that Runx    induces more severe effects than the RUNX ETO chimeric gene in adult hematopoiesis  The M subtype shows minimally differentiated morphology  while the M subtype is characteristically well differentiated  Therefore  the differing residual activities of RUNX seem to be important  In addition  influence on the activities of other RUNX family genes must be considered because RUNX MTG or PEBPbeta  MYH chimeric genes could inhibit RUNX and RUNX as well  while RUNX    status is a defect in RUNX alone  Careful study will resolve this intriguing issue  Table    RUNX mutations in AML M subtype  Table    RUNX mutations in AML M subtype   Unfortunately we are unable to provide accessible alternative text for this  If you require assistance to access this image  please contact help nature com or the author Full table  Of note  the biallelic type mutation is strictly confined to the status with two copies of RUNX  Biallelic mutation in trisomy  is not associated with the AML M subtype  but is associated with myeloid malignancies other than M  This phenotypic difference will be discussed later   Entire distribution including the C terminal moiety and MDS AML secondary leukemia The RUNX point mutations are also frequently found in MDS and subsequent overt leukemia    The mutations are characteristically distributed  as they were also detected in the C terminal region which is rarely targeted in other types of leukemias   Since this distribution was only recently identified  C terminal mutations were overlooked in a number of previous studies  Half of the studies did not investigate exons B and  in the C terminal region  With regard to MDS AML  the incidence covering both exons   and exons   were evaluated less frequently than those of exons   alone   Thus  C terminal mutations may be underestimated in MDS AML  However  C terminal mutations still seem rare in AML  since C terminal mutations were not identified in the study with a relatively large number of cases  Therefore  RUNX point mutations including the C terminal moiety appear to be found predominantly in MDS AML  Table    RUNX mutations in MDS  Table    RUNX mutations in MDS   Unfortunately we are unable to provide accessible alternative text for this  If you require assistance to access this image  please contact help nature com or the author Full table  Harada et al   from Hiroshima  one of the two cities suffered from atomic bomb  reported two further implications of RUNX mutations in secondary MDS or AML   including therapy related and atomic bomb related MDS AML  Firstly  the relationship to radiation seems important in terms of the social issue  In addition to the victims in the above two cities  an estimated   US soldiers and a significant number of local residents and passengers have been irradiated in the aboveground nuclear weapon test sites at Nevada  Bikini atoll  and others in the late s to s  Many of these people are thought to have succumbed to MDS AML caused by RUNX point mutation  Secondly  the involvement in therapy related MDS AML seems to have more general impact  With the improvement of treatments for primary cancer  the incidence of therapy related MDS AML continues to increase  Therapy related MDS AML accounts for roughly    of all new cases of MDS and AML in standard hospitals  Therefore  the incidence of secondary MDS AML with RUNX mutation is comparable to those of t AML M subtype and inv AML MEo subtype   Considering its poor prognosis  clinical implication of RUNX point mutation in secondary MDS AML would be rather more significant than we expected   Table    Estimated frequency of leukemia with RUNX point mutation  Table    Estimated frequency of leukemia with RUNX point mutation   Unfortunately we are unable to provide accessible alternative text for this  If you require assistance to access this image  please contact help nature com or the author Full table  Leukemic incidence rates were based on several textbooks and public databases from CDC  WHO  and National Cancer Institute   CLL accounts for     which may explain the relatively low incidences of t  inv  and t   Acquired  trisomy  and non MO myeloid malignancy Preudhomme et al   first reported the relationship between trisomy  and RUNX point mutation   Sporadic cases of myeloid but not lymphoid malignancy with trisomy  were associated with a high frequency of RUNX point mutations  Taketani et al   subsequently surveyed RUNX mutation in congenital cases of trisomy   that is  the Down syndrome related hematopoietic diseases  However  point mutation in congenital cases appears to be rare except for the enigmatic HN variation  which has nearly normal function  Allelic analysis of sporadic cases with acquired trisomy  showed the following combination    mutant vs   wild type  The co occurrence of mutation and trisomy  should be associated with the allelic combination with one mutant and two WT combination  However  this combination has never been found  These findings suggest that trisomy  increases the copy number of the mutant allele  which already exists in the status of two copies of RUNX  In other words  trisomy  is thought to be a secondary change to RUNX    status  Trisomy  as a second hit will be discussed later  Table    Trisomy  and RUNX mutation  Table    Trisomy  and RUNX mutation   Unfortunately we are unable to provide accessible alternative text for this  If you require assistance to access this image  please contact help nature com or the author Full table  As mentioned earlier  biallelic RUNX mutations are tightly liked to the AML M subtype  However  interestingly  acquired   carrying a RUNX point mutation has never been observed in the AML M subtype  even in cases with biallelic mutations   This  discrepancy  is not explained by the differing residual activities of RUNX  as half of the M cases with RUNX mutation are monoallelic  Leukemic phenotype may be determined by an as yet undetermined mechanism   Top of page Familial leukemia  FPD AML Heterozygous germline mutation in the RUNX gene causes familial leukemia  familial platelet disorder with predisposition to acute myelogenous leukemia   A total of  pedigrees have been identified so far  mainly in individuals of European descent   FPD alone  before the onset of AML  is not fatal but significantly inhibits blood clotting  which could be life threatening after surgery  injury  dental treatment  or childbirth  The incidence of leukemia among affected individuals varies from  to     and is highest in familial leukemia syndromes including Fanconi anemia  Bloom syndrome  and Kostman syndrome  The affected subjects seem to develop myeloid malignancies throughout their life span  but not later in life as described in some previous reports  Leukemia developed before the age of  years in four of  cases  These features of FPD AML were compared with a similar dominant hereditary disease  Huntington disease   The subjects of HD are healthy until disease onset at   years of age and die   years later  In addition to the low penetrance and or low severity of symptoms  late onset is a key factor for hereditary disease  since patients often have children before disease symptoms appear  On the other hands  FPD AML seems detrimental to the subjects in any factors of hereditary disease  How can such a pedigree full of drawbacks exist  FPD AML may persist as a disease due to the easy mutability of RUNX  As discussed before  RUNX indeed contains easy mutable primary sequences  triplets containing CpG dinucleotides extremely prone to transition type mutations facilitated by methylation of the cytidine residue   This mechanism could work particularly well in germ cells  The FPD AML pedigree may readily disappear in the human population  while a  de novo  pedigree could be continually generated with comparable frequency  Consistent with this argument  of CCD cases caused by RUNX mutation  approximately half are the  de novo  and not the familial type   Irradiation readily generates CCD like phenotypes in mice due to Runx alterations  RUNX shares the above noted easy mutable sequences  In the case of RUNX  healthy British subjects have two kinds of variations  LS and Ssyn with  and   frequency  respectively   Japanese subjects also have several recurrent enigmatic variations  GR  Isyn  and HN  Interestingly  GR was identified in both leukemia patients and healthy volunteers   These variations  probably including polymorphisms  might reflect the above noted easy mutability of RUNX   As mentioned above  FPD AML is extremely rare worldwide  Therefore  this clinical entity had been overlooked by physicians until recently  Bone marrow transplantaion was carried out from an affected sibling with mild FPD symptoms to a patient suffering from AML   The recipient succumbed to leukemia again  clearly from the hematopoietic cells derived from the donor   Top of page Genetic alterations cooperating with RUNX    status As mentioned earlier  RUNX    status is frequently found in the AML M subtype  Thus  RUNX may appear to act as a classical tumor suppressor gene  requiring inactivation of two alleles according to Knudson s two hits theory  However  biallelic inactivation is not required in all cases of leukemogenesis  in fact  the overall frequency of biallelic cases is relatively low   Biallelic cases of FPD AML have never been reported  Therefore  RUNX    per se appears to be leukemogenic  The dominant negative effect associated with missense mutations might play a role in leukemogenesis  but not to the same extent as gene inactivation  In any case  heterozygous mutation appears to be a step towards leukemogenesis  Therefore  a mutation in the wild type RUNX on the remaining allele is considered to be a second hit in addition to RUNX    status  Furthermore  acquired but not congenital trisomy  is also thought to be a second hit  As discussed earlier  the link between   and RUNX mutation is interpreted as follows  a mutation takes place in the two copy status and the mutant allele is duplicated in accordance with    In both RUNX    and   cases  acquirement of another mutant allele is therefore thought to be an additional genetic alteration cooperating with RUNX    status    Figure   Figure    Unfortunately we are unable to provide accessible alternative text for this  If you require assistance to access this image  please contact help nature com or the author Leukemogenesis from Runx     Biallelic mutations of RUNX  are tightly liked to the AML M subtype  However  acquired   carrying a RUNX point mutation  has never been observed in M  despite cases with biallelic mutations  Mutation likely takes place in the two copy status  and the mutant allele is duplicated in accordance with    resulting in myeloid malignancies other than M  FLT  mutations were frequently found in M cases with RUNX point mutations  These genetic alterations are mapped based on such observations  Full figure and legend   In line with    cytogenetics helps to identify second hits  Patients with RUNX point mutations show the following concurrent abnormalities   cases showed              Loss of chromosome  is suspected to be a strong predisposing factor for the second hit in RUNX      FLT  mutations were frequently observed even in monoallelic RUNX mutations in M patients    Hence  the FLT  mutation is also a cooperating genetic alteration   FLT  mutation results in a constitutively active form known to stimulate cell proliferation  Therefore  differentiation blockade by RUNX alteration and growth stimulation by FLT mutation would synergistically contribute to a substantial fraction of AML M subtype cases  Interesting enough  FLT  mutations are known to be rare in the other RUNX leukemias  t and inv  Certain RUNX leukemias seem to cooperate with distinct second hits  Elucidation of this mechanism might shed light on development of RUNX leukemias   Top of page Current problems Region for mutation screening As discussed previously  half of the screening studies did not cover C terminal exons B and  and therefore overlooked C terminal mutations in MDS AML  C terminal screening is unlikely to reveal information about AML but is recommended  Likewise  exons  and  might be affected in particular types of leukemia  Theoretically  the promoter and untranslated regions could also be targeted  Variations Significant numbers of variations are reported  as shown in Table  and Figure   Functional analyses have not yet been performed in all the cases  Each variation must be studied to determine if it is a mutation or a polymorphism  Analyses of polymorphisms might help to characterize susceptibility to leukemia  Dominant negative effect by C terminal mutants in MDS AML In the reporter assay  all C terminal mutants found in MDS AML showed inhibitory effects in a transdominant manner  like the missense mutants in the Runt domain   However  similar C terminal RUNX mutations in CCD patients retain partial transactivation activity and show much milder phenotypes compared to the standard CCD phenotypes  which are caused by simple gene deletions   This discrepancy may be due to the distinct cell context or promoter used for the assay  However  a correlation with MDS but not AML might indicate that C terminal mutants have milder inhibitory effects than Runt domain mutants  The biological assay  but not the reporter assay  may address this issue  Cooperative genetic alterations FLT  mutation and   are thought to be second hits in MDS AML with RUNX mutations  High frequency of chromosome  loss suggests inactivation of the unknown gene on chromosome  plays a synergistic role with RUNX mutations  However  the overall frequency of these mutations does not account for all cases  Furthermore  to induce a leukemia  three to five genetic alterations are considered to be required based on current evidence  Therefore  a number of genetic alterations must be identified to understand the whole leukemogenic mechanism of RUNX leukemia  My laboratory recently established a mouse system that appears to be quite useful  Myeloid leukemia prone mice mimicking FPD AML were generated by backcrossing Runx    onto the BXH strain  in which retroviral insertional mutagenesis occurs spontaneously with the eventual development of myeloid leukemia in over   of the cases   The proviral integration sites in the leukemias provide powerful genetic tags for disease gene identification  Therefore  collaborating oncogenes with Runx    would be readily detected  Loss of function of RUNX is widely believed to be the common underlying mechanism of RUNX leukemia  RUNX    status is considered to be the minimum alteration required for several forms of RUNX leukemias  Therefore  the mouse model for FPD AML mentioned above should be useful for searching for cooperative genetic alterations in the other RUNX leukemias  t and inv  These genetic alterations identified by the retroviral tagging system might explain the leukemogenic mechanisms not only of RUNX point mutations but also of other RUNX leukemias  Top of page Future directions Why is RUNX    status leukemogenic  This question seems to be the most fundamental question not only for RUNX point mutation but also all RUNX leukemias  Determining the means by which RUNX    is leukemogenic may not be possible by the currently used approaches to evaluate differentiation  proliferation  and apoptosis at the cellular level and transcriptional control at the molecular level  A novel approach is warranted  For example  alteration of  stemness  should be investigated in RUNX    status  Stemness could provide cells with a time window sufficient for sequential genetic alterations  Intrinsic molecular functions relevant to leukemogenesis  other than transcriptional control  might be identified by novel approaches'],\n",
       "      dtype=object)"
      ]
     },
     "execution_count": 29,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "sentences \n",
    "for sentence in np.array(training_text['Text']):"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {},
   "outputs": [],
   "source": [
    "model = gensim.models.Word2Vec(np.array(training_text['Text']), size = 100, window = 5, min_count = 5, workers =4)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 28,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "dict_items([('C', <gensim.models.keyedvectors.Vocab object at 0x7fca5e0ab518>), ('y', <gensim.models.keyedvectors.Vocab object at 0x7fca5e0ab4a8>), ('c', <gensim.models.keyedvectors.Vocab object at 0x7fca5e0ab438>), ('l', <gensim.models.keyedvectors.Vocab object at 0x7fca5e0ab358>), ('i', <gensim.models.keyedvectors.Vocab object at 0x7fca5e0ab2e8>), ('n', <gensim.models.keyedvectors.Vocab object at 0x7fca5e0ab3c8>), (' ', <gensim.models.keyedvectors.Vocab object at 0x7fca5e0ab278>), ('d', <gensim.models.keyedvectors.Vocab object at 0x7fca5e0ab208>), ('e', <gensim.models.keyedvectors.Vocab object at 0x7fca5e0ab198>), ('p', <gensim.models.keyedvectors.Vocab object at 0x7fca5e0ab128>), ('t', <gensim.models.keyedvectors.Vocab object at 0x7fca5e0ab0b8>), ('k', <gensim.models.keyedvectors.Vocab object at 0x7fca5e0ab048>), ('a', <gensim.models.keyedvectors.Vocab object at 0x7fca5e0a8f98>), ('s', <gensim.models.keyedvectors.Vocab object at 0x7fca5e0a8f28>), ('r', <gensim.models.keyedvectors.Vocab object at 0x7fca5e0a8eb8>), ('g', <gensim.models.keyedvectors.Vocab object at 0x7fca5e0a8e48>), ('u', <gensim.models.keyedvectors.Vocab object at 0x7fca5e0a8dd8>), ('v', <gensim.models.keyedvectors.Vocab object at 0x7fca5e0a8d68>), ('o', <gensim.models.keyedvectors.Vocab object at 0x7fca5e0a8cf8>), ('f', <gensim.models.keyedvectors.Vocab object at 0x7fca5e0a8c88>), ('m', <gensim.models.keyedvectors.Vocab object at 0x7fca5e0a8c18>), ('D', <gensim.models.keyedvectors.Vocab object at 0x7fca5e0a8ba8>), ('K', <gensim.models.keyedvectors.Vocab object at 0x7fca5e0a8b38>), ('h', <gensim.models.keyedvectors.Vocab object at 0x7fca5e0a8ac8>), ('w', <gensim.models.keyedvectors.Vocab object at 0x7fca5e0a8a58>), ('b', <gensim.models.keyedvectors.Vocab object at 0x7fca5e0a89e8>), ('P', <gensim.models.keyedvectors.Vocab object at 0x7fca5e0a8978>), ('E', <gensim.models.keyedvectors.Vocab object at 0x7fca5e0a8908>), ('T', <gensim.models.keyedvectors.Vocab object at 0x7fca5e0a8898>), ('S', <gensim.models.keyedvectors.Vocab object at 0x7fca5e0a8828>), ('M', <gensim.models.keyedvectors.Vocab object at 0x7fca5e0a87b8>), ('A', <gensim.models.keyedvectors.Vocab object at 0x7fca5e0a8748>), ('x', <gensim.models.keyedvectors.Vocab object at 0x7fca5e0a86d8>), ('H', <gensim.models.keyedvectors.Vocab object at 0x7fca5e0a8668>), ('F', <gensim.models.keyedvectors.Vocab object at 0x7fca5e0a85f8>), ('R', <gensim.models.keyedvectors.Vocab object at 0x7fca5e0a8588>), ('W', <gensim.models.keyedvectors.Vocab object at 0x7fca5e0a8518>), ('q', <gensim.models.keyedvectors.Vocab object at 0x7fca5e0a84a8>), ('z', <gensim.models.keyedvectors.Vocab object at 0x7fca5e0a8438>), ('N', <gensim.models.keyedvectors.Vocab object at 0x7fca5e0a83c8>), ('Y', <gensim.models.keyedvectors.Vocab object at 0x7fca5e0a8358>), ('I', <gensim.models.keyedvectors.Vocab object at 0x7fca5e0a82e8>), ('B', <gensim.models.keyedvectors.Vocab object at 0x7fca5e0a8278>), ('G', <gensim.models.keyedvectors.Vocab object at 0x7fca5e0a8208>), ('J', <gensim.models.keyedvectors.Vocab object at 0x7fca5e0a8198>), ('L', <gensim.models.keyedvectors.Vocab object at 0x7fca5e0a8128>), ('Z', <gensim.models.keyedvectors.Vocab object at 0x7fca5e0a80b8>), ('V', <gensim.models.keyedvectors.Vocab object at 0x7fca5e095dd8>), ('μ', <gensim.models.keyedvectors.Vocab object at 0x7fca5e095208>), ('Q', <gensim.models.keyedvectors.Vocab object at 0x7fca5e095048>), ('j', <gensim.models.keyedvectors.Vocab object at 0x7fca5e095eb8>), ('X', <gensim.models.keyedvectors.Vocab object at 0x7fca5e095828>), ('O', <gensim.models.keyedvectors.Vocab object at 0x7fca5e095978>), ('β', <gensim.models.keyedvectors.Vocab object at 0x7fca5e095128>), ('α', <gensim.models.keyedvectors.Vocab object at 0x7fca5e0959e8>), ('U', <gensim.models.keyedvectors.Vocab object at 0x7fca5e095f28>), ('µ', <gensim.models.keyedvectors.Vocab object at 0x7fca5e095908>), ('ü', <gensim.models.keyedvectors.Vocab object at 0x7fca5e0957b8>), ('Å', <gensim.models.keyedvectors.Vocab object at 0x7fca5e095f98>), ('Δ', <gensim.models.keyedvectors.Vocab object at 0x7fca5e095e48>), ('γ', <gensim.models.keyedvectors.Vocab object at 0x7fca5e095898>), ('ε', <gensim.models.keyedvectors.Vocab object at 0x7fca5e095c88>), ('λ', <gensim.models.keyedvectors.Vocab object at 0x7fca5e095d68>), ('δ', <gensim.models.keyedvectors.Vocab object at 0x7fca5e095278>), ('\\t', <gensim.models.keyedvectors.Vocab object at 0x7fca5e095198>), ('\\u2003', <gensim.models.keyedvectors.Vocab object at 0x7fca5e095358>), ('º', <gensim.models.keyedvectors.Vocab object at 0x7fca5e0952e8>), ('\\u2009', <gensim.models.keyedvectors.Vocab object at 0x7fca5e095438>), ('é', <gensim.models.keyedvectors.Vocab object at 0x7fca5e095668>), ('ö', <gensim.models.keyedvectors.Vocab object at 0x7fca5e0953c8>), ('ã', <gensim.models.keyedvectors.Vocab object at 0x7fca5e0955f8>), ('ó', <gensim.models.keyedvectors.Vocab object at 0x7fca5e095588>), ('á', <gensim.models.keyedvectors.Vocab object at 0x7fca5e095518>), ('κ', <gensim.models.keyedvectors.Vocab object at 0x7fca5e095cf8>), ('σ', <gensim.models.keyedvectors.Vocab object at 0x7fca5e095748>), ('ë', <gensim.models.keyedvectors.Vocab object at 0x7fca5e0950b8>), ('ϵ', <gensim.models.keyedvectors.Vocab object at 0x7fca5e095c18>), ('\\u2008', <gensim.models.keyedvectors.Vocab object at 0x7fca5e0954a8>), ('ρ', <gensim.models.keyedvectors.Vocab object at 0x7fca5e095ba8>), ('ï', <gensim.models.keyedvectors.Vocab object at 0x7fca5e095b38>), ('\\u2002', <gensim.models.keyedvectors.Vocab object at 0x7fca5e095ac8>), ('ô', <gensim.models.keyedvectors.Vocab object at 0x7fca5e095a58>), ('Ã', <gensim.models.keyedvectors.Vocab object at 0x7fca5e9ec2b0>), ('Ÿ', <gensim.models.keyedvectors.Vocab object at 0x7fca5e9ec278>), ('Â', <gensim.models.keyedvectors.Vocab object at 0x7fca5e9ec4e0>), ('â', <gensim.models.keyedvectors.Vocab object at 0x7fca5e9ec208>), ('Œ', <gensim.models.keyedvectors.Vocab object at 0x7fca5e9ec588>), ('χ', <gensim.models.keyedvectors.Vocab object at 0x7fca5e9ec4a8>), ('\\u200a', <gensim.models.keyedvectors.Vocab object at 0x7fca5e9ec518>), ('ß', <gensim.models.keyedvectors.Vocab object at 0x7fca5e9ec240>), ('π', <gensim.models.keyedvectors.Vocab object at 0x7fca5e9ec470>), ('ﬁ', <gensim.models.keyedvectors.Vocab object at 0x7fca5e9ec550>), ('ﬂ', <gensim.models.keyedvectors.Vocab object at 0x7fca5d6dafd0>), ('ê', <gensim.models.keyedvectors.Vocab object at 0x7fca5d6da400>), ('þ', <gensim.models.keyedvectors.Vocab object at 0x7fca5e97a940>), ('ª', <gensim.models.keyedvectors.Vocab object at 0x7fca5e97aef0>), ('ÿ', <gensim.models.keyedvectors.Vocab object at 0x7fca5e97ac18>), ('Φ', <gensim.models.keyedvectors.Vocab object at 0x7fca5e97aa20>), ('\\u2005', <gensim.models.keyedvectors.Vocab object at 0x7fca5e97af98>), ('Ψ', <gensim.models.keyedvectors.Vocab object at 0x7fca5e97a240>), ('ζ', <gensim.models.keyedvectors.Vocab object at 0x7fca5e97a898>), ('θ', <gensim.models.keyedvectors.Vocab object at 0x7fca5e97ac88>), ('η', <gensim.models.keyedvectors.Vocab object at 0x7fca5e97add8>), ('ı', <gensim.models.keyedvectors.Vocab object at 0x7fca5e97ada0>), ('è', <gensim.models.keyedvectors.Vocab object at 0x7fca5e97ae48>), ('Σ', <gensim.models.keyedvectors.Vocab object at 0x7fca5e97a6a0>), ('ɛ', <gensim.models.keyedvectors.Vocab object at 0x7fca5e97a9e8>), ('φ', <gensim.models.keyedvectors.Vocab object at 0x7fca5e97a4a8>), ('\\x80', <gensim.models.keyedvectors.Vocab object at 0x7fca5e97aeb8>), ('\\x81', <gensim.models.keyedvectors.Vocab object at 0x7fca5e97a0f0>), ('Æ', <gensim.models.keyedvectors.Vocab object at 0x7fca5e97acc0>), ('Ì', <gensim.models.keyedvectors.Vocab object at 0x7fca5e97a160>), ('Β', <gensim.models.keyedvectors.Vocab object at 0x7fca5e97a550>), ('Ø', <gensim.models.keyedvectors.Vocab object at 0x7fca5e97a780>), ('\\x01', <gensim.models.keyedvectors.Vocab object at 0x7fca5e97a748>), ('ä', <gensim.models.keyedvectors.Vocab object at 0x7fca5e97a6d8>), ('ι', <gensim.models.keyedvectors.Vocab object at 0x7fca5e97af60>), ('à', <gensim.models.keyedvectors.Vocab object at 0x7fca5e97a630>), ('í', <gensim.models.keyedvectors.Vocab object at 0x7fca5e97ae80>), ('Á', <gensim.models.keyedvectors.Vocab object at 0x7fca5e97afd0>), ('Í', <gensim.models.keyedvectors.Vocab object at 0x7fca5e97af28>), ('Ó', <gensim.models.keyedvectors.Vocab object at 0x7fca5e97a908>), ('å', <gensim.models.keyedvectors.Vocab object at 0x7fca5e95ad68>), ('ç', <gensim.models.keyedvectors.Vocab object at 0x7fca5e95acf8>), ('ϕ', <gensim.models.keyedvectors.Vocab object at 0x7fca5e95a828>), ('Š', <gensim.models.keyedvectors.Vocab object at 0x7fca5e95ae10>), ('ψ', <gensim.models.keyedvectors.Vocab object at 0x7fca5e95af98>), ('Å', <gensim.models.keyedvectors.Vocab object at 0x7fca5e95a898>), ('Ò', <gensim.models.keyedvectors.Vocab object at 0x7fca5e95a9b0>), ('τ', <gensim.models.keyedvectors.Vocab object at 0x7fca5e95a978>), ('ˆ', <gensim.models.keyedvectors.Vocab object at 0x7fca5e95a9e8>), ('Ï', <gensim.models.keyedvectors.Vocab object at 0x7fca5e95a908>), ('Ð', <gensim.models.keyedvectors.Vocab object at 0x7fca5e95a8d0>), ('ΰ', <gensim.models.keyedvectors.Vocab object at 0x7fca5e95a940>), ('Ι', <gensim.models.keyedvectors.Vocab object at 0x7fca5e95a518>), ('Ł', <gensim.models.keyedvectors.Vocab object at 0x7fca5e95a4a8>), ('ù', <gensim.models.keyedvectors.Vocab object at 0x7fca5e95a470>), ('ø', <gensim.models.keyedvectors.Vocab object at 0x7fca5e95a4e0>)])"
      ]
     },
     "execution_count": 28,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "model.wv.vocab.items()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 24,
   "metadata": {},
   "outputs": [
    {
     "ename": "KeyError",
     "evalue": "\"word 'woman' not in vocabulary\"",
     "output_type": "error",
     "traceback": [
      "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
      "\u001b[0;31mKeyError\u001b[0m                                  Traceback (most recent call last)",
      "\u001b[0;32m<ipython-input-24-bbcd6746eaa6>\u001b[0m in \u001b[0;36m<module>\u001b[0;34m()\u001b[0m\n\u001b[0;32m----> 1\u001b[0;31m \u001b[0mmodel\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mwv\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mmost_similar\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mpositive\u001b[0m\u001b[0;34m=\u001b[0m\u001b[0;34m[\u001b[0m\u001b[0;34m'woman'\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0;34m'strong'\u001b[0m\u001b[0;34m]\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mnegative\u001b[0m\u001b[0;34m=\u001b[0m\u001b[0;34m[\u001b[0m\u001b[0;34m'man'\u001b[0m\u001b[0;34m]\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m",
      "\u001b[0;32m~/anaconda3/lib/python3.6/site-packages/gensim/models/keyedvectors.py\u001b[0m in \u001b[0;36mmost_similar\u001b[0;34m(self, positive, negative, topn, restrict_vocab, indexer)\u001b[0m\n\u001b[1;32m    358\u001b[0m                 \u001b[0mmean\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mappend\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mweight\u001b[0m \u001b[0;34m*\u001b[0m \u001b[0mword\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    359\u001b[0m             \u001b[0;32melse\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m--> 360\u001b[0;31m                 \u001b[0mmean\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mappend\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mweight\u001b[0m \u001b[0;34m*\u001b[0m \u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mword_vec\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mword\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0muse_norm\u001b[0m\u001b[0;34m=\u001b[0m\u001b[0;32mTrue\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m    361\u001b[0m                 \u001b[0;32mif\u001b[0m \u001b[0mword\u001b[0m \u001b[0;32min\u001b[0m \u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mvocab\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    362\u001b[0m                     \u001b[0mall_words\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0madd\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mvocab\u001b[0m\u001b[0;34m[\u001b[0m\u001b[0mword\u001b[0m\u001b[0;34m]\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mindex\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0;32m~/anaconda3/lib/python3.6/site-packages/gensim/models/keyedvectors.py\u001b[0m in \u001b[0;36mword_vec\u001b[0;34m(self, word, use_norm)\u001b[0m\n\u001b[1;32m    267\u001b[0m             \u001b[0;32mreturn\u001b[0m \u001b[0mresult\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    268\u001b[0m         \u001b[0;32melse\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m--> 269\u001b[0;31m             \u001b[0;32mraise\u001b[0m \u001b[0mKeyError\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m\"word '%s' not in vocabulary\"\u001b[0m \u001b[0;34m%\u001b[0m \u001b[0mword\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m    270\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    271\u001b[0m     \u001b[0;32mdef\u001b[0m \u001b[0mget_vector\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mself\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mword\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0;31mKeyError\u001b[0m: \"word 'woman' not in vocabulary\""
     ]
    }
   ],
   "source": [
    "model.wv.most_similar(positive=['woman', 'strong'], negative=['man'])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "training_variants = pd.read_csv(\"../Data/training_variants\", delimiter= ',')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "test_text = pd.read_csv(\"../Data/test_text\", delimiter='\\|\\|', engine='python')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "test_variants = pd.read_csv(\"../Data/test_variants\", delimiter=',')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "(3321, 2)\n",
      "(3321, 4)\n",
      "(5668, 2)\n",
      "(5668, 3)\n"
     ]
    }
   ],
   "source": [
    "print(training_text.shape)\n",
    "print(training_variants.shape)\n",
    "print(test_text.shape)\n",
    "print(test_variants.shape)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 14,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "training_set = training_text.merge(training_variants, on='ID')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 15,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style>\n",
       "    .dataframe thead tr:only-child th {\n",
       "        text-align: right;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: left;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>ID</th>\n",
       "      <th>Text</th>\n",
       "      <th>Gene</th>\n",
       "      <th>Variation</th>\n",
       "      <th>Class</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>0</td>\n",
       "      <td>Cyclin-dependent kinases (CDKs) regulate a var...</td>\n",
       "      <td>FAM58A</td>\n",
       "      <td>Truncating Mutations</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>1</td>\n",
       "      <td>Abstract Background  Non-small cell lung canc...</td>\n",
       "      <td>CBL</td>\n",
       "      <td>W802*</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>2</td>\n",
       "      <td>Abstract Background  Non-small cell lung canc...</td>\n",
       "      <td>CBL</td>\n",
       "      <td>Q249E</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>3</td>\n",
       "      <td>Recent evidence has demonstrated that acquired...</td>\n",
       "      <td>CBL</td>\n",
       "      <td>N454D</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>4</td>\n",
       "      <td>Oncogenic mutations in the monomeric Casitas B...</td>\n",
       "      <td>CBL</td>\n",
       "      <td>L399V</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "   ID                                               Text    Gene  \\\n",
       "0   0  Cyclin-dependent kinases (CDKs) regulate a var...  FAM58A   \n",
       "1   1   Abstract Background  Non-small cell lung canc...     CBL   \n",
       "2   2   Abstract Background  Non-small cell lung canc...     CBL   \n",
       "3   3  Recent evidence has demonstrated that acquired...     CBL   \n",
       "4   4  Oncogenic mutations in the monomeric Casitas B...     CBL   \n",
       "\n",
       "              Variation  Class  \n",
       "0  Truncating Mutations      1  \n",
       "1                 W802*      2  \n",
       "2                 Q249E      2  \n",
       "3                 N454D      3  \n",
       "4                 L399V      4  "
      ]
     },
     "execution_count": 15,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "training_set.head()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 16,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "test_set = test_text.merge(test_variants, on='ID')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 17,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style>\n",
       "    .dataframe thead tr:only-child th {\n",
       "        text-align: right;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: left;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>ID</th>\n",
       "      <th>Text</th>\n",
       "      <th>Gene</th>\n",
       "      <th>Variation</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>0</td>\n",
       "      <td>2. This mutation resulted in a myeloproliferat...</td>\n",
       "      <td>ACSL4</td>\n",
       "      <td>R570S</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>1</td>\n",
       "      <td>Abstract The Large Tumor Suppressor 1 (LATS1)...</td>\n",
       "      <td>NAGLU</td>\n",
       "      <td>P521L</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>2</td>\n",
       "      <td>Vascular endothelial growth factor receptor (V...</td>\n",
       "      <td>PAH</td>\n",
       "      <td>L333F</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>3</td>\n",
       "      <td>Inflammatory myofibroblastic tumor (IMT) is a ...</td>\n",
       "      <td>ING1</td>\n",
       "      <td>A148D</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>4</td>\n",
       "      <td>Abstract Retinoblastoma is a pediatric retina...</td>\n",
       "      <td>TMEM216</td>\n",
       "      <td>G77A</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "   ID                                               Text     Gene Variation\n",
       "0   0  2. This mutation resulted in a myeloproliferat...    ACSL4     R570S\n",
       "1   1   Abstract The Large Tumor Suppressor 1 (LATS1)...    NAGLU     P521L\n",
       "2   2  Vascular endothelial growth factor receptor (V...      PAH     L333F\n",
       "3   3  Inflammatory myofibroblastic tumor (IMT) is a ...     ING1     A148D\n",
       "4   4   Abstract Retinoblastoma is a pediatric retina...  TMEM216      G77A"
      ]
     },
     "execution_count": 17,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "test_set.head()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 29,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "\" abstract background  non-small cell lung cancer (nsclc) is a heterogeneous group of disorders with a number of genetic and proteomic alterations. c-cbl is an e3 ubiquitin ligase and adaptor molecule important in normal homeostasis and cancer. we determined the genetic variations of c-cbl, relationship to receptor tyrosine kinases (egfr and met), and functionality in nsclc.  methods and findings  using archival formalin-fixed paraffin embedded (ffpe) extracted genomic dna, we show that c-cbl mutations occur in somatic fashion for lung cancers. c-cbl mutations were not mutually exclusive of met or egfr mutations; however they were independent of p53 and kras mutations. in normal/tumor pairwise analysis, there was significant loss of heterozygosity (loh) for the c-cbl locus (22%, n\\u200a=\\u200a8/37) and none of these samples revealed any mutation in the remaining copy of c-cbl. the c-cbl loh also positively correlated with egfr and met mutations observed in the same samples. using select c-cbl somatic mutations such as s80n/h94y, q249e and w802* (obtained from caucasian, taiwanese and african-american samples, respectively) transfected in nsclc cell lines, there was increased cell viability and cell motility.  conclusions  taking the overall mutation rate of c-cbl to be a combination as somatic missense mutation and loh, it is clear that c-cbl is highly mutated in lung cancers and may play an essential role in lung tumorigenesis and metastasis.  go to: introduction in the us alone, each year approximately 219,400 people are diagnosed with lung cancers, out of which more than 145,000 of them succumb to the disease [1]. this number is roughly equivalent to the combined mortality rates of cancers of the breast, prostate, colon, liver, kidney and melanoma [1]. in addition the prognosis is usually poor and the five-year survival rate is less than 15%. there are also significant ethnic differences for lung cancer, and the outcome is worse for blacks compared to whites. gender differences are also striking with women having significantly better prognosis as compared to men. there are a number of genetic alterations that can occur in lung cancer. as an example, in nsclc, mutations in kras, p53, egfr and met have been identified. many of these pathways, especially receptor tyrosine kinases (rtks) are controlled by c-cbl.  cbl (casitas b-lineage lymphoma) is a mammalian gene located on human chromosome 11q23.3 [2] and is involved in cell signaling and protein ubiquitination [3]. cbl proteins belong to the ring finger class of ubiquitin ligases (e3) and there are three homologues c-cbl, cbl-b, cbl-3 [4]. the c-cbl and cbl-b genes are ubiquitously expressed with the highest levels in hematopoietic tissues [5]. c-cbl consists of four regions encoding for functionally distinct protein domains: the n-terminal tyrosine kinase binding (tkb) domain, the linker region, the catalytic ring finger domain, the proline-rich region and the c-terminal ubiquitin-associated (uba) domain that also overlaps with a leucine-zipper (lz) domain [3]. both tkb and ring finger domains are essential for ligand-induced ubiquitination of rtks [6], [7], [8], [9]. the ring finger domain is required for the recruitment of e2 ubiquitin-conjugating enzymes. the tkb domain includes four-helix bundle (4h), a calcium-biding ef hand, and a modified sh2 domain, which binds to phosphotyrosine residues [3], [10], [11], [12]. in addition, the proline-rich region of c-cbl can associate with the sh3 domain of grb2, which can indirectly recruit c-cbl to rtks via the grb2 adaptor protein [7], [13], [14].  c-cbl also binds to egfr and acts as the e3 that targets egfr for ubiquitination and degradation. furthermore, cbl desensitizes egf signaling and opposes cellular proliferation induced by egf [15]. egf activation also appears to activate the tyrosine kinase src, which phosphorylates c-cbl and in turn activates the ubiquitination and degradation of egfr [16], [17], [18]. a recent study shows that defective endocytosis of egfr is characterized by a deletion mutant and the point mutation l858r, whereby its association with c-cbl and subsequent ubiquitination are impaired [19]. recently, the first human c-cbl mutations were reported in acute myeloid leukemia (aml) patients [20]. the mutation r420q inhibits fms-like tyrosine kinase 3 (flt3) internalization and ubiquitination [20].  not only can e3 activity be important in oncogenesis, c-cbl has a dual but separate function as a signal transduction molecule. we have previously shown that c-cbl is important in binding crkl and bcr/abl in hematopoietic cells. also, it can bind and modulate functions of cytoskeleton by binding to proteins like talin and paxillin. the tkb domain is important in binding to a number of molecules, and they then function in signal transduction.  given the critical role of cbl in normal homeostasis and cancer, we hypothesized that it might be mutated in lung cancers. in this study, we report novel c-cbl somatic mutations s80n/h94y, q249e and w802* in caucasian, taiwanese and african-american lung cancer patients, respectively. expressing these mutations in nsclc cell lines lead to increased proliferation and cell motility. we show that c-cbl mutations occur with or without met or egfr mutations but are mutually exclusive of a loh at the c-cbl locus. additionally, c-cbl loh is associated with either met or egfr mutations. we thus hypothesize that c-cbl mutations might contribute to the oncogenic potential of met and egfr in lung cancer.  go to: methods ethics statement  written consent on all research on human subjects has been obtained from the institutional review board, university of chicago and covers all research performed in the laboratory. the following is their contact information: institutional review board, the university of chicago, mcgiffert hall, 5751 s. woodlawn ave., 2nd floor, chicago, il 60637. written informed consents were received from all patients whose tissue samples were used for this study.  tissue samples  lung cancer tissue and paired adjacent normal lung tissues were obtained from 50 caucasian, 29 african-americans and 40 taiwanese nsclc patients who were recruited at the university of chicago hospital (chicago, usa) (caucasian and african-american patients) and taipei veterans general hospital of taiwan (taiwanese patients) after obtaining appropriate institutional review board permission and informed consent from the patients. out of 119 samples, 77 were men, 38 were women and 4 were unknown with age at diagnosis ranging from 47 to 90 years. in terms of tumor types, 53 were adenocarcinoma, 32 were squamous cell carcinoma and 34 were large cell carcinoma. 49 were stage i, 14 were stage ii, 34 were stage iii, and 13 were stage iv (table s1).  cell culture  human non-small cell lung carcinoma cells a549 and h358 were maintained in dmem and rpmi-1640, respectively. human embryonic kidney 293t cells were cultured in dmem. media were supplemented with 10% fetal bovine serum, 100 units/ml of penicillin, and 100 µg/ml of streptomycin (invitrogen, carlsbad, ca). cells were cultured at 37°c in a humidified incubator containing 5% co2.  c-cbl gene mutational analysis  exons 2 to 16 of c-cbl gene were individually amplified by polymerase chain reaction (pcr). primers are listed in table s2. pcr conditions were 1 cycle of 95°c for 5 minutes; 35 cycles of 94°c for 30 seconds, 58°c for 30 seconds and 72°c for 2 minutes; and one cycle of 72°c for 10 minutes. pcr products were treated with exosap-it (usb corporation, cleveland, oh) and sequenced by big-dye terminator chemistry (applied biosystems, foster city, ca). sequencing was performed on the forward coding strand with confirmation of c-cbl alterations performed by sequencing the reverse strand as well. chromatograms were analyzed for mutations using mutation surveyor v2.61 (softgenetics, state college, pa).  plasmid constructs and site-directed mutagenesis  the wild-type c-cbl cdna insert was subcloned into the paltermax expression vector using xhoi and sali restriction enzyme sites (promega, madison, wi). using this parental plasmid paltermax-c-cbl, the tkb domain double mutation (s80n/h94y), the point mutation (q249e), and the c-terminal point mutation w802* of c-cbl were created using the following primers: 5′-gctggcgctaaagaataacccaccttatatcttagac-3′ and 5′-ctaccagatacctaccagtatctccgtactatcttgtc-3′ for the double mutation s80n/h94y; 5′-ctttacccgactctttgagccctggtcctctttgc-3′ for q249e, and 5′-cagctcctcctttggctgattgtctctggatggtgatc-3′ for w802* along with their complementary primers using the quickchange site-directed mutagenesis xl kit (stratagene, la jolla, ca) according to the manufacturer's instructions. the constructs were confirmed for the point mutations by standard dna sequencing of both strands.  loss of heterozygosity (loh) analysis  five microsatellites on chromosome 11 (3 on 11q at or within 200 kb up or downstream of the c-cbl gene and 2 control markers on 11p) were selected for analysis (table s3). established microsatellite markers and respective primer sequences were selected from the geneloc database (http://genecards.weizmann.ac.il/geneloc/index.shtml, weizmann institute of science, rehovot, israel). primers were custom designed and each forward primer was fluorescently labeled at the 5′ end with fam, pet, ned, or vic (applied biosystems). primer annealing temperatures and duplex scores were evaluated with nist primer tools (http://yellow.nist.gov:8444/dnaanalysis/primertoolspage.do; national institute of standards and technology, gaithersburg, md). primers were verified by performing pcr with control dna (isolated from tk6 cells) and resolving the products on agarose gels. bands were visualized with an uv transilluminator. genomic dna was extracted from tumor samples and paired normal lung tissue. primers were grouped into multiplex combinations shown in table s4. marker d11s929 served as an internal control to check for consistency in pcrs and of peaks from capillary electrophoresis. multiplex pcrs were carried out in a volume of 10 µl that contained 1 µl genomic dna (20–50 ng), 0.5 µm of each primer (1.0 µm total for each primer pair), 400 µm dntps, 1x pcr buffer containing mgcl2, and 0.2 u taq dna polymerase. pcr was performed on the abi geneamp 9700 pcr system under the following conditions: 5 min at 94°c; 30 cycles of 30 sec at 94°c, 1 min at 60°c, 1 min at 72°c; and 5 min at 72°c. the pcr products were separated by capillary electrophoresis on an abi 3130xl dna analyzer. chromatograms were analyzed with peak scanner 1.0 and genemapper 3.7 software (applied biosystems) for allelic alterations. the area of the peaks produced by the dna pcr products was quantified for each allele. the ratio of the allelic areas was calculated for each tumor and paired normal dna sample. when the qloh (allelic ratio for the tumor peaks divided by the allelic ratio of paired normal sample) was ≤0.5 or ≥2.0 for c-cbl and at least one other 11q marker in at least two separate experiments, the sample was considered as having an allelic imbalance and interpreted as loh. samples were evaluated in at least two separate experiments and samples showing prospective loh at c-cbl repeated a third time which included a new control marker at the bax locus (data not shown) on chromosome 19 to verify integrity of sample dna.  transfection of c-cbl constructs  the a549 cell line was transfected using the fugene hd (roche, nutley, nj) reagent according to the manufacturer's instructions. eight µg of plasmid dna, containing either no insert (empty vector), wild-type c-cbl, s80n/h94y c-cbl, q249e c-cbl or w802* cbl was used for transfection in a 6-well culture plate. cells were harvested 48 h after transfection and analyzed for expression.  c-cbl knockdown  c-cbl knockdown was performed using lentiviral transduction using mission lentiviral transduction particles (sigma-aldrich, st. louis, mo) as per manufacturer's instructions. briefly, 1×105 h358 cells/well were seeded in 6-well plates and infected the following day with c-cbl lentiviral shrna constructs. to generate stable c-cbl knockdown cell lines, cells were selected for 2 days with 1 µg/ml puromycin. c-cbl levels were determined using whole cell lysates by immunoblotting with anti-cbl antibody (santa cruz biotechnologies, santa cruz, ca).  cell viability assay  cells were transfected as described above in the transfection assay. forty-eight hours after transfection, viability of cells was assessed using trypan blue exclusion.  wound healing assay  a549 cells were seeded in 6-well plates and cultured for 48 h until 100% confluent. the medium was then changed and the cells were transfected as described in the transfection assay. twelve hours after transfection, a straight scratch was made across the cell layer using a 1 ml pipette tip. the cells were then gently washed with 1× pbs to remove cellular debris and the media was replaced. photographs were taken of the wound region every 12 h until 48 h.  western blot analysis  forty eight hours after transfection, cells were collected and washed twice in 1x pbs, then lysed in ice-cold lysis buffer (0.5m tris-hcl with ph 7.4, 1.5 m nacl, 2.5% deoxycholic acid, 10 mm edta, 10% np-40, 0.5 mm dtt, 1 mm phenylmethylsulfonyl fluoride, 5 µg/ml leupeptin, and 10 µg/ml aprotinin) for 5 minutes. the lysate was centrifuged at 13,000 rpm for 20 minutes at 4°c, and protein content of the supernatant was measured. total cell lysates (50 µg/well) were separated by sds-page electrophoresis and the gels transferred onto nitrocellulose membranes (whatman, piscataway, nj). membranes were blocked with 5% non-fat dry milk in phosphate-buffered saline containing tween-20 (pbst) (1x pbs, 0.1% tween-20) for 1 h at room temperature and incubated with the appropriate primary antibody at 4°c overnight. membranes then were washed three times with pbst and probed with appropriate horseradish peroxidase (hrp)-conjugated secondary antibody for 1 h at room temperature. the membranes were again washed three times in pbst and bands were visualized using western blot chemiluminescence reagent (biorad, valencia, ca) on a chemidoc gel documentation system (biorad, valencia, ca). antibodies were obtained from santa cruz biotechnologies and used at the following dilutions (c-cbl, 1∶5000; c-met, 1∶5000; egfr, 1∶5000; ubiquitin, 1∶1000; ha, 1∶5000 and β-actin, 1∶10,000).  flow cytometry  cell cycle analysis was carried out by flow cytometry. approximately 2×106 cells were grown in media containing 10% fbs. cells were harvested by trypsin/edta treatment, washed with 1x pbs three times and fixed with ice-cold 70% ethanol for 2 h. cells were washed again with cold pbs and stained with a solution containing 25 µg/ml propidium iodide, 200 µg/ml rnase a, and 0.1% triton x-100 for 30 minutes in the dark. cell cycle analysis was performed using a guava pca-96 flow cytometer (guava technologies, millipore, billerica, ma).  ubiquitin ligase activity  293t cells were maintained in culture in dmem supplemented with 10% fbs and 1% penicillin (100 units/ml) and streptomycin (100 µg/ml) were transfected with 0.2 µg egfr-pcdna3 and 2 µg ha-tagged c-cbl constructs as indicated using calcium phosphate according to manufacturer's protocol (profection, promega, madison, wi). twenty-four hours post-transfection, cells were starved overnight in dmem supplemented with 0.5% fbs, and then treated with or without egf (100 ng/ml) for 15 min. the cells were collected and washed two times in ice-cold pbs containing 0.2 mm sodium orthovanadate then lysed in ice-cold lysis buffer (10 mm tris hcl, ph 7.5, 150 mm nacl, 5 mm edta, 1% triton x100, 10% glycerol, 2 mm sodium orthovanadate and protease inhibitors). lysates were cleared of debris by centrifugation at 16,000 g for 10 min at 4°c. egfr immunoprecipitations were performed on 200 µg of cleared lysate using 250 ng of rabbit-anti-egfr and protein a/g plus sepharose overnight at 4°c. precipitations were washed 5 times in lysis buffer before boiling in laemmli buffer. elutions were immunoblotted with anti-ubiquitin and egfr. twenty micrograms of cleared lysate were immunoblotted for each of the c-cbl constructs using anti-ha.  statistical analysis  mutation rates between different groups were compared using fisher's exact test. for continuous variables, group comparisons were performed using analysis of variance (anova) followed by sidak's adjustment for multiple comparisons. experiments involving repeated measurements over time were analyzed using repeated measures anova with the greenhouse-geisser adjustment to the degrees of freedom. analyses were conducted using stata (v10.1) software (stata corporation, college station, tx).  go to: results c-cbl gene mutations in lung cancer  to investigate the role of c-cbl in lung cancer, we analyzed its genomic dna in tumor and paired normal samples drawn from multiple ethnicities. the lung tumor samples represented caucasians (n\\u200a=\\u200a50), african-americans (n\\u200a=\\u200a29), and taiwanese (n\\u200a=\\u200a40) lung cancer patients. we designed 12 pairs of primers to sequence the coding region of c-cbl gene that spans exons 2 to 16 (table s2). we identified 8 unique somatic mutations in c-cbl exons among 8 different patients. a variation l620f, a known snp (rs2227988) in exon 11 was also detected. importantly, the eight novel non-synonymous mutations were confirmed by sequencing both strands of c-cbl genomic dna obtained from lung tumor samples (table 1). moreover, none of the 8 mutations were detected in the corresponding normal tissue, indicating that these were somatic mutations. four synonymous single nucleotide variations (snvs) were also identified but were not used further in this study.  table 1 table 1 c-cbl mutation analysis in 119 lung cancer patient tumor tissues. three of the 8 novel non-synonymous mutations were located in the tkb (tyrosine kinase binding) domain (s80n, h94y, and q249e), one in the ring finger domain (v391i), one in the proline-rich region (72515_72517 del atg), and three in the c-terminal region (w802*, r830k, and a848t) of the c-cbl protein (figure 1a and figure s1). in figure 1b, we show model chromatograms of representative samples.  figure 1 figure 1 c-cbl mutations and loh in non-small cell lung cancer. 11q loh of c-cbl gene  paired lung tumor and normal lung tissue samples from taiwanese patients (n\\u200a=\\u200a37) were investigated for loh. eight (21.6%) showed loh at the c-cbl locus on chromosome 11 while 29 samples (78.4%) revealed normal allelic contribution at the microsatellite markers (figures 1c and d).  c-cbl mutations in different ethnic groups  the c-cbl double mutant s80n/h94y was found in the same patient and the overall mutation rate for c-cbl in lung tumors was 6.7% (8/119). the frequency of c-cbl mutation was highest in large cell carcinoma (14.7%; 5 of 34 patients) followed by squamous carcinoma (6.3%; 2 of 32 patients) and the least was observed in adenocarcinoma (ad) (1.8%; 1 of 53 patients), although these rates were not statistically significant (p\\u200a=\\u200a0.292). mutation rates were 6.0% among caucasians (0 of 20 in ad; 0 of 10 in sq; and 3 of 20 in lc), 13.8% in african-americans (1 of 10 in ad; 1 of 10 in sq; and 2 of 9 in lc), and 2.5% (0 of 23 in ad; 1 of 12 in sq; and 0 of 5 in lc) in the taiwanese population. additionally two taiwanese patients with lung cancer (one squamous and one adenocarcinoma) had the known snp l620f. ethnic differences were not statistically significant, however the power to detect differences was low.  mutations in met and egfr can be co-associated with c-cbl alterations  since east asians with lung cancer have a higher frequency of egfr and met mutations in lung tumors [21], [22], we also determined mutations in egfr and met in the same taiwanese cohort samples and compared the results with the observed c-cbl alterations (loh and/or mutations). in the 37 samples tested, we did not find any overlap between c-cbl mutations and c-cbl loh (figure 2). of the three c-cbl mutants (including the known l620f snp, rs2227988), one of the samples had a met mutation (n375s) and the other had an egfr mutation (l858r). among the 8 samples that had a loh at the c-cbl locus 5 had an additional mutation in met (n375s) and 2 had an egfr exon 19 deletion. twenty-six samples had neither c-cbl mutation nor c-cbl loh (3 patients had a c-cbl mutation but no c-cbl loh). among these 26 samples 9 had a met mutation (8 n375s, 1 l211w), 13 had an egfr mutation (7 exon 9 deletion, 6 l858r) and 4 had no other met or egfr mutation. thus the rate of met or egfr mutations among patients with loh at the c-cbl locus (7 of 8) was similar to that seen in patients without c-cbl mutation or loh (22 of 26 patients) (p\\u200a=\\u200a0.99). these 4 patients with no identifiable mutation in c-cbl, met or egfr represented 10.8% of the 37 patients analyzed in the taiwanese patient cohort. conversely, 89.2% taiwanese lung cancer patients have an identifiable mutation in either c-cbl, met or egfr or a combination of the three genes (figure 2). additionally, we determined p53 and kras mutations in these taiwanese cohorts. two p53 and 1 kras mutation were detected. the single kras mutation overlapped with one p53 mutation. this patient also had the egfr exon 19 deletion but had no c-cbl mutation. the other p53 mutation sample had a c-cbl loh with concurrent met n375s mutation. thus, in the taiwanese samples analyzed, p53/kras mutations and c-cbl mutations were mutually exclusive (data not shown).  figure 2 figure 2 c-cbl mutations and relationship to met and egfr mutations in lung cancer. cellular functions of c-cbl alterations in the context of lung tumorigenesis  a. e3 activity is intact in the mutant c-cbl proteins to investigate whether the different c-cbl mutations affect the e3 activity, egfr was chosen as a model substrate for c-cbl e3 function. all of the c-cbl mutants tested enhanced ubiquitination of the activated egfr similar to the wild-type c-cbl protein. this result demonstrates that the catalytic activity of the c-cbl mutants is not impaired when egfr was the substrate. (figure 3a).  figure 3 figure 3 ubiquitination, viability, expression and cell cycle analysis of various c-cbl mutants. b. effect on lung cancer cell viability the effect of a representative c-cbl mutant from each of the three ethnic backgrounds on lung cancer cell viability in cell lines was determined. s80n/h94y double mutation, q249e, and w802* were identified in lung tumor samples obtained from a caucasian, a taiwanese and an african-american, respectively. as described in methods, the c-cbl wild-type (wt) and the above three mutants were expressed after cloning them into paltermax vector in a549 cells. these cells express relatively low basal levels of endogenous c-cbl (data not shown). transfection efficiency was comparable between different groups and the number of cells transfected with c-cbl wild-type construct was about 70% compared to control cells that were transfected with the empty vector. cells transfected with s80n/h94y, q249e and w802* c-cbl mutant constructs resulted in increased number of viable cells that was 132.3%, 120.8% and 147.9% higher respectively, relative to the empty vector control transfected cells and significantly different from the wild-type construct (p\\u200a=\\u200a0.022, p\\u200a=\\u200a0.049, and p\\u200a=\\u200a0.008, respectively) (figure 3b). relative levels of c-cbl protein in whole cell lysates prepared from samples obtained from a parallel experiment was determined. the c-cbl protein levels in samples representing untransfected and empty vector transfected cells were comparable and those representing the c-cbl wt and the three c-cbl mutants were comparable (figure 3c).  c. effect on cell cycle to investigate if the increases in cell viability in different c-cbl mutants are due to increased cellular proliferation, a cell cycle analysis was performed. a549 cells were transfected with the c-cbl wt or the three different mutants: s80n/h94y, q249e and w802*. the empty vector transfectant was used as a control. forty-eight hours after transfection cell cycle analysis was performed as described in materials and methods. there was no significant change in the subg1, g1 or the s phase of the cell cycle among the different mutants compared to the wt construct (p\\u200a=\\u200a0.64, p\\u200a=\\u200a0.40, and p\\u200a=\\u200a0.28, respectively). the g2/m phase of the cell cycle showed an increase in cell numbers for the three mutants, s80n/h94y, q249e and w802*, when compared to the wt but again the difference was not statistically significant (p\\u200a=\\u200a0.25) (figure 3d).  d. effect on cell motility to investigate the effect of the expression of the above three c-cbl mutants on cell migration, we carried out wound healing assay as described in materials and methods. the closing of the scratch or the wound was monitored at 0, 12, 24, 36, and 48 h. (figure 4a). in all the samples, that represented cells transfected with mutants, the wound gap was much smaller than that seen in the sample that represented cells transfected with c-cbl wt (p<0.001). we also determined the rate of wound closure for all the five groups. at 48 h, wild-type c-cbl transfectants showed 61.1% open wound while the s80n/h94y, q249e and w802* mutants showed 18.7%, 23.9% and 34.3% open wound respectively (p<0.001) (figure 4b).  figure 4 figure 4 c-cbl mutations affect wound healing in a549 cells. e. c-cbl knockdown increases cell viability it is hypothesized that the loh seen in our samples could lead to decreased expression of c-cbl. thus we tested the effect of c-cbl knockdown in lung cancer cells. compared to a549, h358 lung cancer cells express relatively high levels of endogenous c-cbl (data not shown). c-cbl expression was knocked down using lentiviral construct that expressed c-cbl specific shrna and compared the results with those that were transduced with scrambled shrna and the results are shown in figure 5. we identified several clones that revealed varying degrees of c-cbl knockdown showing different sets of c-cbl lentiviral shrna knockdown efficiency (figure 5a). of all the clones tested, clone 27 was chosen for further experiments. equal amount of cells were seeded in a 6-well plate and the cell proliferation was measured at various times and the results are depicted in figure 5b. as expected, number of cells increased in a time dependent fashion from 100 to 190% relative to scrambled shrna as control in a span of 48 h (p\\u200a=\\u200a0.0002) (figure 5b). the cell cycle phases in h358 cells that were knocked down with c-cbl shrna were looked at and compared with the scrambled shrna. there were no discernable differences between these two constructs in the different phases of the cell cycle (data not shown).  figure 5 figure 5 knockdown of c-cbl using an shrna increases cell proliferation. go to: discussion our results demonstrate that c-cbl is somatically mutated (or has loh) in lung cancers, and can significantly contribute to enhanced cell viability and motility. there was also a high prevalence of loh with respect to c-cbl in lung tumors that harbored met or egfr mutation.  in the present study, we have demonstrated the occurrence of c-cbl mutations in lung cancer patients, especially with different ancestral variations. mutations in c-cbl have been recently reported in juvenile myelomonocytic leukemia and myeloid malignancies. in the aml study, the mutation r420q located in the junction of the ring finger and the linker region inhibited fms-like tyrosine kinase 3 (flt3) internalization and ubiquitination [20], thus contributing to the gain-in-function for the rtk. in addition, mutations such as h398y, c384r, and l380p mapped to the ring finger domain and the linker region of c-cbl that is required for its e3 activity [23], [24], [25], [26], [27]. additionally, homozygous mutations in the ring finger domain of the c-cbl gene were described as a result of acquired uniparental disomy (upd) [26]. it is important to note that our results indicate loh at 11q23 locus and these are mutually exclusive from missense mutations of c-cbl. the somatic mutations were all heterozygous. the mutations in aml led to abrogation of the e3 activity leading to prolonged rtk activation. in addition mutants located on the linker region surrounding the ring finger domain exhibited enhanced akt signaling in response to cytokine stimulation [26]. in addition, it was shown in nh3t3 cells, that neither mutations in the ring finger nor the linker region causes transformation, however while certain mutations perturbs the ubiquitination, others affect receptor recycling and prolong kinase activity [28].  we report here c-cbl mutations that mapped not only to the ring finger domain, but also to the tkb domain, proline-rich domain and the c-terminal region, but none mapped to the linker region as reported in the aml studies described above [23], [24], [25], [26], [29]. in addition, 8 mutants that we detected were found in different ethnic backgrounds. for example, s80n/h94y, q249e, w802* were detected in caucasians, taiwanese and african-americans, respectively. the results point out not only the difference between lung cancer and other cancers, but also genetic polymorphism among different races in the same cancer. interestingly, there is a large disparity between african-american and other ethnic populations with lung cancer [30]. we have previously shown that there was a low frequency of egfr and met mutation in african-americans as compared with taiwanese and caucasians [31]. in this study, the number of african-american samples analyzed was relatively fewer and we found 3 mutations that are unique to this ethnicity. it would now behoove us to further study the genetic alterations that can occur and determine the targeted therapeutics for african-americans. our results provide evidence of the importance of c-cbl in tumorigenesis and potential signaling. our prediction, based on the aml data, would be that the v391i ring finger domain mutation would affect the e3 activity. also, it will be important to determine the binding partners for c-cbl in the tkb domain and proline rich domain mutations. it has previously been shown that the tkb domain can bind to growth factor receptors and it will be important to determine the cross-binding of these mutants to met and or egfr. it would also be important in the future to look at fluorescence in situ hybridization/copy number changes in c-cbl in lung cancer.  c-cbl plays an important role in down regulating rtk-mediated signaling through k63 poly-ubiquitination and subsequent downregulation of rtks followed by lysosomal degredation [3]. mono-ubiquitination or ubiquitinated with k63-linked chains of substrates by c-cbl may lead to enhancement of biological and biochemical functions (reviewed in hermann et al, 2007 [32]). the mutations that we analyzed in our studies all point out to the fact that the e3 activity of c-cbl on egfr is intact; the egfr levels in the various mutants remain same (figure s2). multiple kinases, both rtks and non-rtks could be acted upon by c-cbl, including erbs, pdgfr, fms, met, c-kit, vegfr, flt-1, ron, fgfr, ir, as well as syk, fyn, lck, fgr, lyn and c-abl [3]. in lung cancers the relevant substrates of c-cbl in terms of degradation or signal transduction are yet to be identified.  the observation that c-cbl somatic mutations, especially s80n/h94y, q249e and w802* showed increased cell viability and cell motility that is in agreement with the physiological role for d-cbl in the regulation of apoptosis and differentiation identified in drosophila is very significant [33]. it has been previously shown that activating c-cbl mutation downregulates egfr signaling and decreases cellular proliferation and migration in breast cancer cell lines [34]. although the role of c-cbl in the negative regulation of rtks is well substantiated, thereby suggesting that it is a natural tumor suppressor, studies in cancer cells have revealed both tumor suppressor and tumor promoting activities depending on the type of c-cbl mutation and the number of alleles at the c-cbl locus [24]. in agreement with the above, the three c-cbl mutants described here appear to have tumor growth and metastasis promoting properties. although these mutants are outside of the ring finger or the linker region of c-cbl, their downstream effects are significant so as to cause increased proliferation and migration, but the substrate affected by these mutations are not known yet. this raises the possibility that some of the cellular functions of c-cbl are independent of its ubiquitin-ligase activity, an area that we are currently investigating. the oncogenic nature of rtks, addiction of cancers to growth signals and given the clustering of c-cbl, egfr and met mutations, it is possible that the transforming effect of c-cbl mutations is most likely a combinatorial effect of the three. we also show that loh for c-cbl was found in a significant number of samples that harbored met or egfr mutations. the fact that about 7% of lung tumor samples are likely to have c-cbl mutations and an additional 22% are likely to harbor c-cbl-related loh, makes c-cbl a highly mutated molecule in lung cancer. since loh alone is not enough to cause a transforming event [35], [36], [37], associated mutation in the met or egfr locus or yet another rtk discussed above may play a role in carcinogenesis. we predict that this loh in c-cbl results in haploinsufficiency that downplays rtk ubiquitination leading to hyperactivity of the rtks. however, whether this is sufficient cause for tumorigenesis remains to be determined. consistent with our hypothesis is the fact that c-cbl-/- mice have increased kinase activity in lymphocytes, but is not sufficient for tumor formation [35], [36], [37]. c-cbl loh could also lead to increased expression of c-cbl from the other allele to compensate for a loss of an allele. alternately, there could be some form of synergy working with reduced c-cbl levels and mutated receptors that exacerbate the phenotype of each alone.  previous studies from our lab and others have shown that east asians with lung cancers have relatively high frequencies of gain-of-function of mutations in rtks such as egfr and met [31]. in a cohort of japanese patients an activating met mutation was identified in the splice region that deletes the juxtamembrane domain that is involved e3 activity of c-cbl [38]. this study also found that activation of met is mutually exclusive of egfr, kras and her2 gene mutations [38]. we failed to detect such mutations in significant numbers in lung tumor samples obtained from african-americans (n\\u200a=\\u200a29) and caucasian (n\\u200a=\\u200a50) patients. one met mutation was identified in each of the groups whereas 1 and 3 egfr mutations were identified in the african-american and caucasian cohorts respectively. egfr mutations have earlier been identified as one of the key mutations affecting lung adenocarcinoma patients in a comprehensive study of 188 patients [39]. our study encompasses different histologies of nsclc. however, the published series did not find any mutations in c-cbl or met unlike our study that encompassed different subtypes of nsclc. it is important to note, we have recently shown that met mutations in lung cancer are in majority germline [31]. we have reported earlier c-cbl mutations in a small cohort of taiwanese lung cancer samples [40]. in our efforts to understand the ethnic differences in the lung oncogenome, we also looked at pax transcription factors such as pax5 and pax8 that are highly expressed in lung cancers; however there was no preferential expression or mutations of the above genes in lung tumor samples of african-americans. in this study, we show relatively high frequency of c-cbl mutations in lung cancers, especially in the large cell type among caucasians and particularly among african-americans. we therefore propose c-cbl as an efficacious target for lung cancers in african-americans that needs to be further substantiated. this is all the more important because the prognosis for african-americans with lung cancer, especially for men is much poorer compared to their caucasian counterparts [41].  in conclusion, the results presented in this study demonstrate that c-cbl is frequently mutated or even lost in lung cancers. our results support a role for c-cbl mutants that are independent of its ubiquitination activity. given the relatively high mutation rates in c-cbl as well as rtks such as met and egfr, it is likely that their combined effect could be synergistic in promoting tumorigenesis.\""
      ]
     },
     "execution_count": 29,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "training_set['Text'].iloc[1].lower()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [
    {
     "ename": "ModuleNotFoundError",
     "evalue": "No module named 'gensim'",
     "output_type": "error",
     "traceback": [
      "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
      "\u001b[0;31mModuleNotFoundError\u001b[0m                       Traceback (most recent call last)",
      "\u001b[0;32m<ipython-input-4-ec15f3ffe0f1>\u001b[0m in \u001b[0;36m<module>\u001b[0;34m()\u001b[0m\n\u001b[0;32m----> 1\u001b[0;31m \u001b[0;32mimport\u001b[0m \u001b[0mgensim\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m",
      "\u001b[0;31mModuleNotFoundError\u001b[0m: No module named 'gensim'"
     ]
    }
   ],
   "source": [
    "import gensim"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.6.4"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
