{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [],
   "source": [
    "import pandas as pd\n",
    "import gensim, itertools\n",
    "data = training_text = pd.read_csv(\"../Data/training_text\", delimiter='\\|\\|', engine = 'python')\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [],
   "source": [
    "data['Text'].replace('[!\"#%\\'()*+,-./:;<=>?@\\[\\]^_`{|}~1234567890’”“′‘\\\\\\]',' ',inplace=True,regex=True)\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [],
   "source": [
    "sentences = list(itertools.chain(data['Text'].str.split(' ')))\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "['Cyclin',\n",
       " 'dependent',\n",
       " 'kinases',\n",
       " '',\n",
       " 'CDKs',\n",
       " '',\n",
       " 'regulate',\n",
       " 'a',\n",
       " 'variety',\n",
       " 'of',\n",
       " 'fundamental',\n",
       " 'cellular',\n",
       " 'processes',\n",
       " '',\n",
       " 'CDK',\n",
       " '',\n",
       " '',\n",
       " 'stands',\n",
       " 'out',\n",
       " 'as',\n",
       " 'one',\n",
       " 'of',\n",
       " 'the',\n",
       " 'last',\n",
       " 'orphan',\n",
       " 'CDKs',\n",
       " 'for',\n",
       " 'which',\n",
       " 'no',\n",
       " 'activating',\n",
       " 'cyclin',\n",
       " 'has',\n",
       " 'been',\n",
       " 'identified',\n",
       " 'and',\n",
       " 'no',\n",
       " 'kinase',\n",
       " 'activity',\n",
       " 'revealed',\n",
       " '',\n",
       " 'Previous',\n",
       " 'work',\n",
       " 'has',\n",
       " 'shown',\n",
       " 'that',\n",
       " 'CDK',\n",
       " '',\n",
       " '',\n",
       " 'silencing',\n",
       " 'increases',\n",
       " 'ETS',\n",
       " '',\n",
       " '',\n",
       " 'v',\n",
       " 'ets',\n",
       " 'erythroblastosis',\n",
       " 'virus',\n",
       " 'E',\n",
       " '',\n",
       " '',\n",
       " 'oncogene',\n",
       " 'homolog',\n",
       " '',\n",
       " '',\n",
       " '',\n",
       " 'driven',\n",
       " 'activation',\n",
       " 'of',\n",
       " 'the',\n",
       " 'MAPK',\n",
       " 'pathway',\n",
       " '',\n",
       " 'which',\n",
       " 'confers',\n",
       " 'tamoxifen',\n",
       " 'resistance',\n",
       " 'to',\n",
       " 'breast',\n",
       " 'cancer',\n",
       " 'cells',\n",
       " '',\n",
       " 'The',\n",
       " 'precise',\n",
       " 'mechanisms',\n",
       " 'by',\n",
       " 'which',\n",
       " 'CDK',\n",
       " '',\n",
       " '',\n",
       " 'modulates',\n",
       " 'ETS',\n",
       " '',\n",
       " 'activity',\n",
       " '',\n",
       " 'and',\n",
       " 'more',\n",
       " 'generally',\n",
       " 'the',\n",
       " 'functions',\n",
       " 'of',\n",
       " 'CDK',\n",
       " '',\n",
       " '',\n",
       " '',\n",
       " 'remain',\n",
       " 'elusive',\n",
       " '',\n",
       " 'Here',\n",
       " 'we',\n",
       " 'demonstrate',\n",
       " 'that',\n",
       " 'CDK',\n",
       " '',\n",
       " '',\n",
       " 'is',\n",
       " 'a',\n",
       " 'cyclin',\n",
       " 'dependent',\n",
       " 'kinase',\n",
       " 'by',\n",
       " 'identifying',\n",
       " 'cyclin',\n",
       " 'M',\n",
       " 'as',\n",
       " 'an',\n",
       " 'activating',\n",
       " 'cyclin',\n",
       " '',\n",
       " 'Cyclin',\n",
       " 'M',\n",
       " '',\n",
       " 'an',\n",
       " 'orphan',\n",
       " 'cyclin',\n",
       " '',\n",
       " 'is',\n",
       " 'the',\n",
       " 'product',\n",
       " 'of',\n",
       " 'FAM',\n",
       " '',\n",
       " 'A',\n",
       " '',\n",
       " 'whose',\n",
       " 'mutations',\n",
       " 'cause',\n",
       " 'STAR',\n",
       " 'syndrome',\n",
       " '',\n",
       " 'a',\n",
       " 'human',\n",
       " 'developmental',\n",
       " 'anomaly',\n",
       " 'whose',\n",
       " 'features',\n",
       " 'include',\n",
       " 'toe',\n",
       " 'syndactyly',\n",
       " '',\n",
       " 'telecanthus',\n",
       " '',\n",
       " 'and',\n",
       " 'anogenital',\n",
       " 'and',\n",
       " 'renal',\n",
       " 'malformations',\n",
       " '',\n",
       " 'We',\n",
       " 'show',\n",
       " 'that',\n",
       " 'STAR',\n",
       " 'syndrome',\n",
       " 'associated',\n",
       " 'cyclin',\n",
       " 'M',\n",
       " 'mutants',\n",
       " 'are',\n",
       " 'unable',\n",
       " 'to',\n",
       " 'interact',\n",
       " 'with',\n",
       " 'CDK',\n",
       " '',\n",
       " '',\n",
       " '',\n",
       " 'Cyclin',\n",
       " 'M',\n",
       " 'silencing',\n",
       " 'phenocopies',\n",
       " 'CDK',\n",
       " '',\n",
       " '',\n",
       " 'silencing',\n",
       " 'in',\n",
       " 'increasing',\n",
       " 'c',\n",
       " 'Raf',\n",
       " 'and',\n",
       " 'in',\n",
       " 'conferring',\n",
       " 'tamoxifen',\n",
       " 'resistance',\n",
       " 'to',\n",
       " 'breast',\n",
       " 'cancer',\n",
       " 'cells',\n",
       " '',\n",
       " 'CDK',\n",
       " '',\n",
       " '',\n",
       " 'cyclin',\n",
       " 'M',\n",
       " 'phosphorylates',\n",
       " 'ETS',\n",
       " '',\n",
       " 'in',\n",
       " 'vitro',\n",
       " '',\n",
       " 'and',\n",
       " 'in',\n",
       " 'cells',\n",
       " 'it',\n",
       " 'positively',\n",
       " 'controls',\n",
       " 'ETS',\n",
       " '',\n",
       " 'degradation',\n",
       " 'by',\n",
       " 'the',\n",
       " 'proteasome',\n",
       " '',\n",
       " 'ETS',\n",
       " '',\n",
       " 'protein',\n",
       " 'levels',\n",
       " 'are',\n",
       " 'increased',\n",
       " 'in',\n",
       " 'cells',\n",
       " 'derived',\n",
       " 'from',\n",
       " 'a',\n",
       " 'STAR',\n",
       " 'patient',\n",
       " '',\n",
       " 'and',\n",
       " 'this',\n",
       " 'increase',\n",
       " 'is',\n",
       " 'attributable',\n",
       " 'to',\n",
       " 'decreased',\n",
       " 'cyclin',\n",
       " 'M',\n",
       " 'levels',\n",
       " '',\n",
       " 'Altogether',\n",
       " '',\n",
       " 'our',\n",
       " 'results',\n",
       " 'reveal',\n",
       " 'an',\n",
       " 'additional',\n",
       " 'regulatory',\n",
       " 'mechanism',\n",
       " 'for',\n",
       " 'ETS',\n",
       " '',\n",
       " '',\n",
       " 'which',\n",
       " 'plays',\n",
       " 'key',\n",
       " 'roles',\n",
       " 'in',\n",
       " 'cancer',\n",
       " 'and',\n",
       " 'development',\n",
       " '',\n",
       " 'They',\n",
       " 'also',\n",
       " 'shed',\n",
       " 'light',\n",
       " 'on',\n",
       " 'the',\n",
       " 'molecular',\n",
       " 'mechanisms',\n",
       " 'underlying',\n",
       " 'STAR',\n",
       " 'syndrome',\n",
       " 'Cyclin',\n",
       " 'dependent',\n",
       " 'kinases',\n",
       " '',\n",
       " 'CDKs',\n",
       " '',\n",
       " 'play',\n",
       " 'a',\n",
       " 'pivotal',\n",
       " 'role',\n",
       " 'in',\n",
       " 'the',\n",
       " 'control',\n",
       " 'of',\n",
       " 'a',\n",
       " 'number',\n",
       " 'of',\n",
       " 'fundamental',\n",
       " 'cellular',\n",
       " 'processes',\n",
       " '',\n",
       " '',\n",
       " '',\n",
       " '',\n",
       " '',\n",
       " 'The',\n",
       " 'human',\n",
       " 'genome',\n",
       " 'contains',\n",
       " '',\n",
       " '',\n",
       " '',\n",
       " 'genes',\n",
       " 'encoding',\n",
       " 'proteins',\n",
       " 'that',\n",
       " 'can',\n",
       " 'be',\n",
       " 'considered',\n",
       " 'as',\n",
       " 'members',\n",
       " 'of',\n",
       " 'the',\n",
       " 'CDK',\n",
       " 'family',\n",
       " 'owing',\n",
       " 'to',\n",
       " 'their',\n",
       " 'sequence',\n",
       " 'similarity',\n",
       " 'with',\n",
       " 'bona',\n",
       " 'fide',\n",
       " 'CDKs',\n",
       " '',\n",
       " 'those',\n",
       " 'known',\n",
       " 'to',\n",
       " 'be',\n",
       " 'activated',\n",
       " 'by',\n",
       " 'cyclins',\n",
       " '',\n",
       " '',\n",
       " '',\n",
       " '',\n",
       " '',\n",
       " 'Although',\n",
       " 'discovered',\n",
       " 'almost',\n",
       " '',\n",
       " '',\n",
       " '',\n",
       " 'y',\n",
       " 'ago',\n",
       " '',\n",
       " '',\n",
       " '',\n",
       " '',\n",
       " '',\n",
       " '',\n",
       " '',\n",
       " '',\n",
       " 'CDK',\n",
       " '',\n",
       " '',\n",
       " 'remains',\n",
       " 'one',\n",
       " 'of',\n",
       " 'the',\n",
       " 'two',\n",
       " 'CDKs',\n",
       " 'without',\n",
       " 'an',\n",
       " 'identified',\n",
       " 'cyclin',\n",
       " 'partner',\n",
       " '',\n",
       " 'This',\n",
       " 'knowledge',\n",
       " 'gap',\n",
       " 'has',\n",
       " 'largely',\n",
       " 'impeded',\n",
       " 'the',\n",
       " 'exploration',\n",
       " 'of',\n",
       " 'its',\n",
       " 'biological',\n",
       " 'functions',\n",
       " '',\n",
       " 'CDK',\n",
       " '',\n",
       " '',\n",
       " 'can',\n",
       " 'act',\n",
       " 'as',\n",
       " 'a',\n",
       " 'positive',\n",
       " 'cell',\n",
       " 'cycle',\n",
       " 'regulator',\n",
       " 'in',\n",
       " 'some',\n",
       " 'cells',\n",
       " '',\n",
       " '',\n",
       " '',\n",
       " '',\n",
       " '',\n",
       " '',\n",
       " '',\n",
       " 'or',\n",
       " 'as',\n",
       " 'a',\n",
       " 'tumor',\n",
       " 'suppressor',\n",
       " 'in',\n",
       " 'others',\n",
       " '',\n",
       " '',\n",
       " '',\n",
       " '',\n",
       " '',\n",
       " '',\n",
       " '',\n",
       " '',\n",
       " 'CDK',\n",
       " '',\n",
       " '',\n",
       " 'interacts',\n",
       " 'with',\n",
       " 'the',\n",
       " 'ETS',\n",
       " '',\n",
       " '',\n",
       " 'v',\n",
       " 'ets',\n",
       " 'erythroblastosis',\n",
       " 'virus',\n",
       " 'E',\n",
       " '',\n",
       " '',\n",
       " 'oncogene',\n",
       " 'homolog',\n",
       " '',\n",
       " '',\n",
       " '',\n",
       " 'transcription',\n",
       " 'factor',\n",
       " 'and',\n",
       " 'inhibits',\n",
       " 'its',\n",
       " 'transcriptional',\n",
       " 'activity',\n",
       " 'through',\n",
       " 'an',\n",
       " 'unknown',\n",
       " 'mechanism',\n",
       " '',\n",
       " '',\n",
       " '',\n",
       " '',\n",
       " '',\n",
       " 'CDK',\n",
       " '',\n",
       " '',\n",
       " 'knockdown',\n",
       " 'derepresses',\n",
       " 'ETS',\n",
       " '',\n",
       " '',\n",
       " 'which',\n",
       " 'increases',\n",
       " 'the',\n",
       " 'expression',\n",
       " 'of',\n",
       " 'the',\n",
       " 'c',\n",
       " 'Raf',\n",
       " 'protein',\n",
       " 'kinase',\n",
       " '',\n",
       " 'activates',\n",
       " 'the',\n",
       " 'MAPK',\n",
       " 'pathway',\n",
       " '',\n",
       " 'and',\n",
       " 'induces',\n",
       " 'resistance',\n",
       " 'of',\n",
       " 'MCF',\n",
       " '',\n",
       " 'cells',\n",
       " 'to',\n",
       " 'tamoxifen',\n",
       " '',\n",
       " '',\n",
       " '',\n",
       " '',\n",
       " 'Here',\n",
       " '',\n",
       " 'we',\n",
       " 'deorphanize',\n",
       " 'CDK',\n",
       " '',\n",
       " '',\n",
       " 'by',\n",
       " 'identifying',\n",
       " 'cyclin',\n",
       " 'M',\n",
       " '',\n",
       " 'the',\n",
       " 'product',\n",
       " 'of',\n",
       " 'FAM',\n",
       " '',\n",
       " 'A',\n",
       " '',\n",
       " 'as',\n",
       " 'a',\n",
       " 'binding',\n",
       " 'partner',\n",
       " '',\n",
       " 'Mutations',\n",
       " 'in',\n",
       " 'this',\n",
       " 'gene',\n",
       " 'that',\n",
       " 'predict',\n",
       " 'absence',\n",
       " 'or',\n",
       " 'truncation',\n",
       " 'of',\n",
       " 'cyclin',\n",
       " 'M',\n",
       " 'are',\n",
       " 'associated',\n",
       " 'with',\n",
       " 'STAR',\n",
       " 'syndrome',\n",
       " '',\n",
       " 'whose',\n",
       " 'features',\n",
       " 'include',\n",
       " 'toe',\n",
       " 'syndactyly',\n",
       " '',\n",
       " 'telecanthus',\n",
       " '',\n",
       " 'and',\n",
       " 'anogenital',\n",
       " 'and',\n",
       " 'renal',\n",
       " 'malformations',\n",
       " 'in',\n",
       " 'heterozygous',\n",
       " 'females',\n",
       " '',\n",
       " '',\n",
       " '',\n",
       " '',\n",
       " '',\n",
       " '',\n",
       " 'However',\n",
       " '',\n",
       " 'both',\n",
       " 'the',\n",
       " 'functions',\n",
       " 'of',\n",
       " 'cyclin',\n",
       " 'M',\n",
       " 'and',\n",
       " 'the',\n",
       " 'pathogenesis',\n",
       " 'of',\n",
       " 'STAR',\n",
       " 'syndrome',\n",
       " 'remain',\n",
       " 'unknown',\n",
       " '',\n",
       " 'We',\n",
       " 'show',\n",
       " 'that',\n",
       " 'a',\n",
       " 'recombinant',\n",
       " 'CDK',\n",
       " '',\n",
       " '',\n",
       " 'cyclin',\n",
       " 'M',\n",
       " 'heterodimer',\n",
       " 'is',\n",
       " 'an',\n",
       " 'active',\n",
       " 'protein',\n",
       " 'kinase',\n",
       " 'that',\n",
       " 'phosphorylates',\n",
       " 'ETS',\n",
       " '',\n",
       " 'in',\n",
       " 'vitro',\n",
       " '',\n",
       " 'Cyclin',\n",
       " 'M',\n",
       " 'silencing',\n",
       " 'phenocopies',\n",
       " 'CDK',\n",
       " '',\n",
       " '',\n",
       " 'silencing',\n",
       " 'in',\n",
       " 'increasing',\n",
       " 'c',\n",
       " 'Raf',\n",
       " 'and',\n",
       " 'phospho',\n",
       " 'ERK',\n",
       " 'expression',\n",
       " 'levels',\n",
       " 'and',\n",
       " 'in',\n",
       " 'inducing',\n",
       " 'tamoxifen',\n",
       " 'resistance',\n",
       " 'in',\n",
       " 'estrogen',\n",
       " 'receptor',\n",
       " '',\n",
       " 'ER',\n",
       " '',\n",
       " '',\n",
       " 'breast',\n",
       " 'cancer',\n",
       " 'cells',\n",
       " '',\n",
       " 'We',\n",
       " 'show',\n",
       " 'that',\n",
       " 'CDK',\n",
       " '',\n",
       " '',\n",
       " 'cyclin',\n",
       " 'M',\n",
       " 'positively',\n",
       " 'controls',\n",
       " 'ETS',\n",
       " '',\n",
       " 'degradation',\n",
       " 'by',\n",
       " 'the',\n",
       " 'proteasome',\n",
       " '',\n",
       " 'through',\n",
       " 'the',\n",
       " 'phosphorylation',\n",
       " 'of',\n",
       " 'two',\n",
       " 'neighboring',\n",
       " 'serines',\n",
       " '',\n",
       " 'Finally',\n",
       " '',\n",
       " 'we',\n",
       " 'detect',\n",
       " 'an',\n",
       " 'increased',\n",
       " 'ETS',\n",
       " '',\n",
       " 'expression',\n",
       " 'level',\n",
       " 'in',\n",
       " 'cells',\n",
       " 'derived',\n",
       " 'from',\n",
       " 'a',\n",
       " 'STAR',\n",
       " 'patient',\n",
       " '',\n",
       " 'and',\n",
       " 'we',\n",
       " 'demonstrate',\n",
       " 'that',\n",
       " 'it',\n",
       " 'is',\n",
       " 'attributable',\n",
       " 'to',\n",
       " 'the',\n",
       " 'decreased',\n",
       " 'cyclin',\n",
       " 'M',\n",
       " 'expression',\n",
       " 'level',\n",
       " 'observed',\n",
       " 'in',\n",
       " 'these',\n",
       " 'cells',\n",
       " 'Previous',\n",
       " 'SectionNext',\n",
       " 'SectionResultsA',\n",
       " 'yeast',\n",
       " 'two',\n",
       " 'hybrid',\n",
       " '',\n",
       " 'Y',\n",
       " 'H',\n",
       " '',\n",
       " 'screen',\n",
       " 'unveiled',\n",
       " 'an',\n",
       " 'interaction',\n",
       " 'signal',\n",
       " 'between',\n",
       " 'CDK',\n",
       " '',\n",
       " '',\n",
       " 'and',\n",
       " 'a',\n",
       " 'mouse',\n",
       " 'protein',\n",
       " 'whose',\n",
       " 'C',\n",
       " 'terminal',\n",
       " 'half',\n",
       " 'presents',\n",
       " 'a',\n",
       " 'strong',\n",
       " 'sequence',\n",
       " 'homology',\n",
       " 'with',\n",
       " 'the',\n",
       " 'human',\n",
       " 'FAM',\n",
       " '',\n",
       " 'A',\n",
       " 'gene',\n",
       " 'product',\n",
       " '',\n",
       " 'whose',\n",
       " 'proposed',\n",
       " 'name',\n",
       " 'is',\n",
       " 'cyclin',\n",
       " 'M',\n",
       " '',\n",
       " '',\n",
       " '',\n",
       " '',\n",
       " '',\n",
       " '',\n",
       " '',\n",
       " 'We',\n",
       " 'thus',\n",
       " 'performed',\n",
       " 'Y',\n",
       " 'H',\n",
       " 'mating',\n",
       " 'assays',\n",
       " 'to',\n",
       " 'determine',\n",
       " 'whether',\n",
       " 'human',\n",
       " 'CDK',\n",
       " '',\n",
       " '',\n",
       " 'interacts',\n",
       " 'with',\n",
       " 'human',\n",
       " 'cyclin',\n",
       " 'M',\n",
       " '',\n",
       " 'Fig',\n",
       " '',\n",
       " '',\n",
       " '',\n",
       " 'A–C',\n",
       " '',\n",
       " '',\n",
       " 'The',\n",
       " 'longest',\n",
       " 'CDK',\n",
       " '',\n",
       " '',\n",
       " 'isoform',\n",
       " '',\n",
       " 'P',\n",
       " '',\n",
       " '',\n",
       " 'expressed',\n",
       " 'as',\n",
       " 'a',\n",
       " 'bait',\n",
       " 'protein',\n",
       " 'produced',\n",
       " 'a',\n",
       " 'strong',\n",
       " 'interaction',\n",
       " 'phenotype',\n",
       " 'with',\n",
       " 'full',\n",
       " 'length',\n",
       " 'cyclin',\n",
       " 'M',\n",
       " '',\n",
       " 'expressed',\n",
       " 'as',\n",
       " 'a',\n",
       " 'prey',\n",
       " 'protein',\n",
       " '',\n",
       " 'but',\n",
       " 'no',\n",
       " 'detectable',\n",
       " 'phenotype',\n",
       " 'with',\n",
       " 'cyclin',\n",
       " 'D',\n",
       " '',\n",
       " '',\n",
       " 'p',\n",
       " '',\n",
       " '',\n",
       " '',\n",
       " 'CIP',\n",
       " '',\n",
       " '',\n",
       " '',\n",
       " 'and',\n",
       " 'Cdi',\n",
       " '',\n",
       " '',\n",
       " 'KAP',\n",
       " '',\n",
       " '',\n",
       " 'which',\n",
       " 'are',\n",
       " 'known',\n",
       " 'binding',\n",
       " 'partners',\n",
       " 'of',\n",
       " 'other',\n",
       " 'CDKs',\n",
       " '',\n",
       " 'Fig',\n",
       " '',\n",
       " '',\n",
       " 'B',\n",
       " '',\n",
       " '',\n",
       " 'CDK',\n",
       " '',\n",
       " 'and',\n",
       " 'CDK',\n",
       " '',\n",
       " 'also',\n",
       " 'produced',\n",
       " 'Y',\n",
       " 'H',\n",
       " 'signals',\n",
       " 'with',\n",
       " 'cyclin',\n",
       " 'M',\n",
       " '',\n",
       " 'albeit',\n",
       " 'notably',\n",
       " 'weaker',\n",
       " 'than',\n",
       " 'that',\n",
       " 'observed',\n",
       " 'with',\n",
       " 'CDK',\n",
       " '',\n",
       " '',\n",
       " '',\n",
       " 'Fig',\n",
       " '',\n",
       " '',\n",
       " 'B',\n",
       " '',\n",
       " '',\n",
       " 'An',\n",
       " 'interaction',\n",
       " 'phenotype',\n",
       " 'was',\n",
       " 'also',\n",
       " 'observed',\n",
       " 'between',\n",
       " 'full',\n",
       " 'length',\n",
       " 'cyclin',\n",
       " 'M',\n",
       " 'and',\n",
       " 'CDK',\n",
       " '',\n",
       " '',\n",
       " 'proteins',\n",
       " 'expressed',\n",
       " 'as',\n",
       " 'bait',\n",
       " 'and',\n",
       " 'prey',\n",
       " '',\n",
       " 'respectively',\n",
       " '',\n",
       " 'Fig',\n",
       " '',\n",
       " 'S',\n",
       " 'A',\n",
       " '',\n",
       " '',\n",
       " 'We',\n",
       " 'then',\n",
       " 'tested',\n",
       " 'different',\n",
       " 'isoforms',\n",
       " 'of',\n",
       " 'CDK',\n",
       " '',\n",
       " '',\n",
       " 'and',\n",
       " 'cyclin',\n",
       " 'M',\n",
       " 'originating',\n",
       " 'from',\n",
       " 'alternative',\n",
       " 'gene',\n",
       " 'splicing',\n",
       " '',\n",
       " 'and',\n",
       " 'two',\n",
       " 'truncated',\n",
       " 'cyclin',\n",
       " 'M',\n",
       " 'proteins',\n",
       " 'corresponding',\n",
       " 'to',\n",
       " 'the',\n",
       " 'hypothetical',\n",
       " 'products',\n",
       " 'of',\n",
       " 'two',\n",
       " 'mutated',\n",
       " 'FAM',\n",
       " '',\n",
       " 'A',\n",
       " 'genes',\n",
       " 'found',\n",
       " 'in',\n",
       " 'STAR',\n",
       " 'syndrome',\n",
       " 'patients',\n",
       " '',\n",
       " '',\n",
       " '',\n",
       " '',\n",
       " '',\n",
       " '',\n",
       " 'None',\n",
       " 'of',\n",
       " 'these',\n",
       " 'shorter',\n",
       " 'isoforms',\n",
       " 'produced',\n",
       " 'interaction',\n",
       " 'phenotypes',\n",
       " '',\n",
       " 'Fig',\n",
       " '',\n",
       " '',\n",
       " '',\n",
       " 'A',\n",
       " 'and',\n",
       " 'C',\n",
       " 'and',\n",
       " 'Fig',\n",
       " '',\n",
       " 'S',\n",
       " 'A',\n",
       " '',\n",
       " 'Fig',\n",
       " '',\n",
       " '',\n",
       " '',\n",
       " 'In',\n",
       " 'a',\n",
       " 'new',\n",
       " 'window',\n",
       " 'Download',\n",
       " 'PPTFig',\n",
       " ...]"
      ]
     },
     "execution_count": 4,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "sentences[0]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 27,
   "metadata": {},
   "outputs": [],
   "source": [
    "import numpy as np\n",
    "i = 0\n",
    "for e in sentences :    \n",
    "    if type(e) == float :\n",
    "        print(e)\n",
    "        sentences.remove(e)\n",
    "        "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 28,
   "metadata": {},
   "outputs": [],
   "source": [
    "model = gensim.models.Word2Vec(sentences, size=100, window=5, min_count=5, workers=4)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 31,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "[('weak', 0.5693380832672119),\n",
       " ('slight', 0.566807210445404),\n",
       " ('significant', 0.5563120245933533),\n",
       " ('modest', 0.5223612785339355),\n",
       " ('good', 0.5219054222106934),\n",
       " ('substantial', 0.5095189213752747),\n",
       " ('weaker', 0.4907917082309723),\n",
       " ('considerable', 0.49012285470962524),\n",
       " ('remarkable', 0.4860406219959259),\n",
       " ('stronger', 0.468654066324234)]"
      ]
     },
     "execution_count": 31,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "model.wv.most_similar(positive=['woman', 'strong'], negative=['man'])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 32,
   "metadata": {},
   "outputs": [],
   "source": [
    "word_vectors = model.wv\n",
    "del model"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "print (word_vectors.most_similar('strong'))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "from sklearn.manifold import TSNE\n",
    "\n",
    "X = word_vectors[word_vectors.vocab]\n",
    "tsne = TSNE(n_components=2)\n",
    "X_tsne = tsne.fit_transform(X)\n",
    "plt.scatter(X_tsne[:, 0], X_tsne[:, 1])\n",
    "plt.show()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 40,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{'Cyclin': <gensim.models.keyedvectors.Vocab at 0x2cd53e0f748>,\n",
       " 'dependent': <gensim.models.keyedvectors.Vocab at 0x2cd53e0f8d0>,\n",
       " 'kinases': <gensim.models.keyedvectors.Vocab at 0x2cd53e0f4a8>,\n",
       " '': <gensim.models.keyedvectors.Vocab at 0x2cd53e0f400>,\n",
       " 'CDKs': <gensim.models.keyedvectors.Vocab at 0x2cd53e0f518>,\n",
       " 'regulate': <gensim.models.keyedvectors.Vocab at 0x2cd53e0f3c8>,\n",
       " 'a': <gensim.models.keyedvectors.Vocab at 0x2cd53e0f4e0>,\n",
       " 'variety': <gensim.models.keyedvectors.Vocab at 0x2cd53e0f0f0>,\n",
       " 'of': <gensim.models.keyedvectors.Vocab at 0x2cd53e0f0b8>,\n",
       " 'fundamental': <gensim.models.keyedvectors.Vocab at 0x2cd53e0f198>,\n",
       " 'cellular': <gensim.models.keyedvectors.Vocab at 0x2cd53e0f128>,\n",
       " 'processes': <gensim.models.keyedvectors.Vocab at 0x2cd53e0f160>,\n",
       " 'CDK': <gensim.models.keyedvectors.Vocab at 0x2cd53e0f1d0>,\n",
       " 'stands': <gensim.models.keyedvectors.Vocab at 0x2cd53e0f208>,\n",
       " 'out': <gensim.models.keyedvectors.Vocab at 0x2cd53e0f278>,\n",
       " 'as': <gensim.models.keyedvectors.Vocab at 0x2cd53e0f240>,\n",
       " 'one': <gensim.models.keyedvectors.Vocab at 0x2cd53e0f2b0>,\n",
       " 'the': <gensim.models.keyedvectors.Vocab at 0x2cd53e0f2e8>,\n",
       " 'last': <gensim.models.keyedvectors.Vocab at 0x2cd53e0f390>,\n",
       " 'orphan': <gensim.models.keyedvectors.Vocab at 0x2cd53e0f320>,\n",
       " 'for': <gensim.models.keyedvectors.Vocab at 0x2cd53e0f358>,\n",
       " 'which': <gensim.models.keyedvectors.Vocab at 0x2cd53e0fda0>,\n",
       " 'no': <gensim.models.keyedvectors.Vocab at 0x2cd53e0fef0>,\n",
       " 'activating': <gensim.models.keyedvectors.Vocab at 0x2cd53e0feb8>,\n",
       " 'cyclin': <gensim.models.keyedvectors.Vocab at 0x2cd53e0ff28>,\n",
       " 'has': <gensim.models.keyedvectors.Vocab at 0x2cd53e0ff60>,\n",
       " 'been': <gensim.models.keyedvectors.Vocab at 0x2cd53e0ff98>,\n",
       " 'identified': <gensim.models.keyedvectors.Vocab at 0x2cd53e0ffd0>,\n",
       " 'and': <gensim.models.keyedvectors.Vocab at 0x2cd53e4d048>,\n",
       " 'kinase': <gensim.models.keyedvectors.Vocab at 0x2cd53e4d080>,\n",
       " 'activity': <gensim.models.keyedvectors.Vocab at 0x2cd53e4d0b8>,\n",
       " 'revealed': <gensim.models.keyedvectors.Vocab at 0x2cd53e4d0f0>,\n",
       " 'Previous': <gensim.models.keyedvectors.Vocab at 0x2cd53e4d128>,\n",
       " 'work': <gensim.models.keyedvectors.Vocab at 0x2cd53e4d160>,\n",
       " 'shown': <gensim.models.keyedvectors.Vocab at 0x2cd53e4d198>,\n",
       " 'that': <gensim.models.keyedvectors.Vocab at 0x2cd53e4d1d0>,\n",
       " 'silencing': <gensim.models.keyedvectors.Vocab at 0x2cd53e4d208>,\n",
       " 'increases': <gensim.models.keyedvectors.Vocab at 0x2cd53e4d240>,\n",
       " 'ETS': <gensim.models.keyedvectors.Vocab at 0x2cd53e4d278>,\n",
       " 'v': <gensim.models.keyedvectors.Vocab at 0x2cd53e4d2b0>,\n",
       " 'ets': <gensim.models.keyedvectors.Vocab at 0x2cd53e4d2e8>,\n",
       " 'erythroblastosis': <gensim.models.keyedvectors.Vocab at 0x2cd53e4d320>,\n",
       " 'virus': <gensim.models.keyedvectors.Vocab at 0x2cd53e4d358>,\n",
       " 'E': <gensim.models.keyedvectors.Vocab at 0x2cd53e4d390>,\n",
       " 'oncogene': <gensim.models.keyedvectors.Vocab at 0x2cd53e4d3c8>,\n",
       " 'homolog': <gensim.models.keyedvectors.Vocab at 0x2cd53e4d400>,\n",
       " 'driven': <gensim.models.keyedvectors.Vocab at 0x2cd53e4d438>,\n",
       " 'activation': <gensim.models.keyedvectors.Vocab at 0x2cd53e4d470>,\n",
       " 'MAPK': <gensim.models.keyedvectors.Vocab at 0x2cd53e4d4a8>,\n",
       " 'pathway': <gensim.models.keyedvectors.Vocab at 0x2cd53e4d4e0>,\n",
       " 'confers': <gensim.models.keyedvectors.Vocab at 0x2cd53e4d518>,\n",
       " 'tamoxifen': <gensim.models.keyedvectors.Vocab at 0x2cd53e4d550>,\n",
       " 'resistance': <gensim.models.keyedvectors.Vocab at 0x2cd53e4d588>,\n",
       " 'to': <gensim.models.keyedvectors.Vocab at 0x2cd53e4d5c0>,\n",
       " 'breast': <gensim.models.keyedvectors.Vocab at 0x2cd53e4d5f8>,\n",
       " 'cancer': <gensim.models.keyedvectors.Vocab at 0x2cd53e4d630>,\n",
       " 'cells': <gensim.models.keyedvectors.Vocab at 0x2cd53e4d668>,\n",
       " 'The': <gensim.models.keyedvectors.Vocab at 0x2cd53e4d6a0>,\n",
       " 'precise': <gensim.models.keyedvectors.Vocab at 0x2cd53e4d6d8>,\n",
       " 'mechanisms': <gensim.models.keyedvectors.Vocab at 0x2cd53e4d710>,\n",
       " 'by': <gensim.models.keyedvectors.Vocab at 0x2cd53e4d748>,\n",
       " 'modulates': <gensim.models.keyedvectors.Vocab at 0x2cd53e4d780>,\n",
       " 'more': <gensim.models.keyedvectors.Vocab at 0x2cd53e4d7b8>,\n",
       " 'generally': <gensim.models.keyedvectors.Vocab at 0x2cd53e4d7f0>,\n",
       " 'functions': <gensim.models.keyedvectors.Vocab at 0x2cd53e4d828>,\n",
       " 'remain': <gensim.models.keyedvectors.Vocab at 0x2cd53e4d860>,\n",
       " 'elusive': <gensim.models.keyedvectors.Vocab at 0x2cd53e4d898>,\n",
       " 'Here': <gensim.models.keyedvectors.Vocab at 0x2cd53e4d8d0>,\n",
       " 'we': <gensim.models.keyedvectors.Vocab at 0x2cd53e4d908>,\n",
       " 'demonstrate': <gensim.models.keyedvectors.Vocab at 0x2cd53e4d940>,\n",
       " 'is': <gensim.models.keyedvectors.Vocab at 0x2cd53e4d978>,\n",
       " 'identifying': <gensim.models.keyedvectors.Vocab at 0x2cd53e4d9b0>,\n",
       " 'M': <gensim.models.keyedvectors.Vocab at 0x2cd53e4d9e8>,\n",
       " 'an': <gensim.models.keyedvectors.Vocab at 0x2cd53e4da20>,\n",
       " 'product': <gensim.models.keyedvectors.Vocab at 0x2cd53e4da58>,\n",
       " 'FAM': <gensim.models.keyedvectors.Vocab at 0x2cd53e4da90>,\n",
       " 'A': <gensim.models.keyedvectors.Vocab at 0x2cd53e4dac8>,\n",
       " 'whose': <gensim.models.keyedvectors.Vocab at 0x2cd53e4db00>,\n",
       " 'mutations': <gensim.models.keyedvectors.Vocab at 0x2cd53e4db38>,\n",
       " 'cause': <gensim.models.keyedvectors.Vocab at 0x2cd53e4db70>,\n",
       " 'STAR': <gensim.models.keyedvectors.Vocab at 0x2cd53e4dba8>,\n",
       " 'syndrome': <gensim.models.keyedvectors.Vocab at 0x2cd53e4dbe0>,\n",
       " 'human': <gensim.models.keyedvectors.Vocab at 0x2cd53e4dc18>,\n",
       " 'developmental': <gensim.models.keyedvectors.Vocab at 0x2cd53e4dc50>,\n",
       " 'anomaly': <gensim.models.keyedvectors.Vocab at 0x2cd53e4dc88>,\n",
       " 'features': <gensim.models.keyedvectors.Vocab at 0x2cd53e4dcc0>,\n",
       " 'include': <gensim.models.keyedvectors.Vocab at 0x2cd53e4dcf8>,\n",
       " 'toe': <gensim.models.keyedvectors.Vocab at 0x2cd53e4dd30>,\n",
       " 'syndactyly': <gensim.models.keyedvectors.Vocab at 0x2cd53e4dd68>,\n",
       " 'telecanthus': <gensim.models.keyedvectors.Vocab at 0x2cd53e4dda0>,\n",
       " 'renal': <gensim.models.keyedvectors.Vocab at 0x2cd53e4ddd8>,\n",
       " 'malformations': <gensim.models.keyedvectors.Vocab at 0x2cd53e4de10>,\n",
       " 'We': <gensim.models.keyedvectors.Vocab at 0x2cd53e4de48>,\n",
       " 'show': <gensim.models.keyedvectors.Vocab at 0x2cd53e4de80>,\n",
       " 'associated': <gensim.models.keyedvectors.Vocab at 0x2cd53e4deb8>,\n",
       " 'mutants': <gensim.models.keyedvectors.Vocab at 0x2cd53e4def0>,\n",
       " 'are': <gensim.models.keyedvectors.Vocab at 0x2cd53e4df28>,\n",
       " 'unable': <gensim.models.keyedvectors.Vocab at 0x2cd53e4df60>,\n",
       " 'interact': <gensim.models.keyedvectors.Vocab at 0x2cd53e4df98>,\n",
       " 'with': <gensim.models.keyedvectors.Vocab at 0x2cd53e4dfd0>,\n",
       " 'phenocopies': <gensim.models.keyedvectors.Vocab at 0x2cd53e50048>,\n",
       " 'in': <gensim.models.keyedvectors.Vocab at 0x2cd53e50080>,\n",
       " 'increasing': <gensim.models.keyedvectors.Vocab at 0x2cd53e500b8>,\n",
       " 'c': <gensim.models.keyedvectors.Vocab at 0x2cd53e500f0>,\n",
       " 'Raf': <gensim.models.keyedvectors.Vocab at 0x2cd53e50128>,\n",
       " 'conferring': <gensim.models.keyedvectors.Vocab at 0x2cd53e50160>,\n",
       " 'phosphorylates': <gensim.models.keyedvectors.Vocab at 0x2cd53e50198>,\n",
       " 'vitro': <gensim.models.keyedvectors.Vocab at 0x2cd53e501d0>,\n",
       " 'it': <gensim.models.keyedvectors.Vocab at 0x2cd53e50208>,\n",
       " 'positively': <gensim.models.keyedvectors.Vocab at 0x2cd53e50240>,\n",
       " 'controls': <gensim.models.keyedvectors.Vocab at 0x2cd53e50278>,\n",
       " 'degradation': <gensim.models.keyedvectors.Vocab at 0x2cd53e502b0>,\n",
       " 'proteasome': <gensim.models.keyedvectors.Vocab at 0x2cd53e502e8>,\n",
       " 'protein': <gensim.models.keyedvectors.Vocab at 0x2cd53e50320>,\n",
       " 'levels': <gensim.models.keyedvectors.Vocab at 0x2cd53e50358>,\n",
       " 'increased': <gensim.models.keyedvectors.Vocab at 0x2cd53e50390>,\n",
       " 'derived': <gensim.models.keyedvectors.Vocab at 0x2cd53e503c8>,\n",
       " 'from': <gensim.models.keyedvectors.Vocab at 0x2cd53e50400>,\n",
       " 'patient': <gensim.models.keyedvectors.Vocab at 0x2cd53e50438>,\n",
       " 'this': <gensim.models.keyedvectors.Vocab at 0x2cd53e50470>,\n",
       " 'increase': <gensim.models.keyedvectors.Vocab at 0x2cd53e504a8>,\n",
       " 'attributable': <gensim.models.keyedvectors.Vocab at 0x2cd53e504e0>,\n",
       " 'decreased': <gensim.models.keyedvectors.Vocab at 0x2cd53e50518>,\n",
       " 'Altogether': <gensim.models.keyedvectors.Vocab at 0x2cd53e50550>,\n",
       " 'our': <gensim.models.keyedvectors.Vocab at 0x2cd53e50588>,\n",
       " 'results': <gensim.models.keyedvectors.Vocab at 0x2cd53e505c0>,\n",
       " 'reveal': <gensim.models.keyedvectors.Vocab at 0x2cd53e505f8>,\n",
       " 'additional': <gensim.models.keyedvectors.Vocab at 0x2cd53e50630>,\n",
       " 'regulatory': <gensim.models.keyedvectors.Vocab at 0x2cd53e50668>,\n",
       " 'mechanism': <gensim.models.keyedvectors.Vocab at 0x2cd53e506a0>,\n",
       " 'plays': <gensim.models.keyedvectors.Vocab at 0x2cd53e506d8>,\n",
       " 'key': <gensim.models.keyedvectors.Vocab at 0x2cd53e50710>,\n",
       " 'roles': <gensim.models.keyedvectors.Vocab at 0x2cd53e50748>,\n",
       " 'development': <gensim.models.keyedvectors.Vocab at 0x2cd53e50780>,\n",
       " 'They': <gensim.models.keyedvectors.Vocab at 0x2cd53e507b8>,\n",
       " 'also': <gensim.models.keyedvectors.Vocab at 0x2cd53e507f0>,\n",
       " 'shed': <gensim.models.keyedvectors.Vocab at 0x2cd53e50828>,\n",
       " 'light': <gensim.models.keyedvectors.Vocab at 0x2cd53e50860>,\n",
       " 'on': <gensim.models.keyedvectors.Vocab at 0x2cd53e50898>,\n",
       " 'molecular': <gensim.models.keyedvectors.Vocab at 0x2cd53e508d0>,\n",
       " 'underlying': <gensim.models.keyedvectors.Vocab at 0x2cd53e50908>,\n",
       " 'play': <gensim.models.keyedvectors.Vocab at 0x2cd53e50940>,\n",
       " 'pivotal': <gensim.models.keyedvectors.Vocab at 0x2cd53e50978>,\n",
       " 'role': <gensim.models.keyedvectors.Vocab at 0x2cd53e509b0>,\n",
       " 'control': <gensim.models.keyedvectors.Vocab at 0x2cd53e509e8>,\n",
       " 'number': <gensim.models.keyedvectors.Vocab at 0x2cd53e50a20>,\n",
       " 'genome': <gensim.models.keyedvectors.Vocab at 0x2cd53e50a58>,\n",
       " 'contains': <gensim.models.keyedvectors.Vocab at 0x2cd53e50a90>,\n",
       " 'genes': <gensim.models.keyedvectors.Vocab at 0x2cd53e50ac8>,\n",
       " 'encoding': <gensim.models.keyedvectors.Vocab at 0x2cd53e50b00>,\n",
       " 'proteins': <gensim.models.keyedvectors.Vocab at 0x2cd53e50b38>,\n",
       " 'can': <gensim.models.keyedvectors.Vocab at 0x2cd53e50b70>,\n",
       " 'be': <gensim.models.keyedvectors.Vocab at 0x2cd53e50ba8>,\n",
       " 'considered': <gensim.models.keyedvectors.Vocab at 0x2cd53e50be0>,\n",
       " 'members': <gensim.models.keyedvectors.Vocab at 0x2cd53e50c18>,\n",
       " 'family': <gensim.models.keyedvectors.Vocab at 0x2cd53e50c50>,\n",
       " 'owing': <gensim.models.keyedvectors.Vocab at 0x2cd53e50c88>,\n",
       " 'their': <gensim.models.keyedvectors.Vocab at 0x2cd53e50cc0>,\n",
       " 'sequence': <gensim.models.keyedvectors.Vocab at 0x2cd53e50cf8>,\n",
       " 'similarity': <gensim.models.keyedvectors.Vocab at 0x2cd53e50d30>,\n",
       " 'bona': <gensim.models.keyedvectors.Vocab at 0x2cd53e50d68>,\n",
       " 'fide': <gensim.models.keyedvectors.Vocab at 0x2cd53e50da0>,\n",
       " 'those': <gensim.models.keyedvectors.Vocab at 0x2cd53e50dd8>,\n",
       " 'known': <gensim.models.keyedvectors.Vocab at 0x2cd53e50e10>,\n",
       " 'activated': <gensim.models.keyedvectors.Vocab at 0x2cd53e50e48>,\n",
       " 'cyclins': <gensim.models.keyedvectors.Vocab at 0x2cd53e50e80>,\n",
       " 'Although': <gensim.models.keyedvectors.Vocab at 0x2cd53e50eb8>,\n",
       " 'discovered': <gensim.models.keyedvectors.Vocab at 0x2cd53e50ef0>,\n",
       " 'almost': <gensim.models.keyedvectors.Vocab at 0x2cd53e50f28>,\n",
       " 'y': <gensim.models.keyedvectors.Vocab at 0x2cd53e50f60>,\n",
       " 'ago': <gensim.models.keyedvectors.Vocab at 0x2cd53e50f98>,\n",
       " 'remains': <gensim.models.keyedvectors.Vocab at 0x2cd53e50fd0>,\n",
       " 'two': <gensim.models.keyedvectors.Vocab at 0x2cd53e53048>,\n",
       " 'without': <gensim.models.keyedvectors.Vocab at 0x2cd53e53080>,\n",
       " 'partner': <gensim.models.keyedvectors.Vocab at 0x2cd53e530b8>,\n",
       " 'This': <gensim.models.keyedvectors.Vocab at 0x2cd53e530f0>,\n",
       " 'knowledge': <gensim.models.keyedvectors.Vocab at 0x2cd53e53128>,\n",
       " 'gap': <gensim.models.keyedvectors.Vocab at 0x2cd53e53160>,\n",
       " 'largely': <gensim.models.keyedvectors.Vocab at 0x2cd53e53198>,\n",
       " 'impeded': <gensim.models.keyedvectors.Vocab at 0x2cd53e531d0>,\n",
       " 'exploration': <gensim.models.keyedvectors.Vocab at 0x2cd53e53208>,\n",
       " 'its': <gensim.models.keyedvectors.Vocab at 0x2cd53e53240>,\n",
       " 'biological': <gensim.models.keyedvectors.Vocab at 0x2cd53e53278>,\n",
       " 'act': <gensim.models.keyedvectors.Vocab at 0x2cd53e532b0>,\n",
       " 'positive': <gensim.models.keyedvectors.Vocab at 0x2cd53e532e8>,\n",
       " 'cell': <gensim.models.keyedvectors.Vocab at 0x2cd53e53320>,\n",
       " 'cycle': <gensim.models.keyedvectors.Vocab at 0x2cd53e53358>,\n",
       " 'regulator': <gensim.models.keyedvectors.Vocab at 0x2cd53e53390>,\n",
       " 'some': <gensim.models.keyedvectors.Vocab at 0x2cd53e533c8>,\n",
       " 'or': <gensim.models.keyedvectors.Vocab at 0x2cd53e53400>,\n",
       " 'tumor': <gensim.models.keyedvectors.Vocab at 0x2cd53e53438>,\n",
       " 'suppressor': <gensim.models.keyedvectors.Vocab at 0x2cd53e53470>,\n",
       " 'others': <gensim.models.keyedvectors.Vocab at 0x2cd53e534a8>,\n",
       " 'interacts': <gensim.models.keyedvectors.Vocab at 0x2cd53e534e0>,\n",
       " 'transcription': <gensim.models.keyedvectors.Vocab at 0x2cd53e53518>,\n",
       " 'factor': <gensim.models.keyedvectors.Vocab at 0x2cd53e53550>,\n",
       " 'inhibits': <gensim.models.keyedvectors.Vocab at 0x2cd53e53588>,\n",
       " 'transcriptional': <gensim.models.keyedvectors.Vocab at 0x2cd53e535c0>,\n",
       " 'through': <gensim.models.keyedvectors.Vocab at 0x2cd53e535f8>,\n",
       " 'unknown': <gensim.models.keyedvectors.Vocab at 0x2cd53e53630>,\n",
       " 'knockdown': <gensim.models.keyedvectors.Vocab at 0x2cd53e53668>,\n",
       " 'derepresses': <gensim.models.keyedvectors.Vocab at 0x2cd53e536a0>,\n",
       " 'expression': <gensim.models.keyedvectors.Vocab at 0x2cd53e536d8>,\n",
       " 'activates': <gensim.models.keyedvectors.Vocab at 0x2cd53e53710>,\n",
       " 'induces': <gensim.models.keyedvectors.Vocab at 0x2cd53e53748>,\n",
       " 'MCF': <gensim.models.keyedvectors.Vocab at 0x2cd53e53780>,\n",
       " 'binding': <gensim.models.keyedvectors.Vocab at 0x2cd53e537b8>,\n",
       " 'Mutations': <gensim.models.keyedvectors.Vocab at 0x2cd53e537f0>,\n",
       " 'gene': <gensim.models.keyedvectors.Vocab at 0x2cd53e53828>,\n",
       " 'predict': <gensim.models.keyedvectors.Vocab at 0x2cd53e53860>,\n",
       " 'absence': <gensim.models.keyedvectors.Vocab at 0x2cd53e53898>,\n",
       " 'truncation': <gensim.models.keyedvectors.Vocab at 0x2cd53e538d0>,\n",
       " 'heterozygous': <gensim.models.keyedvectors.Vocab at 0x2cd53e53908>,\n",
       " 'females': <gensim.models.keyedvectors.Vocab at 0x2cd53e53940>,\n",
       " 'However': <gensim.models.keyedvectors.Vocab at 0x2cd53e53978>,\n",
       " 'both': <gensim.models.keyedvectors.Vocab at 0x2cd53e539b0>,\n",
       " 'pathogenesis': <gensim.models.keyedvectors.Vocab at 0x2cd53e539e8>,\n",
       " 'recombinant': <gensim.models.keyedvectors.Vocab at 0x2cd53e53a20>,\n",
       " 'heterodimer': <gensim.models.keyedvectors.Vocab at 0x2cd53e53a58>,\n",
       " 'active': <gensim.models.keyedvectors.Vocab at 0x2cd53e53a90>,\n",
       " 'phospho': <gensim.models.keyedvectors.Vocab at 0x2cd53e53ac8>,\n",
       " 'ERK': <gensim.models.keyedvectors.Vocab at 0x2cd53e53b00>,\n",
       " 'inducing': <gensim.models.keyedvectors.Vocab at 0x2cd53e53b38>,\n",
       " 'estrogen': <gensim.models.keyedvectors.Vocab at 0x2cd53e53b70>,\n",
       " 'receptor': <gensim.models.keyedvectors.Vocab at 0x2cd53e53ba8>,\n",
       " 'ER': <gensim.models.keyedvectors.Vocab at 0x2cd53e53be0>,\n",
       " 'phosphorylation': <gensim.models.keyedvectors.Vocab at 0x2cd53e53c18>,\n",
       " 'neighboring': <gensim.models.keyedvectors.Vocab at 0x2cd53e53c50>,\n",
       " 'serines': <gensim.models.keyedvectors.Vocab at 0x2cd53e53c88>,\n",
       " 'Finally': <gensim.models.keyedvectors.Vocab at 0x2cd53e53cc0>,\n",
       " 'detect': <gensim.models.keyedvectors.Vocab at 0x2cd53e53cf8>,\n",
       " 'level': <gensim.models.keyedvectors.Vocab at 0x2cd53e53d30>,\n",
       " 'observed': <gensim.models.keyedvectors.Vocab at 0x2cd53e53d68>,\n",
       " 'these': <gensim.models.keyedvectors.Vocab at 0x2cd53e53da0>,\n",
       " 'SectionNext': <gensim.models.keyedvectors.Vocab at 0x2cd53e53dd8>,\n",
       " 'yeast': <gensim.models.keyedvectors.Vocab at 0x2cd53e53e10>,\n",
       " 'hybrid': <gensim.models.keyedvectors.Vocab at 0x2cd53e53e48>,\n",
       " 'Y': <gensim.models.keyedvectors.Vocab at 0x2cd53e53e80>,\n",
       " 'H': <gensim.models.keyedvectors.Vocab at 0x2cd53e53eb8>,\n",
       " 'screen': <gensim.models.keyedvectors.Vocab at 0x2cd53e53ef0>,\n",
       " 'unveiled': <gensim.models.keyedvectors.Vocab at 0x2cd53e53f28>,\n",
       " 'interaction': <gensim.models.keyedvectors.Vocab at 0x2cd53e53f60>,\n",
       " 'signal': <gensim.models.keyedvectors.Vocab at 0x2cd53e53f98>,\n",
       " 'between': <gensim.models.keyedvectors.Vocab at 0x2cd53e53fd0>,\n",
       " 'mouse': <gensim.models.keyedvectors.Vocab at 0x2cd53e56048>,\n",
       " 'C': <gensim.models.keyedvectors.Vocab at 0x2cd53e56080>,\n",
       " 'terminal': <gensim.models.keyedvectors.Vocab at 0x2cd53e560b8>,\n",
       " 'half': <gensim.models.keyedvectors.Vocab at 0x2cd53e560f0>,\n",
       " 'presents': <gensim.models.keyedvectors.Vocab at 0x2cd53e56128>,\n",
       " 'strong': <gensim.models.keyedvectors.Vocab at 0x2cd53e56160>,\n",
       " 'homology': <gensim.models.keyedvectors.Vocab at 0x2cd53e56198>,\n",
       " 'proposed': <gensim.models.keyedvectors.Vocab at 0x2cd53e561d0>,\n",
       " 'name': <gensim.models.keyedvectors.Vocab at 0x2cd53e56208>,\n",
       " 'thus': <gensim.models.keyedvectors.Vocab at 0x2cd53e56240>,\n",
       " 'performed': <gensim.models.keyedvectors.Vocab at 0x2cd53e56278>,\n",
       " 'mating': <gensim.models.keyedvectors.Vocab at 0x2cd53e562b0>,\n",
       " 'assays': <gensim.models.keyedvectors.Vocab at 0x2cd53e562e8>,\n",
       " 'determine': <gensim.models.keyedvectors.Vocab at 0x2cd53e56320>,\n",
       " 'whether': <gensim.models.keyedvectors.Vocab at 0x2cd53e56358>,\n",
       " 'Fig': <gensim.models.keyedvectors.Vocab at 0x2cd53e56390>,\n",
       " 'Aâ€': <gensim.models.keyedvectors.Vocab at 0x2cd53e563c8>,\n",
       " 'longest': <gensim.models.keyedvectors.Vocab at 0x2cd53e56400>,\n",
       " 'isoform': <gensim.models.keyedvectors.Vocab at 0x2cd53e56438>,\n",
       " 'P': <gensim.models.keyedvectors.Vocab at 0x2cd53e56470>,\n",
       " 'expressed': <gensim.models.keyedvectors.Vocab at 0x2cd53e564a8>,\n",
       " 'bait': <gensim.models.keyedvectors.Vocab at 0x2cd53e564e0>,\n",
       " 'produced': <gensim.models.keyedvectors.Vocab at 0x2cd53e56518>,\n",
       " 'phenotype': <gensim.models.keyedvectors.Vocab at 0x2cd53e56550>,\n",
       " 'full': <gensim.models.keyedvectors.Vocab at 0x2cd53e56588>,\n",
       " 'length': <gensim.models.keyedvectors.Vocab at 0x2cd53e565c0>,\n",
       " 'prey': <gensim.models.keyedvectors.Vocab at 0x2cd53e565f8>,\n",
       " 'but': <gensim.models.keyedvectors.Vocab at 0x2cd53e56630>,\n",
       " 'detectable': <gensim.models.keyedvectors.Vocab at 0x2cd53e56668>,\n",
       " 'D': <gensim.models.keyedvectors.Vocab at 0x2cd53e566a0>,\n",
       " 'p': <gensim.models.keyedvectors.Vocab at 0x2cd53e566d8>,\n",
       " 'CIP': <gensim.models.keyedvectors.Vocab at 0x2cd53e56710>,\n",
       " 'KAP': <gensim.models.keyedvectors.Vocab at 0x2cd53e56748>,\n",
       " 'partners': <gensim.models.keyedvectors.Vocab at 0x2cd53e56780>,\n",
       " 'other': <gensim.models.keyedvectors.Vocab at 0x2cd53e567b8>,\n",
       " 'B': <gensim.models.keyedvectors.Vocab at 0x2cd53e567f0>,\n",
       " 'signals': <gensim.models.keyedvectors.Vocab at 0x2cd53e56828>,\n",
       " 'albeit': <gensim.models.keyedvectors.Vocab at 0x2cd53e56860>,\n",
       " 'notably': <gensim.models.keyedvectors.Vocab at 0x2cd53e56898>,\n",
       " 'weaker': <gensim.models.keyedvectors.Vocab at 0x2cd53e568d0>,\n",
       " 'than': <gensim.models.keyedvectors.Vocab at 0x2cd53e56908>,\n",
       " 'An': <gensim.models.keyedvectors.Vocab at 0x2cd53e56940>,\n",
       " 'was': <gensim.models.keyedvectors.Vocab at 0x2cd53e56978>,\n",
       " 'respectively': <gensim.models.keyedvectors.Vocab at 0x2cd53e569b0>,\n",
       " 'S': <gensim.models.keyedvectors.Vocab at 0x2cd53e569e8>,\n",
       " 'then': <gensim.models.keyedvectors.Vocab at 0x2cd53e56a20>,\n",
       " 'tested': <gensim.models.keyedvectors.Vocab at 0x2cd53e56a58>,\n",
       " 'different': <gensim.models.keyedvectors.Vocab at 0x2cd53e56a90>,\n",
       " 'isoforms': <gensim.models.keyedvectors.Vocab at 0x2cd53e56ac8>,\n",
       " 'originating': <gensim.models.keyedvectors.Vocab at 0x2cd53e56b00>,\n",
       " 'alternative': <gensim.models.keyedvectors.Vocab at 0x2cd53e56b38>,\n",
       " 'splicing': <gensim.models.keyedvectors.Vocab at 0x2cd53e56b70>,\n",
       " 'truncated': <gensim.models.keyedvectors.Vocab at 0x2cd53e56ba8>,\n",
       " 'corresponding': <gensim.models.keyedvectors.Vocab at 0x2cd53e56be0>,\n",
       " 'hypothetical': <gensim.models.keyedvectors.Vocab at 0x2cd53e56c18>,\n",
       " 'products': <gensim.models.keyedvectors.Vocab at 0x2cd53e56c50>,\n",
       " 'mutated': <gensim.models.keyedvectors.Vocab at 0x2cd53e56c88>,\n",
       " 'found': <gensim.models.keyedvectors.Vocab at 0x2cd53e56cc0>,\n",
       " 'patients': <gensim.models.keyedvectors.Vocab at 0x2cd53e56cf8>,\n",
       " 'None': <gensim.models.keyedvectors.Vocab at 0x2cd53e56d30>,\n",
       " 'shorter': <gensim.models.keyedvectors.Vocab at 0x2cd53e56d68>,\n",
       " 'phenotypes': <gensim.models.keyedvectors.Vocab at 0x2cd53e56da0>,\n",
       " 'In': <gensim.models.keyedvectors.Vocab at 0x2cd53e56dd8>,\n",
       " 'new': <gensim.models.keyedvectors.Vocab at 0x2cd53e56e10>,\n",
       " 'window': <gensim.models.keyedvectors.Vocab at 0x2cd53e56e48>,\n",
       " 'Download': <gensim.models.keyedvectors.Vocab at 0x2cd53e56e80>,\n",
       " 'PPTFig': <gensim.models.keyedvectors.Vocab at 0x2cd53e56eb8>,\n",
       " 'form': <gensim.models.keyedvectors.Vocab at 0x2cc33c90278>,\n",
       " 'complex': <gensim.models.keyedvectors.Vocab at 0x2ccc0d62cc0>,\n",
       " 'Schematic': <gensim.models.keyedvectors.Vocab at 0x2ccc0d62c50>,\n",
       " 'representation': <gensim.models.keyedvectors.Vocab at 0x2cd53e0fe48>,\n",
       " 'analyzed': <gensim.models.keyedvectors.Vocab at 0x2cd53e0fe10>,\n",
       " 'Amino': <gensim.models.keyedvectors.Vocab at 0x2cd53e0fdd8>,\n",
       " 'acid': <gensim.models.keyedvectors.Vocab at 0x2cd53e0fcc0>,\n",
       " 'numbers': <gensim.models.keyedvectors.Vocab at 0x2cd53e0fcf8>,\n",
       " 'indicated': <gensim.models.keyedvectors.Vocab at 0x2cd53e0fd68>,\n",
       " 'Black': <gensim.models.keyedvectors.Vocab at 0x2cd53e0fd30>,\n",
       " 'boxes': <gensim.models.keyedvectors.Vocab at 0x2cd53e0fc88>,\n",
       " 'indicate': <gensim.models.keyedvectors.Vocab at 0x2cd53e0f7f0>,\n",
       " 'internal': <gensim.models.keyedvectors.Vocab at 0x2cd4217c048>,\n",
       " 'deletions': <gensim.models.keyedvectors.Vocab at 0x2cd4217cfd0>,\n",
       " 'red': <gensim.models.keyedvectors.Vocab at 0x2cd4217cdd8>,\n",
       " 'box': <gensim.models.keyedvectors.Vocab at 0x2cd4217c438>,\n",
       " 'indicates': <gensim.models.keyedvectors.Vocab at 0x2cd4217ce10>,\n",
       " 'differing': <gensim.models.keyedvectors.Vocab at 0x2cd4217cc88>,\n",
       " 'amino': <gensim.models.keyedvectors.Vocab at 0x2cd4217c080>,\n",
       " 'compared': <gensim.models.keyedvectors.Vocab at 0x2cd4217cf98>,\n",
       " 'assay': <gensim.models.keyedvectors.Vocab at 0x2cd4217c1d0>,\n",
       " 'set': <gensim.models.keyedvectors.Vocab at 0x2cd4217c828>,\n",
       " 'baits': <gensim.models.keyedvectors.Vocab at 0x2cd4217cef0>,\n",
       " 'fusion': <gensim.models.keyedvectors.Vocab at 0x2cd4217cf60>,\n",
       " 'LexA': <gensim.models.keyedvectors.Vocab at 0x2cd4217ccc0>,\n",
       " 'DNA': <gensim.models.keyedvectors.Vocab at 0x2cd4217ce48>,\n",
       " 'domain': <gensim.models.keyedvectors.Vocab at 0x2cd4217ceb8>,\n",
       " 'interacting': <gensim.models.keyedvectors.Vocab at 0x2cd4217ccf8>,\n",
       " 'activator': <gensim.models.keyedvectors.Vocab at 0x2cd4217ce80>,\n",
       " 'pEG': <gensim.models.keyedvectors.Vocab at 0x2cd4217c550>,\n",
       " 'empty': <gensim.models.keyedvectors.Vocab at 0x2cd4217cd68>,\n",
       " 'plasmids': <gensim.models.keyedvectors.Vocab at 0x2cd4217cda0>,\n",
       " 'expressing': <gensim.models.keyedvectors.Vocab at 0x2cd4217cd30>,\n",
       " 'lacZ': <gensim.models.keyedvectors.Vocab at 0x2cd53e56ef0>,\n",
       " 'used': <gensim.models.keyedvectors.Vocab at 0x2cd53e56f28>,\n",
       " 'reporter': <gensim.models.keyedvectors.Vocab at 0x2cd53e56f60>,\n",
       " 'blue': <gensim.models.keyedvectors.Vocab at 0x2cd53e56f98>,\n",
       " 'indicative': <gensim.models.keyedvectors.Vocab at 0x2cd53e56fd0>,\n",
       " 'region': <gensim.models.keyedvectors.Vocab at 0x2ccc12e8048>,\n",
       " 'assayed': <gensim.models.keyedvectors.Vocab at 0x2ccc12e8080>,\n",
       " 'Western': <gensim.models.keyedvectors.Vocab at 0x2ccc12e80b8>,\n",
       " 'blot': <gensim.models.keyedvectors.Vocab at 0x2ccc12e80f0>,\n",
       " 'analysis': <gensim.models.keyedvectors.Vocab at 0x2ccc12e8128>,\n",
       " 'Myc': <gensim.models.keyedvectors.Vocab at 0x2ccc12e8160>,\n",
       " 'wt': <gensim.models.keyedvectors.Vocab at 0x2ccc12e8198>,\n",
       " 'kd': <gensim.models.keyedvectors.Vocab at 0x2ccc12e81d0>,\n",
       " 'CycM': <gensim.models.keyedvectors.Vocab at 0x2ccc12e8208>,\n",
       " 'V': <gensim.models.keyedvectors.Vocab at 0x2ccc12e8240>,\n",
       " 'His': <gensim.models.keyedvectors.Vocab at 0x2ccc12e8278>,\n",
       " 'transfected': <gensim.models.keyedvectors.Vocab at 0x2ccc12e82b0>,\n",
       " 'HEK': <gensim.models.keyedvectors.Vocab at 0x2ccc12e82e8>,\n",
       " 'immunoprecipitates': <gensim.models.keyedvectors.Vocab at 0x2ccc12e8320>,\n",
       " 'obtained': <gensim.models.keyedvectors.Vocab at 0x2ccc12e8358>,\n",
       " 'using': <gensim.models.keyedvectors.Vocab at 0x2ccc12e8390>,\n",
       " 'anti': <gensim.models.keyedvectors.Vocab at 0x2ccc12e83c8>,\n",
       " 'antibody': <gensim.models.keyedvectors.Vocab at 0x2ccc12e8400>,\n",
       " 'correspond': <gensim.models.keyedvectors.Vocab at 0x2ccc12e8438>,\n",
       " 'Î¼g': <gensim.models.keyedvectors.Vocab at 0x2ccc12e8470>,\n",
       " 'total': <gensim.models.keyedvectors.Vocab at 0x2ccc12e84a8>,\n",
       " 'lysates': <gensim.models.keyedvectors.Vocab at 0x2ccc12e84e0>,\n",
       " 'coexpressing': <gensim.models.keyedvectors.Vocab at 0x2ccc12e8518>,\n",
       " 'F': <gensim.models.keyedvectors.Vocab at 0x2ccc12e8550>,\n",
       " 'goat': <gensim.models.keyedvectors.Vocab at 0x2ccc12e8588>,\n",
       " 'corresponds': <gensim.models.keyedvectors.Vocab at 0x2ccc12e85c0>,\n",
       " 'lower': <gensim.models.keyedvectors.Vocab at 0x2ccc12e85f8>,\n",
       " 'band': <gensim.models.keyedvectors.Vocab at 0x2ccc12e8630>,\n",
       " 'doublet': <gensim.models.keyedvectors.Vocab at 0x2ccc12e8668>,\n",
       " 'upper': <gensim.models.keyedvectors.Vocab at 0x2ccc12e86a0>,\n",
       " 'panel': <gensim.models.keyedvectors.Vocab at 0x2ccc12e86d8>,\n",
       " 'comigrates': <gensim.models.keyedvectors.Vocab at 0x2ccc12e8710>,\n",
       " 'exogenously': <gensim.models.keyedvectors.Vocab at 0x2ccc12e8748>,\n",
       " 'untagged': <gensim.models.keyedvectors.Vocab at 0x2ccc12e8780>,\n",
       " 'endogenous': <gensim.models.keyedvectors.Vocab at 0x2ccc12e87b8>,\n",
       " 'nonspecific': <gensim.models.keyedvectors.Vocab at 0x2ccc12e87f0>,\n",
       " 'demonstrated': <gensim.models.keyedvectors.Vocab at 0x2ccc12e8828>,\n",
       " 'insensitivity': <gensim.models.keyedvectors.Vocab at 0x2ccc12e8860>,\n",
       " 'either': <gensim.models.keyedvectors.Vocab at 0x2ccc12e8898>,\n",
       " 'overexpression': <gensim.models.keyedvectors.Vocab at 0x2ccc12e88d0>,\n",
       " 'seen': <gensim.models.keyedvectors.Vocab at 0x2ccc12e8908>,\n",
       " 'left': <gensim.models.keyedvectors.Vocab at 0x2ccc12e8940>,\n",
       " 'lane': <gensim.models.keyedvectors.Vocab at 0x2ccc12e8978>,\n",
       " 'Another': <gensim.models.keyedvectors.Vocab at 0x2ccc12e89b0>,\n",
       " 'experiment': <gensim.models.keyedvectors.Vocab at 0x2ccc12e89e8>,\n",
       " 'longer': <gensim.models.keyedvectors.Vocab at 0x2ccc12e8a20>,\n",
       " 'gel': <gensim.models.keyedvectors.Vocab at 0x2ccc12e8a58>,\n",
       " 'migration': <gensim.models.keyedvectors.Vocab at 0x2ccc12e8a90>,\n",
       " 'Next': <gensim.models.keyedvectors.Vocab at 0x2ccc12e8ac8>,\n",
       " 'examined': <gensim.models.keyedvectors.Vocab at 0x2ccc12e8b00>,\n",
       " 'ability': <gensim.models.keyedvectors.Vocab at 0x2ccc12e8b38>,\n",
       " 'when': <gensim.models.keyedvectors.Vocab at 0x2ccc12e8b70>,\n",
       " 'wild': <gensim.models.keyedvectors.Vocab at 0x2ccc12e8ba8>,\n",
       " 'type': <gensim.models.keyedvectors.Vocab at 0x2ccc12e8be0>,\n",
       " 'dead': <gensim.models.keyedvectors.Vocab at 0x2ccc12e8c18>,\n",
       " 'mutant': <gensim.models.keyedvectors.Vocab at 0x2ccc12e8c50>,\n",
       " 'bearing': <gensim.models.keyedvectors.Vocab at 0x2ccc12e8c88>,\n",
       " 'substitution': <gensim.models.keyedvectors.Vocab at 0x2ccc12e8cc0>,\n",
       " 'abolishes': <gensim.models.keyedvectors.Vocab at 0x2ccc12e8cf8>,\n",
       " 'ATP': <gensim.models.keyedvectors.Vocab at 0x2ccc12e8d30>,\n",
       " 'embryonic': <gensim.models.keyedvectors.Vocab at 0x2ccc12e8d68>,\n",
       " 'kidney': <gensim.models.keyedvectors.Vocab at 0x2ccc12e8da0>,\n",
       " 'line': <gensim.models.keyedvectors.Vocab at 0x2ccc12e8dd8>,\n",
       " 'significantly': <gensim.models.keyedvectors.Vocab at 0x2ccc12e8e10>,\n",
       " 'upon': <gensim.models.keyedvectors.Vocab at 0x2ccc12e8e48>,\n",
       " 'coexpression': <gensim.models.keyedvectors.Vocab at 0x2ccc12e8e80>,\n",
       " 'lesser': <gensim.models.keyedvectors.Vocab at 0x2ccc12e8eb8>,\n",
       " 'extent': <gensim.models.keyedvectors.Vocab at 0x2ccc12e8ef0>,\n",
       " 'immunoprecipitated': <gensim.models.keyedvectors.Vocab at 0x2ccc12e8f28>,\n",
       " 'detected': <gensim.models.keyedvectors.Vocab at 0x2ccc12e8f60>,\n",
       " 'presence': <gensim.models.keyedvectors.Vocab at 0x2ccc12e8f98>,\n",
       " 'only': <gensim.models.keyedvectors.Vocab at 0x2ccc12e8fd0>,\n",
       " 'were': <gensim.models.keyedvectors.Vocab at 0x2cd53e5a048>,\n",
       " 'coexpressed': <gensim.models.keyedvectors.Vocab at 0x2cd53e5a080>,\n",
       " 'pair': <gensim.models.keyedvectors.Vocab at 0x2cd53e5a0b8>,\n",
       " 'wise': <gensim.models.keyedvectors.Vocab at 0x2cd53e5a0f0>,\n",
       " 'confirmed': <gensim.models.keyedvectors.Vocab at 0x2cd53e5a128>,\n",
       " 'observations': <gensim.models.keyedvectors.Vocab at 0x2cd53e5a160>,\n",
       " 'detecting': <gensim.models.keyedvectors.Vocab at 0x2cd53e5a198>,\n",
       " 'These': <gensim.models.keyedvectors.Vocab at 0x2cd53e5a1d0>,\n",
       " 'experiments': <gensim.models.keyedvectors.Vocab at 0x2cd53e5a208>,\n",
       " 'lack': <gensim.models.keyedvectors.Vocab at 0x2cd53e5a240>,\n",
       " 'robust': <gensim.models.keyedvectors.Vocab at 0x2cd53e5a278>,\n",
       " 'To': <gensim.models.keyedvectors.Vocab at 0x2cd53e5a2b0>,\n",
       " 'immunoprecipitations': <gensim.models.keyedvectors.Vocab at 0x2cd53e5a2e8>,\n",
       " 'nontransfected': <gensim.models.keyedvectors.Vocab at 0x2cd53e5a320>,\n",
       " 'antibodies': <gensim.models.keyedvectors.Vocab at 0x2cd53e5a358>,\n",
       " 'cognate': <gensim.models.keyedvectors.Vocab at 0x2cd53e5a390>,\n",
       " 'blotting': <gensim.models.keyedvectors.Vocab at 0x2cd53e5a3c8>,\n",
       " 'readily': <gensim.models.keyedvectors.Vocab at 0x2cd53e5a400>,\n",
       " 'not': <gensim.models.keyedvectors.Vocab at 0x2cd53e5a438>,\n",
       " 'confirm': <gensim.models.keyedvectors.Vocab at 0x2cd53e5a470>,\n",
       " 'physical': <gensim.models.keyedvectors.Vocab at 0x2cd53e5a4a8>,\n",
       " 'unveil': <gensim.models.keyedvectors.Vocab at 0x2cd53e5a4e0>,\n",
       " 'hypothesized': <gensim.models.keyedvectors.Vocab at 0x2cd53e5a518>,\n",
       " 'GST': <gensim.models.keyedvectors.Vocab at 0x2cd53e5a550>,\n",
       " 'StrepII': <gensim.models.keyedvectors.Vocab at 0x2cd53e5a588>,\n",
       " 'insect': <gensim.models.keyedvectors.Vocab at 0x2cd53e5a5c0>,\n",
       " 'individually': <gensim.models.keyedvectors.Vocab at 0x2cd53e5a5f8>,\n",
       " 'combination': <gensim.models.keyedvectors.Vocab at 0x2cd53e5a630>,\n",
       " 'copurified': <gensim.models.keyedvectors.Vocab at 0x2cd53e5a668>,\n",
       " 'confirming': <gensim.models.keyedvectors.Vocab at 0x2cd53e5a6a0>,\n",
       " 'yet': <gensim.models.keyedvectors.Vocab at 0x2cd53e5a6d8>,\n",
       " 'another': <gensim.models.keyedvectors.Vocab at 0x2cd53e5a710>,\n",
       " 'model': <gensim.models.keyedvectors.Vocab at 0x2cd53e5a748>,\n",
       " 'purified': <gensim.models.keyedvectors.Vocab at 0x2cd53e5a780>,\n",
       " 'histone': <gensim.models.keyedvectors.Vocab at 0x2cd53e5a7b8>,\n",
       " 'generic': <gensim.models.keyedvectors.Vocab at 0x2cd53e5a7f0>,\n",
       " 'substrate': <gensim.models.keyedvectors.Vocab at 0x2cd53e5a828>,\n",
       " 'Histone': <gensim.models.keyedvectors.Vocab at 0x2cd53e5a860>,\n",
       " 'No': <gensim.models.keyedvectors.Vocab at 0x2cd53e5a898>,\n",
       " 'alone': <gensim.models.keyedvectors.Vocab at 0x2cd53e5a8d0>,\n",
       " 'investigated': <gensim.models.keyedvectors.Vocab at 0x2cd53e5a908>,\n",
       " 'whereas': <gensim.models.keyedvectors.Vocab at 0x2cd53e5a940>,\n",
       " 'Lysates': <gensim.models.keyedvectors.Vocab at 0x2cd53e5a978>,\n",
       " 'glutathione': <gensim.models.keyedvectors.Vocab at 0x2cd53e5a9b0>,\n",
       " 'Sepharose': <gensim.models.keyedvectors.Vocab at 0x2cd53e5a9e8>,\n",
       " 'matrix': <gensim.models.keyedvectors.Vocab at 0x2cd53e5aa20>,\n",
       " 'capture': <gensim.models.keyedvectors.Vocab at 0x2cd53e5aa58>,\n",
       " 'STR': <gensim.models.keyedvectors.Vocab at 0x2cd53e5aa90>,\n",
       " 'Purified': <gensim.models.keyedvectors.Vocab at 0x2cd53e5aac8>,\n",
       " 'blots': <gensim.models.keyedvectors.Vocab at 0x2cd53e5ab00>,\n",
       " 'Top': <gensim.models.keyedvectors.Vocab at 0x2cd53e5ab38>,\n",
       " 'Upper': <gensim.models.keyedvectors.Vocab at 0x2cd53e5ab70>,\n",
       " 'Middle': <gensim.models.keyedvectors.Vocab at 0x2cd53e5aba8>,\n",
       " 'determined': <gensim.models.keyedvectors.Vocab at 0x2cd53e5abe0>,\n",
       " 'autoradiography': <gensim.models.keyedvectors.Vocab at 0x2cd53e5ac18>,\n",
       " 'added': <gensim.models.keyedvectors.Vocab at 0x2cd53e5ac50>,\n",
       " 'amounts': <gensim.models.keyedvectors.Vocab at 0x2cd53e5ac88>,\n",
       " 'visualized': <gensim.models.keyedvectors.Vocab at 0x2cd53e5acc0>,\n",
       " 'Coomassie': <gensim.models.keyedvectors.Vocab at 0x2cd53e5acf8>,\n",
       " 'staining': <gensim.models.keyedvectors.Vocab at 0x2cd53e5ad30>,\n",
       " 'Lower': <gensim.models.keyedvectors.Vocab at 0x2cd53e5ad68>,\n",
       " 'Bottom': <gensim.models.keyedvectors.Vocab at 0x2cd53e5ada0>,\n",
       " 'Same': <gensim.models.keyedvectors.Vocab at 0x2cd53e5add8>,\n",
       " 'RAF': <gensim.models.keyedvectors.Vocab at 0x2cd53e5ae10>,\n",
       " 'activate': <gensim.models.keyedvectors.Vocab at 0x2cd53e5ae48>,\n",
       " 'involved': <gensim.models.keyedvectors.Vocab at 0x2cd53e5ae80>,\n",
       " 'aim': <gensim.models.keyedvectors.Vocab at 0x2cd53e5aeb8>,\n",
       " 'at': <gensim.models.keyedvectors.Vocab at 0x2cd53e5aef0>,\n",
       " 'highly': <gensim.models.keyedvectors.Vocab at 0x2cd53e5af28>,\n",
       " 'specific': <gensim.models.keyedvectors.Vocab at 0x2cd53e5af60>,\n",
       " 'siRNA': <gensim.models.keyedvectors.Vocab at 0x2cd53e5af98>,\n",
       " 'pools': <gensim.models.keyedvectors.Vocab at 0x2cd53e5afd0>,\n",
       " 'mix': <gensim.models.keyedvectors.Vocab at 0x2cd53e5f048>,\n",
       " 'four': <gensim.models.keyedvectors.Vocab at 0x2cd53e5f080>,\n",
       " 'siRNAs': <gensim.models.keyedvectors.Vocab at 0x2cd53e5f0b8>,\n",
       " 'low': <gensim.models.keyedvectors.Vocab at 0x2cd53e5f0f0>,\n",
       " 'final': <gensim.models.keyedvectors.Vocab at 0x2cd53e5f128>,\n",
       " 'concentration': <gensim.models.keyedvectors.Vocab at 0x2cd53e5f160>,\n",
       " 'nM': <gensim.models.keyedvectors.Vocab at 0x2cd53e5f198>,\n",
       " 'Both': <gensim.models.keyedvectors.Vocab at 0x2cd53e5f1d0>,\n",
       " 'silenced': <gensim.models.keyedvectors.Vocab at 0x2cd53e5f208>,\n",
       " 'targets': <gensim.models.keyedvectors.Vocab at 0x2cd53e5f240>,\n",
       " 'interestingly': <gensim.models.keyedvectors.Vocab at 0x2cd53e5f278>,\n",
       " 'pool': <gensim.models.keyedvectors.Vocab at 0x2cd53e5f2b0>,\n",
       " 'caused': <gensim.models.keyedvectors.Vocab at 0x2cd53e5f2e8>,\n",
       " 'marked': <gensim.models.keyedvectors.Vocab at 0x2cd53e5f320>,\n",
       " 'decrease': <gensim.models.keyedvectors.Vocab at 0x2cd53e5f358>,\n",
       " 'suggest': <gensim.models.keyedvectors.Vocab at 0x2cd53e5f390>,\n",
       " 'stabilizes': <gensim.models.keyedvectors.Vocab at 0x2cd53e5f3c8>,\n",
       " 'induced': <gensim.models.keyedvectors.Vocab at 0x2cd53e5f400>,\n",
       " 'mRNA': <gensim.models.keyedvectors.Vocab at 0x2cd53e5f438>,\n",
       " 'phosphorylated': <gensim.models.keyedvectors.Vocab at 0x2cd53e5f470>,\n",
       " 'similarly': <gensim.models.keyedvectors.Vocab at 0x2cd53e5f4a8>,\n",
       " 'As': <gensim.models.keyedvectors.Vocab at 0x2cd53e5f4e0>,\n",
       " 'expected': <gensim.models.keyedvectors.Vocab at 0x2cd53e5f518>,\n",
       " 'effects': <gensim.models.keyedvectors.Vocab at 0x2cd53e5f550>,\n",
       " 'sensitivity': <gensim.models.keyedvectors.Vocab at 0x2cd53e5f588>,\n",
       " 'similar': <gensim.models.keyedvectors.Vocab at 0x2cd53e5f5c0>,\n",
       " 'combined': <gensim.models.keyedvectors.Vocab at 0x2cd53e5f5f8>,\n",
       " 'did': <gensim.models.keyedvectors.Vocab at 0x2cd53e5f630>,\n",
       " 'result': <gensim.models.keyedvectors.Vocab at 0x2cd53e5f668>,\n",
       " 'higher': <gensim.models.keyedvectors.Vocab at 0x2cd53e5f6a0>,\n",
       " 'drug': <gensim.models.keyedvectors.Vocab at 0x2cd53e5f6d8>,\n",
       " 'functional': <gensim.models.keyedvectors.Vocab at 0x2cd53e5f710>,\n",
       " 'negatively': <gensim.models.keyedvectors.Vocab at 0x2cd53e5f748>,\n",
       " 'up': <gensim.models.keyedvectors.Vocab at 0x2cd53e5f780>,\n",
       " 'regulates': <gensim.models.keyedvectors.Vocab at 0x2cd53e5f7b8>,\n",
       " 'response': <gensim.models.keyedvectors.Vocab at 0x2cd53e5f7f0>,\n",
       " 'mediated': <gensim.models.keyedvectors.Vocab at 0x2cd53e5f828>,\n",
       " 'quantification': <gensim.models.keyedvectors.Vocab at 0x2cd53e5f860>,\n",
       " 'Quantitative': <gensim.models.keyedvectors.Vocab at 0x2cd53e5f898>,\n",
       " 'RT': <gensim.models.keyedvectors.Vocab at 0x2cd53e5f8d0>,\n",
       " 'PCR': <gensim.models.keyedvectors.Vocab at 0x2cd53e5f908>,\n",
       " 'â‰¤': <gensim.models.keyedvectors.Vocab at 0x2cd53e5f940>,\n",
       " 'wished': <gensim.models.keyedvectors.Vocab at 0x2cd53e5f978>,\n",
       " 'explore': <gensim.models.keyedvectors.Vocab at 0x2cd53e5f9b0>,\n",
       " 'short': <gensim.models.keyedvectors.Vocab at 0x2cd53e5f9e8>,\n",
       " 'lived': <gensim.models.keyedvectors.Vocab at 0x2cd53e5fa20>,\n",
       " 'degraded': <gensim.models.keyedvectors.Vocab at 0x2cd53e5fa58>,\n",
       " 'straightforward': <gensim.models.keyedvectors.Vocab at 0x2cd53e5fa90>,\n",
       " 'hypothesis': <gensim.models.keyedvectors.Vocab at 0x2cd53e5fac8>,\n",
       " 'impact': <gensim.models.keyedvectors.Vocab at 0x2cd53e5fb00>,\n",
       " 'Flag': <gensim.models.keyedvectors.Vocab at 0x2cd53e5fb38>,\n",
       " 'well': <gensim.models.keyedvectors.Vocab at 0x2cd53e5fb70>,\n",
       " 'marginally': <gensim.models.keyedvectors.Vocab at 0x2cd53e5fba8>,\n",
       " 'fluctuated': <gensim.models.keyedvectors.Vocab at 0x2cd53e5fbe0>,\n",
       " 'tagged': <gensim.models.keyedvectors.Vocab at 0x2cd53e5fc18>,\n",
       " 'dramatically': <gensim.models.keyedvectors.Vocab at 0x2cd53e5fc50>,\n",
       " 'excess': <gensim.models.keyedvectors.Vocab at 0x2cd53e5fc88>,\n",
       " 'they': <gensim.models.keyedvectors.Vocab at 0x2cd53e5fcc0>,\n",
       " 'major': <gensim.models.keyedvectors.Vocab at 0x2cd53e5fcf8>,\n",
       " 'involves': <gensim.models.keyedvectors.Vocab at 0x2cd53e5fd30>,\n",
       " 'Treatment': <gensim.models.keyedvectors.Vocab at 0x2cd53e5fd68>,\n",
       " 'inhibitor': <gensim.models.keyedvectors.Vocab at 0x2cd53e5fda0>,\n",
       " 'MG': <gensim.models.keyedvectors.Vocab at 0x2cd53e5fdd8>,\n",
       " 'rescued': <gensim.models.keyedvectors.Vocab at 0x2cd53e5fe10>,\n",
       " 'stability': <gensim.models.keyedvectors.Vocab at 0x2cd53e5fe48>,\n",
       " 'cotransfected': <gensim.models.keyedvectors.Vocab at 0x2cd53e5fe80>,\n",
       " 'vectors': <gensim.models.keyedvectors.Vocab at 0x2cd53e5feb8>,\n",
       " 'latter': <gensim.models.keyedvectors.Vocab at 0x2cd53e5fef0>,\n",
       " 'treated': <gensim.models.keyedvectors.Vocab at 0x2cd53e5ff28>,\n",
       " 'h': <gensim.models.keyedvectors.Vocab at 0x2cd53e5ff60>,\n",
       " 'Proper': <gensim.models.keyedvectors.Vocab at 0x2cd53e5ff98>,\n",
       " 'verified': <gensim.models.keyedvectors.Vocab at 0x2cd53e5ffd0>,\n",
       " 'mass': <gensim.models.keyedvectors.Vocab at 0x2cd53e62048>,\n",
       " 'spectrometry': <gensim.models.keyedvectors.Vocab at 0x2cd53e62080>,\n",
       " 'existence': <gensim.models.keyedvectors.Vocab at 0x2cd53e620b8>,\n",
       " 'multiple': <gensim.models.keyedvectors.Vocab at 0x2cd53e620f0>,\n",
       " 'residues': <gensim.models.keyedvectors.Vocab at 0x2cd53e62128>,\n",
       " 'among': <gensim.models.keyedvectors.Vocab at 0x2cd53e62160>,\n",
       " 'positions': <gensim.models.keyedvectors.Vocab at 0x2cd53e62198>,\n",
       " 'may': <gensim.models.keyedvectors.Vocab at 0x2cd53e621d0>,\n",
       " 'phosphodegron': <gensim.models.keyedvectors.Vocab at 0x2cd53e62208>,\n",
       " 'Figs': <gensim.models.keyedvectors.Vocab at 0x2cd53e62240>,\n",
       " 'â€': <gensim.models.keyedvectors.Vocab at 0x2cd53e62278>,\n",
       " 'finding': <gensim.models.keyedvectors.Vocab at 0x2cd53e622b0>,\n",
       " 'SASA': <gensim.models.keyedvectors.Vocab at 0x2cd53e622e8>,\n",
       " 'alanine': <gensim.models.keyedvectors.Vocab at 0x2cd53e62320>,\n",
       " 'substitutions': <gensim.models.keyedvectors.Vocab at 0x2cd53e62358>,\n",
       " 'sites': <gensim.models.keyedvectors.Vocab at 0x2cd53e62390>,\n",
       " 'small': <gensim.models.keyedvectors.Vocab at 0x2cd53e623c8>,\n",
       " 'reproducible': <gensim.models.keyedvectors.Vocab at 0x2cd53e62400>,\n",
       " 'significant': <gensim.models.keyedvectors.Vocab at 0x2cd53e62438>,\n",
       " 'Ser': <gensim.models.keyedvectors.Vocab at 0x2cd53e62470>,\n",
       " 'establish': <gensim.models.keyedvectors.Vocab at 0x2cd53e624a8>,\n",
       " 'direct': <gensim.models.keyedvectors.Vocab at 0x2cd53e624e0>,\n",
       " 'link': <gensim.models.keyedvectors.Vocab at 0x2cd53e62518>,\n",
       " 'differ': <gensim.models.keyedvectors.Vocab at 0x2cd53e62550>,\n",
       " 'contrary': <gensim.models.keyedvectors.Vocab at 0x2cd53e62588>,\n",
       " 'DBM': <gensim.models.keyedvectors.Vocab at 0x2cd53e625c0>,\n",
       " 'deletion': <gensim.models.keyedvectors.Vocab at 0x2cd53e625f8>,\n",
       " 'N': <gensim.models.keyedvectors.Vocab at 0x2cd53e62630>,\n",
       " 'destruction': <gensim.models.keyedvectors.Vocab at 0x2cd53e62668>,\n",
       " 'previously': <gensim.models.keyedvectors.Vocab at 0x2cd53e626a0>,\n",
       " 'APC': <gensim.models.keyedvectors.Vocab at 0x2cd53e626d8>,\n",
       " 'Cdh': <gensim.models.keyedvectors.Vocab at 0x2cd53e62710>,\n",
       " 'remained': <gensim.models.keyedvectors.Vocab at 0x2cd53e62748>,\n",
       " 'insensitive': <gensim.models.keyedvectors.Vocab at 0x2cd53e62780>,\n",
       " 'directly': <gensim.models.keyedvectors.Vocab at 0x2cd53e627b8>,\n",
       " 'studied': <gensim.models.keyedvectors.Vocab at 0x2cd53e627f0>,\n",
       " 'lymphoblastoid': <gensim.models.keyedvectors.Vocab at 0x2cd53e62828>,\n",
       " 'mutation': <gensim.models.keyedvectors.Vocab at 0x2cd53e62860>,\n",
       " 'G': <gensim.models.keyedvectors.Vocab at 0x2cd53e62898>,\n",
       " 'predicted': <gensim.models.keyedvectors.Vocab at 0x2cd53e628d0>,\n",
       " 'aberrant': <gensim.models.keyedvectors.Vocab at 0x2cd53e62908>,\n",
       " 'accordance': <gensim.models.keyedvectors.Vocab at 0x2cd53e62940>,\n",
       " 'incomplete': <gensim.models.keyedvectors.Vocab at 0x2cd53e62978>,\n",
       " 'skewing': <gensim.models.keyedvectors.Vocab at 0x2cd53e629b0>,\n",
       " 'X': <gensim.models.keyedvectors.Vocab at 0x2cd53e629e8>,\n",
       " 'chromosome': <gensim.models.keyedvectors.Vocab at 0x2cd53e62a20>,\n",
       " 'inactivation': <gensim.models.keyedvectors.Vocab at 0x2cd53e62a58>,\n",
       " 'preceding': <gensim.models.keyedvectors.Vocab at 0x2cd53e62a90>,\n",
       " 'quantitative': <gensim.models.keyedvectors.Vocab at 0x2cd53e62ac8>,\n",
       " 'showed': <gensim.models.keyedvectors.Vocab at 0x2cd53e62b00>,\n",
       " 'indeed': <gensim.models.keyedvectors.Vocab at 0x2cd53e62b38>,\n",
       " 'Decreased': <gensim.models.keyedvectors.Vocab at 0x2cd53e62b70>,\n",
       " 'healthy': <gensim.models.keyedvectors.Vocab at 0x2cd53e62ba8>,\n",
       " 'individual': <gensim.models.keyedvectors.Vocab at 0x2cd53e62be0>,\n",
       " 'same': <gensim.models.keyedvectors.Vocab at 0x2cd53e62c18>,\n",
       " 'vector': <gensim.models.keyedvectors.Vocab at 0x2cd53e62c50>,\n",
       " 'directing': <gensim.models.keyedvectors.Vocab at 0x2cd53e62c88>,\n",
       " 'revealing': <gensim.models.keyedvectors.Vocab at 0x2cd53e62cc0>,\n",
       " 'endogenously': <gensim.models.keyedvectors.Vocab at 0x2cd53e62cf8>,\n",
       " 'era': <gensim.models.keyedvectors.Vocab at 0x2cd53e62d30>,\n",
       " 'genetic': <gensim.models.keyedvectors.Vocab at 0x2cd53e62d68>,\n",
       " 'disease': <gensim.models.keyedvectors.Vocab at 0x2cd53e62da0>,\n",
       " 'so': <gensim.models.keyedvectors.Vocab at 0x2cd53e62dd8>,\n",
       " 'far': <gensim.models.keyedvectors.Vocab at 0x2cd53e62e10>,\n",
       " 'closest': <gensim.models.keyedvectors.Vocab at 0x2cd53e62e48>,\n",
       " 'paralogs': <gensim.models.keyedvectors.Vocab at 0x2cd53e62e80>,\n",
       " 'within': <gensim.models.keyedvectors.Vocab at 0x2cd53e62eb8>,\n",
       " 'L': <gensim.models.keyedvectors.Vocab at 0x2cd53e62ef0>,\n",
       " 'Interestingly': <gensim.models.keyedvectors.Vocab at 0x2cd53e62f28>,\n",
       " 'paralog': <gensim.models.keyedvectors.Vocab at 0x2cd53e62f60>,\n",
       " 'fact': <gensim.models.keyedvectors.Vocab at 0x2cd53e62f98>,\n",
       " 'none': <gensim.models.keyedvectors.Vocab at 0x2cd53e62fd0>,\n",
       " 'robustly': <gensim.models.keyedvectors.Vocab at 0x2cd53e65048>,\n",
       " 'suggests': <gensim.models.keyedvectors.Vocab at 0x2cd53e65080>,\n",
       " 'important': <gensim.models.keyedvectors.Vocab at 0x2cd53e650b8>,\n",
       " 'regulating': <gensim.models.keyedvectors.Vocab at 0x2cd53e650f0>,\n",
       " 'relevance': <gensim.models.keyedvectors.Vocab at 0x2cd53e65128>,\n",
       " 'supported': <gensim.models.keyedvectors.Vocab at 0x2cd53e65160>,\n",
       " 'seem': <gensim.models.keyedvectors.Vocab at 0x2cd53e65198>,\n",
       " 'enhance': <gensim.models.keyedvectors.Vocab at 0x2cd53e651d0>,\n",
       " 'each': <gensim.models.keyedvectors.Vocab at 0x2cd53e65208>,\n",
       " 'otherâ€™s': <gensim.models.keyedvectors.Vocab at 0x2cd53e65240>,\n",
       " 'judged': <gensim.models.keyedvectors.Vocab at 0x2cd53e65278>,\n",
       " 'much': <gensim.models.keyedvectors.Vocab at 0x2cd53e652b0>,\n",
       " 'reduced': <gensim.models.keyedvectors.Vocab at 0x2cd53e652e8>,\n",
       " 'subject': <gensim.models.keyedvectors.Vocab at 0x2cd53e65320>,\n",
       " 'ubiquitin': <gensim.models.keyedvectors.Vocab at 0x2cd53e65358>,\n",
       " 'Our': <gensim.models.keyedvectors.Vocab at 0x2cd53e65390>,\n",
       " 'protects': <gensim.models.keyedvectors.Vocab at 0x2cd53e653c8>,\n",
       " 'such': <gensim.models.keyedvectors.Vocab at 0x2cd53e65400>,\n",
       " 'least': <gensim.models.keyedvectors.Vocab at 0x2cd53e65438>,\n",
       " 'enhanced': <gensim.models.keyedvectors.Vocab at 0x2cd53e65470>,\n",
       " 'independently': <gensim.models.keyedvectors.Vocab at 0x2cd53e654a8>,\n",
       " 'uncover': <gensim.models.keyedvectors.Vocab at 0x2cd53e654e0>,\n",
       " 'demonstrating': <gensim.models.keyedvectors.Vocab at 0x2cd53e65518>,\n",
       " 'named': <gensim.models.keyedvectors.Vocab at 0x2cd53e65550>,\n",
       " 'sole': <gensim.models.keyedvectors.Vocab at 0x2cd53e65588>,\n",
       " 'basis': <gensim.models.keyedvectors.Vocab at 0x2cd53e655c0>,\n",
       " 'genuine': <gensim.models.keyedvectors.Vocab at 0x2cd53e655f8>,\n",
       " 'do': <gensim.models.keyedvectors.Vocab at 0x2cd53e65630>,\n",
       " 'produce': <gensim.models.keyedvectors.Vocab at 0x2cd53e65668>,\n",
       " 'Hence': <gensim.models.keyedvectors.Vocab at 0x2cd53e656a0>,\n",
       " 'regardless': <gensim.models.keyedvectors.Vocab at 0x2cd53e656d8>,\n",
       " 'mRNAs': <gensim.models.keyedvectors.Vocab at 0x2cd53e65710>,\n",
       " 'undergo': <gensim.models.keyedvectors.Vocab at 0x2cd53e65748>,\n",
       " 'nonsense': <gensim.models.keyedvectors.Vocab at 0x2cd53e65780>,\n",
       " 'decay': <gensim.models.keyedvectors.Vocab at 0x2cd53e657b8>,\n",
       " 'suggested': <gensim.models.keyedvectors.Vocab at 0x2cd53e657f0>,\n",
       " 'give': <gensim.models.keyedvectors.Vocab at 0x2cd53e65828>,\n",
       " 'rise': <gensim.models.keyedvectors.Vocab at 0x2cd53e65860>,\n",
       " 'affected': <gensim.models.keyedvectors.Vocab at 0x2cd53e65898>,\n",
       " 'must': <gensim.models.keyedvectors.Vocab at 0x2cd53e658d0>,\n",
       " 'exhibit': <gensim.models.keyedvectors.Vocab at 0x2cd53e65908>,\n",
       " 'compromised': <gensim.models.keyedvectors.Vocab at 0x2cd53e65940>,\n",
       " 'tissues': <gensim.models.keyedvectors.Vocab at 0x2cd53e65978>,\n",
       " 'during': <gensim.models.keyedvectors.Vocab at 0x2cd53e659b0>,\n",
       " 'stages': <gensim.models.keyedvectors.Vocab at 0x2cd53e659e8>,\n",
       " 'interactor': <gensim.models.keyedvectors.Vocab at 0x2cd53e65a20>,\n",
       " 'seems': <gensim.models.keyedvectors.Vocab at 0x2cd53e65a58>,\n",
       " 'exerted': <gensim.models.keyedvectors.Vocab at 0x2cd53e65a90>,\n",
       " 'very': <gensim.models.keyedvectors.Vocab at 0x2cd53e65ac8>,\n",
       " 'fine': <gensim.models.keyedvectors.Vocab at 0x2cd53e65b00>,\n",
       " 'partially': <gensim.models.keyedvectors.Vocab at 0x2cd53e65b38>,\n",
       " 'achieved': <gensim.models.keyedvectors.Vocab at 0x2cd53e65b70>,\n",
       " 'findings': <gensim.models.keyedvectors.Vocab at 0x2cd53e65ba8>,\n",
       " 'offer': <gensim.models.keyedvectors.Vocab at 0x2cd53e65be0>,\n",
       " 'explanation': <gensim.models.keyedvectors.Vocab at 0x2cd53e65c18>,\n",
       " 'already': <gensim.models.keyedvectors.Vocab at 0x2cd53e65c50>,\n",
       " 'reported': <gensim.models.keyedvectors.Vocab at 0x2cd53e65c88>,\n",
       " 'regulation': <gensim.models.keyedvectors.Vocab at 0x2cd53e65cc0>,\n",
       " 'bring': <gensim.models.keyedvectors.Vocab at 0x2cd53e65cf8>,\n",
       " 'evidence': <gensim.models.keyedvectors.Vocab at 0x2cd53e65d30>,\n",
       " 'noncanonical': <gensim.models.keyedvectors.Vocab at 0x2cd53e65d68>,\n",
       " 'Î²': <gensim.models.keyedvectors.Vocab at 0x2cd53e65da0>,\n",
       " 'TRCP': <gensim.models.keyedvectors.Vocab at 0x2cd53e65dd8>,\n",
       " 'Because': <gensim.models.keyedvectors.Vocab at 0x2cd53e65e10>,\n",
       " 'precede': <gensim.models.keyedvectors.Vocab at 0x2cd53e65e48>,\n",
       " 'proline': <gensim.models.keyedvectors.Vocab at 0x2cd53e65e80>,\n",
       " 'conform': <gensim.models.keyedvectors.Vocab at 0x2cd53e65eb8>,\n",
       " 'usual': <gensim.models.keyedvectors.Vocab at 0x2cd53e65ef0>,\n",
       " 'called': <gensim.models.keyedvectors.Vocab at 0x2cd53e65f28>,\n",
       " 'belong': <gensim.models.keyedvectors.Vocab at 0x2cd53e65f60>,\n",
       " 'have': <gensim.models.keyedvectors.Vocab at 0x2cd53e65f98>,\n",
       " 'phosphorylate': <gensim.models.keyedvectors.Vocab at 0x2cd53e65fd0>,\n",
       " 'motifs': <gensim.models.keyedvectors.Vocab at 0x2cd53e69048>,\n",
       " 'nonâ€': <gensim.models.keyedvectors.Vocab at 0x2cd53e69080>,\n",
       " 'directed': <gensim.models.keyedvectors.Vocab at 0x2cd53e690b8>,\n",
       " 'fashion': <gensim.models.keyedvectors.Vocab at 0x2cd53e690f0>,\n",
       " 'especially': <gensim.models.keyedvectors.Vocab at 0x2cd53e69128>,\n",
       " 'context': <gensim.models.keyedvectors.Vocab at 0x2cd53e69160>,\n",
       " 'docking': <gensim.models.keyedvectors.Vocab at 0x2cd53e69198>,\n",
       " 'high': <gensim.models.keyedvectors.Vocab at 0x2cd53e691d0>,\n",
       " 'affinity': <gensim.models.keyedvectors.Vocab at 0x2cd53e69208>,\n",
       " 'Pointed': <gensim.models.keyedvectors.Vocab at 0x2cd53e69240>,\n",
       " 'would': <gensim.models.keyedvectors.Vocab at 0x2cd53e69278>,\n",
       " 'allow': <gensim.models.keyedvectors.Vocab at 0x2cd53e692b0>,\n",
       " 'atypical': <gensim.models.keyedvectors.Vocab at 0x2cd53e692e8>,\n",
       " 'players': <gensim.models.keyedvectors.Vocab at 0x2cd53e69320>,\n",
       " 'including': <gensim.models.keyedvectors.Vocab at 0x2cd53e69358>,\n",
       " 'cullin': <gensim.models.keyedvectors.Vocab at 0x2cd53e69390>,\n",
       " 'RING': <gensim.models.keyedvectors.Vocab at 0x2cd53e693c8>,\n",
       " 'ligase': <gensim.models.keyedvectors.Vocab at 0x2cd53e69400>,\n",
       " 'CRL': <gensim.models.keyedvectors.Vocab at 0x2cd53e69438>,\n",
       " 'formal': <gensim.models.keyedvectors.Vocab at 0x2cd53e69470>,\n",
       " 'identification': <gensim.models.keyedvectors.Vocab at 0x2cd53e694a8>,\n",
       " 'elucidation': <gensim.models.keyedvectors.Vocab at 0x2cd53e694e0>,\n",
       " 'concerted': <gensim.models.keyedvectors.Vocab at 0x2cd53e69518>,\n",
       " 'action': <gensim.models.keyedvectors.Vocab at 0x2cd53e69550>,\n",
       " 'ligases': <gensim.models.keyedvectors.Vocab at 0x2cd53e69588>,\n",
       " 'will': <gensim.models.keyedvectors.Vocab at 0x2cd53e695c0>,\n",
       " 'require': <gensim.models.keyedvectors.Vocab at 0x2cd53e695f8>,\n",
       " 'further': <gensim.models.keyedvectors.Vocab at 0x2cd53e69630>,\n",
       " 'studies': <gensim.models.keyedvectors.Vocab at 0x2cd53e69668>,\n",
       " 'present': <gensim.models.keyedvectors.Vocab at 0x2cd53e696a0>,\n",
       " 'medical': <gensim.models.keyedvectors.Vocab at 0x2cd53e696d8>,\n",
       " 'implications': <gensim.models.keyedvectors.Vocab at 0x2cd53e69710>,\n",
       " 'First': <gensim.models.keyedvectors.Vocab at 0x2cd53e69748>,\n",
       " 'frequently': <gensim.models.keyedvectors.Vocab at 0x2cd53e69780>,\n",
       " 'deregulated': <gensim.models.keyedvectors.Vocab at 0x2cd53e697b8>,\n",
       " 'many': <gensim.models.keyedvectors.Vocab at 0x2cd53e697f0>,\n",
       " 'cancers': <gensim.models.keyedvectors.Vocab at 0x2cd53e69828>,\n",
       " 'Second': <gensim.models.keyedvectors.Vocab at 0x2cd53e69860>,\n",
       " 'contribute': <gensim.models.keyedvectors.Vocab at 0x2cd53e69898>,\n",
       " 'understanding': <gensim.models.keyedvectors.Vocab at 0x2cd53e698d0>,\n",
       " 'causing': <gensim.models.keyedvectors.Vocab at 0x2cd53e69908>,\n",
       " 'like': <gensim.models.keyedvectors.Vocab at 0x2cd53e69940>,\n",
       " 'could': <gensim.models.keyedvectors.Vocab at 0x2cd53e69978>,\n",
       " 'predictive': <gensim.models.keyedvectors.Vocab at 0x2cd53e699b0>,\n",
       " 'clinical': <gensim.models.keyedvectors.Vocab at 0x2cd53e699e8>,\n",
       " 'marker': <gensim.models.keyedvectors.Vocab at 0x2cd53e69a20>,\n",
       " 'hormone': <gensim.models.keyedvectors.Vocab at 0x2cd53e69a58>,\n",
       " 'therapy': <gensim.models.keyedvectors.Vocab at 0x2cd53e69a90>,\n",
       " 'ERÎ±': <gensim.models.keyedvectors.Vocab at 0x2cd53e69ac8>,\n",
       " 'Third': <gensim.models.keyedvectors.Vocab at 0x2cd53e69b00>,\n",
       " 'interesting': <gensim.models.keyedvectors.Vocab at 0x2cd53e69b38>,\n",
       " 'Ets': <gensim.models.keyedvectors.Vocab at 0x2cd53e69b70>,\n",
       " 'transgenic': <gensim.models.keyedvectors.Vocab at 0x2cd53e69ba8>,\n",
       " 'mice': <gensim.models.keyedvectors.Vocab at 0x2cd53e69be0>,\n",
       " 'showing': <gensim.models.keyedvectors.Vocab at 0x2cd53e69c18>,\n",
       " 'less': <gensim.models.keyedvectors.Vocab at 0x2cd53e69c50>,\n",
       " 'twofold': <gensim.models.keyedvectors.Vocab at 0x2cd53e69c88>,\n",
       " 'severe': <gensim.models.keyedvectors.Vocab at 0x2cd53e69cc0>,\n",
       " 'cranial': <gensim.models.keyedvectors.Vocab at 0x2cd53e69cf8>,\n",
       " 'abnormalities': <gensim.models.keyedvectors.Vocab at 0x2cd53e69d30>,\n",
       " 'part': <gensim.models.keyedvectors.Vocab at 0x2cd53e69d68>,\n",
       " 'consequence': <gensim.models.keyedvectors.Vocab at 0x2cd53e69da0>,\n",
       " 'risk': <gensim.models.keyedvectors.Vocab at 0x2cd53e69dd8>,\n",
       " 'develop': <gensim.models.keyedvectors.Vocab at 0x2cd53e69e10>,\n",
       " 'certain': <gensim.models.keyedvectors.Vocab at 0x2cd53e69e48>,\n",
       " 'types': <gensim.models.keyedvectors.Vocab at 0x2cd53e69e80>,\n",
       " 'Studies': <gensim.models.keyedvectors.Vocab at 0x2cd53e69eb8>,\n",
       " 'various': <gensim.models.keyedvectors.Vocab at 0x2cd53e69ef0>,\n",
       " 'models': <gensim.models.keyedvectors.Vocab at 0x2cd53e69f28>,\n",
       " 'Down': <gensim.models.keyedvectors.Vocab at 0x2cd53e69f60>,\n",
       " 'three': <gensim.models.keyedvectors.Vocab at 0x2cd53e69f98>,\n",
       " 'copies': <gensim.models.keyedvectors.Vocab at 0x2cd53e69fd0>,\n",
       " 'exist': <gensim.models.keyedvectors.Vocab at 0x2cd53e6d048>,\n",
       " 'dosage': <gensim.models.keyedvectors.Vocab at 0x2cd53e6d080>,\n",
       " 'repress': <gensim.models.keyedvectors.Vocab at 0x2cd53e6d0b8>,\n",
       " 'promote': <gensim.models.keyedvectors.Vocab at 0x2cd53e6d0f0>,\n",
       " 'growth': <gensim.models.keyedvectors.Vocab at 0x2cd53e6d128>,\n",
       " 'hence': <gensim.models.keyedvectors.Vocab at 0x2cd53e6d160>,\n",
       " 'exerts': <gensim.models.keyedvectors.Vocab at 0x2cd53e6d198>,\n",
       " 'autonomous': <gensim.models.keyedvectors.Vocab at 0x2cd53e6d1d0>,\n",
       " 'few': <gensim.models.keyedvectors.Vocab at 0x2cd53e6d208>,\n",
       " 'diagnosed': <gensim.models.keyedvectors.Vocab at 0x2cd53e6d240>,\n",
       " 'nephroblastoma': <gensim.models.keyedvectors.Vocab at 0x2cd53e6d278>,\n",
       " 'facilitate': <gensim.models.keyedvectors.Vocab at 0x2cd53e6d2b0>,\n",
       " 'general': <gensim.models.keyedvectors.Vocab at 0x2cd53e6d2e8>,\n",
       " 'particular': <gensim.models.keyedvectors.Vocab at 0x2cd53e6d320>,\n",
       " 'division': <gensim.models.keyedvectors.Vocab at 0x2cd53e6d358>,\n",
       " 'suppressive': <gensim.models.keyedvectors.Vocab at 0x2cd53e6d390>,\n",
       " 'retardation': <gensim.models.keyedvectors.Vocab at 0x2cd53e6d3c8>,\n",
       " 'exhibited': <gensim.models.keyedvectors.Vocab at 0x2cd53e6d400>,\n",
       " 'strongly': <gensim.models.keyedvectors.Vocab at 0x2cd53e6d438>,\n",
       " 'proliferation': <gensim.models.keyedvectors.Vocab at 0x2cd53e6d470>,\n",
       " 'cDNAs': <gensim.models.keyedvectors.Vocab at 0x2cd53e6d4a8>,\n",
       " 'plasmid': <gensim.models.keyedvectors.Vocab at 0x2cd53e6d4e0>,\n",
       " 'constructions': <gensim.models.keyedvectors.Vocab at 0x2cd53e6d518>,\n",
       " 'production': <gensim.models.keyedvectors.Vocab at 0x2cd53e6d550>,\n",
       " 'detailed': <gensim.models.keyedvectors.Vocab at 0x2cd53e6d588>,\n",
       " 'SI': <gensim.models.keyedvectors.Vocab at 0x2cd53e6d5c0>,\n",
       " 'Materials': <gensim.models.keyedvectors.Vocab at 0x2cd53e6d5f8>,\n",
       " 'Methods': <gensim.models.keyedvectors.Vocab at 0x2cd53e6d630>,\n",
       " 'Yeast': <gensim.models.keyedvectors.Vocab at 0x2cd53e6d668>,\n",
       " 'Two': <gensim.models.keyedvectors.Vocab at 0x2cd53e6d6a0>,\n",
       " 'Hybrid': <gensim.models.keyedvectors.Vocab at 0x2cd53e6d6d8>,\n",
       " 'Interaction': <gensim.models.keyedvectors.Vocab at 0x2cd53e6d710>,\n",
       " 'Assays': <gensim.models.keyedvectors.Vocab at 0x2cd53e6d748>,\n",
       " 'described': <gensim.models.keyedvectors.Vocab at 0x2cd53e6d780>,\n",
       " 'Mammalian': <gensim.models.keyedvectors.Vocab at 0x2cd53e6d7b8>,\n",
       " 'Cell': <gensim.models.keyedvectors.Vocab at 0x2cd53e6d7f0>,\n",
       " 'Cultures': <gensim.models.keyedvectors.Vocab at 0x2cd53e6d828>,\n",
       " 'Transfections': <gensim.models.keyedvectors.Vocab at 0x2cd53e6d860>,\n",
       " 'grew': <gensim.models.keyedvectors.Vocab at 0x2cd53e6d898>,\n",
       " 'DMEM': <gensim.models.keyedvectors.Vocab at 0x2cd53e6d8d0>,\n",
       " 'supplemented': <gensim.models.keyedvectors.Vocab at 0x2cd53e6d908>,\n",
       " 'vol': <gensim.models.keyedvectors.Vocab at 0x2cd53e6d940>,\n",
       " 'FBS': <gensim.models.keyedvectors.Vocab at 0x2cd53e6d978>,\n",
       " 'Invitrogen': <gensim.models.keyedvectors.Vocab at 0x2cd53e6d9b0>,\n",
       " 'RPMI': <gensim.models.keyedvectors.Vocab at 0x2cd53e6d9e8>,\n",
       " 'GlutaMAX': <gensim.models.keyedvectors.Vocab at 0x2cd53e6da20>,\n",
       " 'Lipofectamine': <gensim.models.keyedvectors.Vocab at 0x2cd53e6da58>,\n",
       " 'RNAiMAX': <gensim.models.keyedvectors.Vocab at 0x2cd53e6da90>,\n",
       " 'Polyplus': <gensim.models.keyedvectors.Vocab at 0x2cd53e6dac8>,\n",
       " 'combinations': <gensim.models.keyedvectors.Vocab at 0x2cd53e6db00>,\n",
       " 'according': <gensim.models.keyedvectors.Vocab at 0x2cd53e6db38>,\n",
       " 'manufacturersâ€™': <gensim.models.keyedvectors.Vocab at 0x2cd53e6db70>,\n",
       " 'instructions': <gensim.models.keyedvectors.Vocab at 0x2cd53e6dba8>,\n",
       " 'electroporation': <gensim.models.keyedvectors.Vocab at 0x2cd53e6dbe0>,\n",
       " 'For': <gensim.models.keyedvectors.Vocab at 0x2cd53e6dc18>,\n",
       " 'after': <gensim.models.keyedvectors.Vocab at 0x2cd53e6dc50>,\n",
       " 'transfection': <gensim.models.keyedvectors.Vocab at 0x2cd53e6dc88>,\n",
       " 'Î¼M': <gensim.models.keyedvectors.Vocab at 0x2cd53e6dcc0>,\n",
       " 'Fisher': <gensim.models.keyedvectors.Vocab at 0x2cd53e6dcf8>,\n",
       " 'Scientific': <gensim.models.keyedvectors.Vocab at 0x2cd53e6dd30>,\n",
       " 'Coimmunoprecipitation': <gensim.models.keyedvectors.Vocab at 0x2cd53e6dd68>,\n",
       " 'Blot': <gensim.models.keyedvectors.Vocab at 0x2cd53e6dda0>,\n",
       " 'Experiments': <gensim.models.keyedvectors.Vocab at 0x2cd53e6ddd8>,\n",
       " 'collected': <gensim.models.keyedvectors.Vocab at 0x2cd53e6de10>,\n",
       " 'scraping': <gensim.models.keyedvectors.Vocab at 0x2cd53e6de48>,\n",
       " 'PBS': <gensim.models.keyedvectors.Vocab at 0x2cd53e6de80>,\n",
       " 'centrifugation': <gensim.models.keyedvectors.Vocab at 0x2cd53e6deb8>,\n",
       " 'lysed': <gensim.models.keyedvectors.Vocab at 0x2cd53e6def0>,\n",
       " 'them': <gensim.models.keyedvectors.Vocab at 0x2cd53e6df28>,\n",
       " 'sonication': <gensim.models.keyedvectors.Vocab at 0x2cd53e6df60>,\n",
       " 'lysis': <gensim.models.keyedvectors.Vocab at 0x2cd53e6df98>,\n",
       " 'buffer': <gensim.models.keyedvectors.Vocab at 0x2cd53e6dfd0>,\n",
       " 'containing': <gensim.models.keyedvectors.Vocab at 0x2cd53e70048>,\n",
       " 'mM': <gensim.models.keyedvectors.Vocab at 0x2cd53e70080>,\n",
       " 'glycerophosphate': <gensim.models.keyedvectors.Vocab at 0x2cd53e700b8>,\n",
       " 'nitrophenylphosphate': <gensim.models.keyedvectors.Vocab at 0x2cd53e700f0>,\n",
       " 'morpholino': <gensim.models.keyedvectors.Vocab at 0x2cd53e70128>,\n",
       " 'propanesulfonic': <gensim.models.keyedvectors.Vocab at 0x2cd53e70160>,\n",
       " 'pH': <gensim.models.keyedvectors.Vocab at 0x2cd53e70198>,\n",
       " 'EGTA': <gensim.models.keyedvectors.Vocab at 0x2cd53e701d0>,\n",
       " 'MgCl': <gensim.models.keyedvectors.Vocab at 0x2cd53e70208>,\n",
       " 'Na': <gensim.models.keyedvectors.Vocab at 0x2cd53e70240>,\n",
       " 'vanadate': <gensim.models.keyedvectors.Vocab at 0x2cd53e70278>,\n",
       " 'NaF': <gensim.models.keyedvectors.Vocab at 0x2cd53e702b0>,\n",
       " 'Nonidet': <gensim.models.keyedvectors.Vocab at 0x2cd53e702e8>,\n",
       " 'protease': <gensim.models.keyedvectors.Vocab at 0x2cd53e70320>,\n",
       " 'mixture': <gensim.models.keyedvectors.Vocab at 0x2cd53e70358>,\n",
       " 'Roche': <gensim.models.keyedvectors.Vocab at 0x2cd53e70390>,\n",
       " 'spun': <gensim.models.keyedvectors.Vocab at 0x2cd53e703c8>,\n",
       " 'min': <gensim.models.keyedvectors.Vocab at 0x2cd53e70400>,\n",
       " 'Ã—': <gensim.models.keyedvectors.Vocab at 0x2cd53e70438>,\n",
       " 'g': <gensim.models.keyedvectors.Vocab at 0x2cd53e70470>,\n",
       " 'Â°C': <gensim.models.keyedvectors.Vocab at 0x2cd53e704a8>,\n",
       " 'supernatants': <gensim.models.keyedvectors.Vocab at 0x2cd53e704e0>,\n",
       " 'content': <gensim.models.keyedvectors.Vocab at 0x2cd53e70518>,\n",
       " 'Bradford': <gensim.models.keyedvectors.Vocab at 0x2cd53e70550>,\n",
       " 'immunoprecipitation': <gensim.models.keyedvectors.Vocab at 0x2cd53e70588>,\n",
       " 'precleared': <gensim.models.keyedvectors.Vocab at 0x2cd53e705c0>,\n",
       " 'Î¼L': <gensim.models.keyedvectors.Vocab at 0x2cd53e705f8>,\n",
       " 'agarose': <gensim.models.keyedvectors.Vocab at 0x2cd53e70630>,\n",
       " 'beads': <gensim.models.keyedvectors.Vocab at 0x2cd53e70668>,\n",
       " 'incubated': <gensim.models.keyedvectors.Vocab at 0x2cd53e706a0>,\n",
       " 'rotating': <gensim.models.keyedvectors.Vocab at 0x2cd53e706d8>,\n",
       " 'wheel': <gensim.models.keyedvectors.Vocab at 0x2cd53e70710>,\n",
       " 's': <gensim.models.keyedvectors.Vocab at 0x2cd53e70748>,\n",
       " 'washed': <gensim.models.keyedvectors.Vocab at 0x2cd53e70780>,\n",
       " 'times': <gensim.models.keyedvectors.Vocab at 0x2cd53e707b8>,\n",
       " 'bead': <gensim.models.keyedvectors.Vocab at 0x2cd53e707f0>,\n",
       " 'Tris': <gensim.models.keyedvectors.Vocab at 0x2cd53e70828>,\n",
       " 'NaCl': <gensim.models.keyedvectors.Vocab at 0x2cd53e70860>,\n",
       " 'EDTA': <gensim.models.keyedvectors.Vocab at 0x2cd53e70898>,\n",
       " 'sample': <gensim.models.keyedvectors.Vocab at 0x2cd53e708d0>,\n",
       " 'pellets': <gensim.models.keyedvectors.Vocab at 0x2cd53e70908>,\n",
       " 'heat': <gensim.models.keyedvectors.Vocab at 0x2cd53e70940>,\n",
       " 'denatured': <gensim.models.keyedvectors.Vocab at 0x2cd53e70978>,\n",
       " 'ran': <gensim.models.keyedvectors.Vocab at 0x2cd53e709b0>,\n",
       " 'samples': <gensim.models.keyedvectors.Vocab at 0x2cd53e709e8>,\n",
       " 'Bis': <gensim.models.keyedvectors.Vocab at 0x2cd53e70a20>,\n",
       " 'SDS': <gensim.models.keyedvectors.Vocab at 0x2cd53e70a58>,\n",
       " 'PAGE': <gensim.models.keyedvectors.Vocab at 0x2cd53e70a90>,\n",
       " 'transferred': <gensim.models.keyedvectors.Vocab at 0x2cd53e70ac8>,\n",
       " 'onto': <gensim.models.keyedvectors.Vocab at 0x2cd53e70b00>,\n",
       " 'Hybond': <gensim.models.keyedvectors.Vocab at 0x2cd53e70b38>,\n",
       " 'nitrocellulose': <gensim.models.keyedvectors.Vocab at 0x2cd53e70b70>,\n",
       " 'membranes': <gensim.models.keyedvectors.Vocab at 0x2cd53e70ba8>,\n",
       " 'processed': <gensim.models.keyedvectors.Vocab at 0x2cd53e70be0>,\n",
       " 'standard': <gensim.models.keyedvectors.Vocab at 0x2cd53e70c18>,\n",
       " 'procedures': <gensim.models.keyedvectors.Vocab at 0x2cd53e70c50>,\n",
       " 'following': <gensim.models.keyedvectors.Vocab at 0x2cd53e70c88>,\n",
       " 'primary': <gensim.models.keyedvectors.Vocab at 0x2cd53e70cc0>,\n",
       " 'Abcam': <gensim.models.keyedvectors.Vocab at 0x2cd53e70cf8>,\n",
       " 'ab': <gensim.models.keyedvectors.Vocab at 0x2cd53e70d30>,\n",
       " 'R': <gensim.models.keyedvectors.Vocab at 0x2cd53e70d68>,\n",
       " 'tubulin': <gensim.models.keyedvectors.Vocab at 0x2cd53e70da0>,\n",
       " 'Santa': <gensim.models.keyedvectors.Vocab at 0x2cd53e70dd8>,\n",
       " 'Cruz': <gensim.models.keyedvectors.Vocab at 0x2cd53e70e10>,\n",
       " 'Biotechnology': <gensim.models.keyedvectors.Vocab at 0x2cd53e70e48>,\n",
       " 'Covalab': <gensim.models.keyedvectors.Vocab at 0x2cd53e70e80>,\n",
       " 'pab': <gensim.models.keyedvectors.Vocab at 0x2cd53e70eb8>,\n",
       " 'home': <gensim.models.keyedvectors.Vocab at 0x2cd53e70ef0>,\n",
       " 'made': <gensim.models.keyedvectors.Vocab at 0x2cd53e70f28>,\n",
       " 'dilution': <gensim.models.keyedvectors.Vocab at 0x2cd53e70f60>,\n",
       " 'Sigma': <gensim.models.keyedvectors.Vocab at 0x2cd53e70f98>,\n",
       " 'actin': <gensim.models.keyedvectors.Vocab at 0x2cd53e70fd0>,\n",
       " 'HRP': <gensim.models.keyedvectors.Vocab at 0x2cd53e73048>,\n",
       " 'coupled': <gensim.models.keyedvectors.Vocab at 0x2cd53e73080>,\n",
       " 'SC': <gensim.models.keyedvectors.Vocab at 0x2cd53e730b8>,\n",
       " 'Bio': <gensim.models.keyedvectors.Vocab at 0x2cd53e730f0>,\n",
       " 'Rad': <gensim.models.keyedvectors.Vocab at 0x2cd53e73128>,\n",
       " 'rabbit': <gensim.models.keyedvectors.Vocab at 0x2cd53e73160>,\n",
       " 'secondary': <gensim.models.keyedvectors.Vocab at 0x2cd53e73198>,\n",
       " 'chemiluminescence': <gensim.models.keyedvectors.Vocab at 0x2cd53e731d0>,\n",
       " 'SuperSignal': <gensim.models.keyedvectors.Vocab at 0x2cd53e73208>,\n",
       " 'West': <gensim.models.keyedvectors.Vocab at 0x2cd53e73240>,\n",
       " 'Femto': <gensim.models.keyedvectors.Vocab at 0x2cd53e73278>,\n",
       " 'Thermo': <gensim.models.keyedvectors.Vocab at 0x2cd53e732b0>,\n",
       " 'Production': <gensim.models.keyedvectors.Vocab at 0x2cd53e732e8>,\n",
       " 'Purification': <gensim.models.keyedvectors.Vocab at 0x2cd53e73320>,\n",
       " 'Recombinant': <gensim.models.keyedvectors.Vocab at 0x2cd53e73358>,\n",
       " 'Proteins': <gensim.models.keyedvectors.Vocab at 0x2cd53e73390>,\n",
       " 'generated': <gensim.models.keyedvectors.Vocab at 0x2cd53e733c8>,\n",
       " 'DH': <gensim.models.keyedvectors.Vocab at 0x2cd53e73400>,\n",
       " 'Bac': <gensim.models.keyedvectors.Vocab at 0x2cd53e73438>,\n",
       " 'Escherichia': <gensim.models.keyedvectors.Vocab at 0x2cd53e73470>,\n",
       " 'coli': <gensim.models.keyedvectors.Vocab at 0x2cd53e734a8>,\n",
       " 'baculoviruses': <gensim.models.keyedvectors.Vocab at 0x2cd53e734e0>,\n",
       " 'Sf': <gensim.models.keyedvectors.Vocab at 0x2cd53e73518>,\n",
       " 'system': <gensim.models.keyedvectors.Vocab at 0x2cd53e73550>,\n",
       " 'provider': <gensim.models.keyedvectors.Vocab at 0x2cd53e73588>,\n",
       " 'infected': <gensim.models.keyedvectors.Vocab at 0x2cd53e735c0>,\n",
       " 'producing': <gensim.models.keyedvectors.Vocab at 0x2cd53e735f8>,\n",
       " 'viruses': <gensim.models.keyedvectors.Vocab at 0x2cd53e73630>,\n",
       " 'coinfected': <gensim.models.keyedvectors.Vocab at 0x2cd53e73668>,\n",
       " 'infection': <gensim.models.keyedvectors.Vocab at 0x2cd53e736a0>,\n",
       " 'purify': <gensim.models.keyedvectors.Vocab at 0x2cd53e736d8>,\n",
       " 'mL': <gensim.models.keyedvectors.Vocab at 0x2cd53e73710>,\n",
       " 'resuspended': <gensim.models.keyedvectors.Vocab at 0x2cd53e73748>,\n",
       " 'pellet': <gensim.models.keyedvectors.Vocab at 0x2cd53e73780>,\n",
       " 'lysate': <gensim.models.keyedvectors.Vocab at 0x2cd53e737b8>,\n",
       " 'soluble': <gensim.models.keyedvectors.Vocab at 0x2cd53e737f0>,\n",
       " 'fraction': <gensim.models.keyedvectors.Vocab at 0x2cd53e73828>,\n",
       " 'time': <gensim.models.keyedvectors.Vocab at 0x2cd53e73860>,\n",
       " 'see': <gensim.models.keyedvectors.Vocab at 0x2cd53e73898>,\n",
       " 'below': <gensim.models.keyedvectors.Vocab at 0x2cd53e738d0>,\n",
       " 'finally': <gensim.models.keyedvectors.Vocab at 0x2cd53e73908>,\n",
       " 'glycerol': <gensim.models.keyedvectors.Vocab at 0x2cd53e73940>,\n",
       " 'storage': <gensim.models.keyedvectors.Vocab at 0x2cd53e73978>,\n",
       " 'transformed': <gensim.models.keyedvectors.Vocab at 0x2cd53e739b0>,\n",
       " 'DE': <gensim.models.keyedvectors.Vocab at 0x2cd53e739e8>,\n",
       " 'Novagen': <gensim.models.keyedvectors.Vocab at 0x2cd53e73a20>,\n",
       " 'isopropyl': <gensim.models.keyedvectors.Vocab at 0x2cd53e73a58>,\n",
       " 'd': <gensim.models.keyedvectors.Vocab at 0x2cd53e73a90>,\n",
       " 'thiogalactopyranoside': <gensim.models.keyedvectors.Vocab at 0x2cd53e73ac8>,\n",
       " 'Imidazole': <gensim.models.keyedvectors.Vocab at 0x2cd53e73b00>,\n",
       " 'DTT': <gensim.models.keyedvectors.Vocab at 0x2cd53e73b38>,\n",
       " 'bar': <gensim.models.keyedvectors.Vocab at 0x2cd53e73b70>,\n",
       " 'After': <gensim.models.keyedvectors.Vocab at 0x2cd53e73ba8>,\n",
       " 'incubation': <gensim.models.keyedvectors.Vocab at 0x2cd53e73be0>,\n",
       " 'elute': <gensim.models.keyedvectors.Vocab at 0x2cd53e73c18>,\n",
       " 'elution': <gensim.models.keyedvectors.Vocab at 0x2cd53e73c50>,\n",
       " 'imidazole': <gensim.models.keyedvectors.Vocab at 0x2cd53e73c88>,\n",
       " 'eluted': <gensim.models.keyedvectors.Vocab at 0x2cd53e73cc0>,\n",
       " 'Protein': <gensim.models.keyedvectors.Vocab at 0x2cd53e73cf8>,\n",
       " 'Kinase': <gensim.models.keyedvectors.Vocab at 0x2cd53e73d30>,\n",
       " 'mixed': <gensim.models.keyedvectors.Vocab at 0x2cd53e73d68>,\n",
       " 'harboring': <gensim.models.keyedvectors.Vocab at 0x2cd53e73da0>,\n",
       " 'monomeric': <gensim.models.keyedvectors.Vocab at 0x2cd53e73dd8>,\n",
       " 'complexed': <gensim.models.keyedvectors.Vocab at 0x2cd53e73e10>,\n",
       " 'BSA': <gensim.models.keyedvectors.Vocab at 0x2cd53e73e48>,\n",
       " 'Î¼Ci': <gensim.models.keyedvectors.Vocab at 0x2cd53e73e80>,\n",
       " 'Î³': <gensim.models.keyedvectors.Vocab at 0x2cd53e73eb8>,\n",
       " 'TrisÂ·HCl': <gensim.models.keyedvectors.Vocab at 0x2cd53e73ef0>,\n",
       " 'heparin': <gensim.models.keyedvectors.Vocab at 0x2cd53e73f28>,\n",
       " 'volume': <gensim.models.keyedvectors.Vocab at 0x2cd53e73f60>,\n",
       " 'reactions': <gensim.models.keyedvectors.Vocab at 0x2cd53e73f98>,\n",
       " 'cut': <gensim.models.keyedvectors.Vocab at 0x2cd53e73fd0>,\n",
       " 'slices': <gensim.models.keyedvectors.Vocab at 0x2cd53e78048>,\n",
       " 'stained': <gensim.models.keyedvectors.Vocab at 0x2cd53e78080>,\n",
       " 'dried': <gensim.models.keyedvectors.Vocab at 0x2cd53e780b8>,\n",
       " ...}"
      ]
     },
     "execution_count": 40,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "word_vectors.vocab"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.6.4"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
